<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-08-09 14:27:59 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>4</td>
          <td>48</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>2</td>
          <td>40</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Epithelial-to-mesenchymal transition (EMT) is a major axis of phenotypic plasticity not only in diseased conditions such as cancer metastasis and fibrosis but also during normal development and wound healing. Yet-another important axis of plasticity with metastatic implications includes the cancer stem cell (CSCs) and non-CSC transitions. However, in both processes, epithelial (E) and mesenchymal (M) phenotypes are not merely binary states. Cancer cells acquire a spectrum of phenotypes with traits, properties, and markers of both E and M phenotypes, giving rise to intermediary hybrid (E/M) phenotypes. E/M cells play an important role in tumor initiation, metastasis, and disease progression in multiple cancers. Furthermore, the hybrid phenotypes also play a major role in causing therapeutic resistance in cancer. Here, we discuss how a systems biology perspective on the problem, which is implicit in the ‘Team Medicine’ approach outlined in the theme of this Special Issue of The Journal of Clinical Medicine and includes an interdisciplinary team of experts, is more likely to shed new light on EMT in cancer and help us to identify novel therapeutics and strategies to target phenotypic plasticity in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6867e0a77d67eee55a122d53d387d619f33219b" target='_blank'>
              Phenotypic Plasticity and Cancer: A System Biology Perspective
              </a>
            </td>
          <td>
            A. R. Subbalakshmi, S. Ramisetty, Atish Mohanty, Siddhika Pareek, Dana Do, Sagun Shrestha, Ajaz Khan, Neel Talwar, Tingting Tan, Priya Vishnubhotla, Sharad S Singhal, Ravi Salgia, Prakash Kulkarni
          </td>
          <td>2024-07-23</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2f585a6d644d3577541d684c98016ab2b165e02" target='_blank'>
              Sensing and guiding cell-state transitions by using genetically encoded endoribonuclease-mediated microRNA sensors.
              </a>
            </td>
          <td>
            Lei Wang, Wenlong Xu, Shun Zhang, Gregory C Gundberg, Christine R Zheng, Zhengpeng Wan, Kamila Mustafina, Fabio Caliendo, Hayden Sandt, Roger Kamm, Ron Weiss
          </td>
          <td>2024-07-09</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Therapeutic resistance is the leading cause of treatment failure and death from cancer. While resistance can be driven by genetic mutations, mounting evidence also points to an epigenetic basis of resistance. Much of this epigenetic, or non-genetic, resistance has been attributed to drug-resistant transcriptional cell states that are either induced by drug treatment or pre-exist in a fraction of cells selected by treatment. However, the extent to which long-term resistance is manifested in the early inherent cellular response to drugs is poorly understood, and which aspects, if any, of this early response to drug-induced transcriptional response are evolutionarily conserved properties of cells. To address these questions, we integrate datasets of long-term drug resistance and early drug response data across multiple cell lines with drug response and resistance data from bacteria and yeast. Our findings suggest that cancer cell states in both the drug-naive populations as well as in populations shortly after treatment share transcriptional properties with fully established resistant cell populations, and CRISPR-cas9 knockout of transcription factors predicted to regulate the resistant transcriptional programs result in inceased drug sensitivity. Furthermore, the resistance states manifested as early drug response are evolutionarily conserved. Finally, we show that early resistant states discriminate responders from non-responders across multiple human cancer trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ad4a4d49f9d38cdc5b6bcd770ac201bba19fb9f" target='_blank'>
              Transcriptional Characterization of Resistance in Early Drug Response
              </a>
            </td>
          <td>
            Cole Ruoff, Allison V. Mitchell, Priya Mondal, Vishaka Gopalan, Michael M. Gottesman, Sridhar Hannenhalli
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Leukemia is characterized by oncogenic lesions that result in a block of differentiation while phenotypic plasticity is retained. A better understanding of how these two phenomena arise during leukemogenesis in humans could help inform diagnosis and treatment strategies. Here, we leveraged the well-defined differentiation states during T-cell development to pinpoint the initiation of T-cell acute lymphoblastic leukemia (T-ALL), an aggressive form of childhood leukemia, and study the emergence of phenotypic plasticity. Single-cell whole genome sequencing of leukemic blasts was combined with multiparameter flow cytometry to couple cell identity and clonal lineages. Irrespective of genetic events, leukemia-initiating cells altered their phenotypes by differentiation and de-differentiation. Construction of phylogenies of individual leukemias using somatic mutations revealed that phenotypic diversity is reflected by the clonal structure of the cancer. The analysis also indicated that the acquired phenotypes are heritable and stable. Together, these results demonstrate a transient period of plasticity during leukemia initiation where phenotypic switches appear unidirectional.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0eb1ac967dc33753add0f41321531d1ce4cb054a" target='_blank'>
              Transient Differentiation-State Plasticity Occurs During Acute Lymphoblastic Leukemia Initiation.
              </a>
            </td>
          <td>
            Vera M. Poort, R. Hagelaar, Markus J van Roosmalen, L. Trabut, J. Buijs-Gladdines, Bram van Wijk, Jules P P Meijerink, R. van Boxtel
          </td>
          <td>2024-06-17</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Cancer resistance refers to the ability of cancer cells to evade the effects of treatments, such as chemotherapy and radiation, making it challenging to eliminate tumors. Cancer cells develop resistance through genetic modifications, alterations in cellular pathways, or microenvironmental adaptations that allow cancer cells to persist, proliferate, and even thrive despite the application of therapies designed to eradicate them. Despite efforts to create more potent target-based drugs, the flexibility of acquired resistance necessitates advanced options. Combination therapy and precision immunotherapy has revolutionized treatment but is limited by patient specificity and requires further development. New approaches like small molecules, peptides, and nanotherapeutics aim to overcome resistance by enhancing site-specific delivery and increasing drug concentration inside cancer cells. This review provides a comprehensive overview of the molecular mechanisms underlying cancer drug resistance, including genetic mutations, epigenetic alterations, efflux pump activity, and the influence of the tumor microenvironment. Additionally, the roles of microRNAs, long noncoding RNAs, and cellular processes such as autophagy and hypoxia in mediating resistance are examined. This review seeks to improve patient care and help to the development of more effective cancer medicines by explaining these complicated systems and investigating novel therapeutic alternatives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c262ab9bdd31ff19f10e8cd7a734094b9985bc60" target='_blank'>
              An overview of molecular mechanisms in cancer drug resistance and therapeutic strategies
              </a>
            </td>
          <td>
            Navin Kumar, Ranjeeta Bansala
          </td>
          <td>2024-07-30</td>
          <td>International Journal of Science and Research Archive</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Deficiencies in DNA mismatch repair (MMRd) leave characteristic footprints of microsatellite instability (MSI) in cancer genomes. We used data from the Cancer Genome Atlas and International Cancer Genome Consortium to conduct a comprehensive analysis of MSI-associated cancers, focusing on indel mutational signatures. We classified MSI-high genomes into two subtypes based on their indel profiles: deletion-dominant (MMRd-del) and insertion-dominant (MMRd-ins). Compared with MMRd-del genomes, MMRd-ins genomes exhibit distinct mutational and transcriptomic features, including a higher prevalence of T>C substitutions and related mutation signatures. Short insertions and deletions in MMRd-ins and MMRd-del genomes target different sets of genes, resulting in distinct indel profiles between the two subtypes. In addition, indels in the MMRd-ins genomes are enriched with subclonal alterations that provide clues about a distinct evolutionary relationship between the MMRd-ins and MMRd-del genomes. Notably, the transcriptome analysis indicated that MMRd-ins cancers upregulate immune-related genes, show a high level of immune cell infiltration, and display an elevated neoantigen burden. The genomic and transcriptomic distinctions between the two types of MMRd genomes highlight the heterogeneity of genetic mechanisms and resulting genomic footprints and transcriptomic changes in cancers, which has potential clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b737e04b95df7e27390442dd66d22a5ba373a014" target='_blank'>
              A Subset of Microsatellite Unstable Cancer Genomes Prone to Short Insertions over Deletions Is Associated with Elevated Anticancer Immunity
              </a>
            </td>
          <td>
            Sunmin Kim, Dong-Jin Han, Seo-Young Lee, Youngbeen Moon, Su Jung Kang, Tae-Min Kim
          </td>
          <td>2024-06-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Outcomes for women with breast cancer have improved dramatically in recent decades. However, many patients present with intrinsic drug resistance and others are initially sensitive to anti-cancer drugs but acquire resistance during the course of their treatment, leading to recurrence and/or metastasis. Drug therapy-induced senescence (TIS) is a form of drug resistance characterised by the induction of cell cycle arrest and the emergence of a senescence-associated secretory phenotype (SASP) that can develop in response to chemo- and targeted- therapies. A wide range of anticancer interventions can lead to cell cycle arrest and SASP induction, by inducing genotoxic stress, hyperactivation of signalling pathways or oxidative stress. TIS can be anti-tumorigenic in the short-term, but pro-tumorigenic in the long-term by creating a pro-inflammatory and immunosuppressive microenvironment. Moreover, the SASP can promote angiogenesis and epithelial-mesenchymal transition in neighbouring cells. In this review, we will describe the characteristics of TIS in breast cancer and detail the changes in phenotype that accompany its induction. We also discuss strategies for targeting senescent cancer cells in order to prevent or delay tumour recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1902ff7d1c0379c707774d89c56e002ae6793dc1" target='_blank'>
              Therapy-induced senescence in breast cancer: an overview
              </a>
            </td>
          <td>
            Suraj Narayanan Chembukavu, Andrew J Lindsay
          </td>
          <td>2024-07-25</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfd5cecdc67c5ea14eee4cb838269e77eddb6f01" target='_blank'>
              Farnesyltransferase inhibition overcomes oncogene-addicted non-small cell lung cancer adaptive resistance to targeted therapies
              </a>
            </td>
          <td>
            S. Figarol, C. Delahaye, R. Gence, A. Doussine, J. Cerapio, Mathylda Brachais, Claudine Tardy, N. Béry, R. Asslan, J. Colinge, J. Villemin, Antonio Maraver, Irene Ferrer, Luis Paz-Ares, Linda Kessler, Francis Burrows, Isabelle Lajoie-Mazenc, Vincent Dongay, Clara Morin, Amélie Florent, S. Pagano, E. Taranchon-Clermont, A. Casanova, A. Pradines, Julien Mazieres, Gilles Favre, O. Calvayrac
          </td>
          <td>2024-06-27</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>32</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a particularly aggressive mammary neoplasia with a high fatality rate, mainly because of the development of resistance to administered chemotherapy, the standard treatment for this disease. In this study, we employ both bulk RNA-sequencing and single-cell RNA-sequencing (scRNA-seq) to investigate the transcriptional landscape of TNBC cells cultured in two-dimensional monolayers or three-dimensional spheroids, before and after developing resistance to the chemotherapeutic agents paclitaxel and doxorubicin. Our findings reveal significant transcriptional heterogeneity within the TNBC cell populations, with the scRNA-seq identifying rare subsets of cells that express resistance-associated genes not detected by the bulk RNA-seq. Furthermore, we observe a partial shift towards a highly mesenchymal phenotype in chemoresistant cells, suggesting the epithelial-to-mesenchymal transition (EMT) as a prevalent mechanism of resistance in subgroups of these cells. These insights highlight potential therapeutic targets, such as the PDGF signaling pathway mediating EMT, which could be exploited in this setting. Our study underscores the importance of single-cell approaches in understanding tumor heterogeneity and developing more effective, personalized treatment strategies to overcome chemoresistance in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33e0aae4bc7f6edfa5e8ffddd5c3bb7acfad4a59" target='_blank'>
              Identification of New Chemoresistance-Associated Genes in Triple-Negative Breast Cancer by Single-Cell Transcriptomic Analysis
              </a>
            </td>
          <td>
            S. Foutadakis, Dimitrios Kordias, G. Vatsellas, A. Magklara
          </td>
          <td>2024-06-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="All routine clinical treatments for colorectal cancer include 5-fluorouracil (5-FU), which cannot counteract recurrence and metastases formation. As the pyrimidine analog 5-FU can impact multiple pathways including both DNA and RNA metabolism, studying its mode of actions could lead to improved therapies. Using a dedicated reporter system for lineage-tracing and deep translatome profiling we demonstrate that 5-FU causes some colorectal cancer cells to tolerate the drug, due to a durable translational reprogramming that sustains cell plasticity. This period of drug tolerance coincides with specific translational activation of genes coding for proteins with major pro-tumoral functions. We unravel a major unexpected translational overexpression of the pro-inflammatory and pro-tumoral IL-8 cytokine, alongside other anti-apoptotic, senescence-associated secretory phenotype and cancer-related senescence phenotype genes. Given the adverse prognostic implications of elevated IL-8 levels across various cancers, our findings suggest IL-8 targeting could counteract 5-FU resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3f76dde38afaccb213072b4b26b46e5ff73bd2b" target='_blank'>
              Translational control of cell plasticity drives 5-FU tolerance
              </a>
            </td>
          <td>
            Mounira Chalabi-Dchar, Olivia Villeronce, J. Ripoll, A. Vincent, T. Fenouil, Rita Khoueiry, Jérôme Kucharczak, Laura Jentschel, Frédéric Catez, Arnaud M Vigneron, Julie Tréguier, Céline Mandier, Céline Bouclier, Jihane Vitre, Louise Lagerqvist, Armelle Choquet, Z. Herceg, C. Machon, Jérôme Guitton, Alexandre David, Eric Solary, David Bernard, Nadine Martin, Eric Rivals, N. D. Venezia, Julie Pannequin, J- J. Diaz
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Copy number alterations (CNAs), resulting from the gain or loss of genetic material from as little as 50 base pairs or as big as entire chromosome(s), have been associated with many congenital diseases, de novo syndromes and cancer. It is established that CNAs disturb the dosage of genomic regions including enhancers/promoters, long non-coding RNA and gene(s) among others, ultimately leading to an altered balance of key cellular functions. In cancer, CNAs have been associated with almost all steps of the disease: predisposition, initiation, development, maintenance, response to treatment, resistance, and relapse. Therefore, understanding how specific CNAs contribute to tumourigenesis may provide prognostic insight and ultimately lead to the development of new therapeutic approaches to improve patient outcomes. In this review, we provide a snapshot of what is currently known about CNAs and cancer, incorporating topics regarding their detection, clinical impact, origin, and nature, and discuss the integration of innovative genetic engineering strategies, to highlight the potential for targeting CNAs using novel, dosage-sensitive and less toxic therapies for CNA-driven cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b107cf5f7a3a29e8e0fe916d4102734572eefc3a" target='_blank'>
              Insights into the Clinical, Biological and Therapeutic Impact of Copy Number Alteration in Cancer
              </a>
            </td>
          <td>
            Shannon L. Carey-Smith, R. Kotecha, L. C. Cheung, Sébastien Malinge
          </td>
          <td>2024-06-21</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Cellular responses to environmental stimuli are typically thought to be governed by genetically encoded programs. We demonstrate that melanoma cells can form and maintain cellular memories during the acquisition of therapy resistance that exhibit characteristics of cellular learning and are dependent on the transcription factor AP-1. We show that cells exposed to a low dose of therapy adapt to become resistant to a high dose, demonstrating that resistance was not purely selective. The application of therapy itself results in the encoding of transient gene expression into cellular memory and that this encoding occurs for both transiently induced and probabilistically arising expression. Chromatin accessibility showed concomitant persistence. A two-color AP-1 reporter system showed that these memories are encoded in cis, constituting an example of activating cis epigenetics. Our findings establish the formation and maintenance of cellular memories as a critical aspect of gene regulation during the development of therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a15dd395d0e2a350d250357de4baa259bd2f20" target='_blank'>
              AP-1 Mediates Cellular Adaptation and Memory Formation During Therapy Resistance
              </a>
            </td>
          <td>
            Jingxin Li, Pavithran T. Ravindran, Aoife O’Farrell, Gianna T. Busch, Ryan H. Boe, Zijian Niu, Sean Woo, Margaret C. Dunagin, Naveen Jain, Yogesh Goyal, K. Sarma, M. Herlyn, Arjun Raj
          </td>
          <td>2024-07-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/821cee27b336dd922ca1d195350feabe9168efcf" target='_blank'>
              CRISPR-Cas9 potential for identifying novel therapeutic targets in muscle-invasive bladder cancer.
              </a>
            </td>
          <td>
            Danielle J Smith, S. Lunj, Antony D Adamson, Sankari Nagarajan, Tim A D Smith, Kimberley J Reeves, P. Hoskin, Ananya Choudhury
          </td>
          <td>2024-07-01</td>
          <td>Nature reviews. Urology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eaccde1d6ccb6dacff234a0f6ddffc263c341a3" target='_blank'>
              Targeting oncogene-induced cellular plasticity for tumor therapy
              </a>
            </td>
          <td>
            Bin Li, Lingling Zheng, Jianhua Yang, Lianghu Qu
          </td>
          <td>2024-07-15</td>
          <td>Advanced Biotechnology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ae5740675a47153265ae470439ce458fde35c02" target='_blank'>
              Single-cell sensor analyses reveal signaling programs enabling Ras-G12C drug resistance.
              </a>
            </td>
          <td>
            Jason Z Zhang, Shao-En Ong, D. Baker, D. Maly
          </td>
          <td>2024-08-05</td>
          <td>Nature chemical biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Genomic instability is one of the main drivers of tumorigenesis and the development of hematological malignancies. Cancer cells can remedy chemotherapeutic-induced DNA damage by upregulating DNA-repair genes and ultimately inducing therapy resistance. Nevertheless, the association between the DNA-repair genes, drug resistance, and disease relapse has not been well characterized in acute lymphoblastic leukemia (ALL). This study aimed to explore the role of the DNA-repair machinery and the molecular mechanisms by which it is regulated in early- and late-relapsing pediatric ALL patients. We performed secondary data analysis on the Therapeutically Applicable Research to Generate Effective Treatments (TARGET)—ALL expansion phase II trial of 198 relapsed pediatric precursor B-cell ALL. Comprehensive genetic and epigenetic investigations of 147 DNA-repair genes were conducted in the study. Gene expression was assessed using Microarray and RNA-sequencing platforms. Genomic alternations, methylation status, and miRNA transcriptome were investigated for the candidate DNA-repair genes. We identified three DNA-repair genes, ALKBH3, NHEJ1, and PARP1, that were upregulated in early relapsers compared to late relapsers (p < 0.05). Such upregulation at diagnosis was significantly associated with disease-free survival and overall survival in precursor-B-ALL (p < 0.05). Moreover, PARP1 upregulation accompanied a significant downregulation of its targeting miRNA, miR-1301-3p (p = 0.0152), which was strongly linked with poorer disease-free and overall survivals. Upregulation of DNA-repair genes, PARP1 in particular, increases the likelihood of early relapse of precursor-B-ALL in children. The observation that PARP1 was upregulated in early relapsers relative to late relapsers might serve as a valid rationale for proposing alternative treatment approaches, such as using PARP inhibitors with chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51e9e18af70423e8a01753d2f90a2a9bf475d2db" target='_blank'>
              Genetic and Epigenetic Biomarkers Associated with Early Relapse in Pediatric Acute Lymphoblastic Leukemia: A Focused Bioinformatics Study on DNA-Repair Genes
              </a>
            </td>
          <td>
            Walaa F Albaqami, Ali A. Alshamrani, Ali A. Almubarak, Faris E. Alotaibi, Basil Alotaibi, Abdulrahman M. Alanazi, M. Alotaibi, Ali R. Alhoshani, Homood M. As Sobeai
          </td>
          <td>2024-08-05</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="A fundamental obstacle to tackling the antimicrobial resistance crsisis is identifying mutations that lead to resistance in a given genomic background and environment. We present a high-throughput technique – Quantitative Mutational Scan Sequencing (QMS-Seq) – that enables quantitative comparison of which genes are under antibiotic selection and captures how genetic background influences resistance evolution. We compared four E. coli strains exposed to ciprofloxacin, cycloserine, or nitrofurantoin and identified 975 resistance mutations, many in genes and regulatory regions not previously associated with resistance. QMS-Seq revealed that multi-drug and antibiotic-specific resistance are acquired through categorically different types of mutations, and that minor genotypic differences significantly influence evolutionary routes to resistance. By quantifying mutation frequency with single base pair resolution, QMS-Seq informs about the underlying mechanisms of resistance and identifies mutational hotspots within genes. Our method provides a way to rapidly screen for resistance mutations while assessing the impact of multiple confounding factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baea8e2ed712d39dbedd898121a034064f71df57" target='_blank'>
              High-throughput method rapidly characterizes hundreds of novel antibiotic resistance mutations
              </a>
            </td>
          <td>
            Matthew J. Jago, Jake K. Soley, Stepan Denisov, Calum J. Walsh, Danna R. Gifford, Benjamin P. Howden, M. Lagator
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Epigenetic evolution is a common mechanism used by cancer cells to evade the therapeutic effects of drug treatment. In ovarian cancers, epigenetically-driven resistance may be responsible for a large number of late-stage patient deaths. Here, we describe the first investigation into the role of G-quadruplex (G4) DNA secondary structures in mediating epigenetic regulation in drug-resistant ovarian cancer cells. Through genome-wide mapping of G4s in paired drug-sensitive and drug-resistant cell lines, we find that increased G4 formation is associated with significant increase in gene expression, with high enrichment in signalling pathways previously established to promote drug-resistant states. However, in contrast to previous studies, the expression-enhancing effects of G4s were not found at gene promoters, but intergenic and intronic regions, indicating that G4s promote long-range transcriptional regulation in drug-resistant cells. Furthermore, we discovered that clusters of G4s (super-G4s) are associated with particularly high levels of transcriptional enhancement that surpass the effects of super-enhancers, which act as well established regulatory sites in many cancers. Finally, we demonstrate that targeting G4s with small molecules results in significant down-regulation of pathways associated with drug-resistance, which results in resensitisation of resistant cells to chemotherapy agents. These findings indicate that G4 structures are critical for the epigenetic regulatory networks of drug-resistant cells and may represent a promising target to treat drug-tolerant ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fef77f99d4d1172d6a547e45d37053b1f52fb2ad" target='_blank'>
              G-quadruplex structures regulate long-range transcriptional reprogramming to promote drug resistance in ovarian cancer
              </a>
            </td>
          <td>
            Jenna Robinson, Gem Flint, I. Garner, Silvia Galli, Thomas E. Maher, Marina K. Kuimova, Ramon Vilar, I. McNeish, Robert Brown, Hector Keun, Marco Di Antonio
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Multiple myeloma (MM) is a genetically heterogeneous disease and the management of relapses is one of the biggest clinical challenges. TP53 alterations are established high‐risk markers and are included in the current disease staging criteria. KRAS is the most frequently mutated gene affecting around 20% of MM patients. Applying Clonal Competition Assays (CCA) by co‐culturing color‐labeled genetically modified cell models, we recently showed that mono‐ and biallelic alterations in TP53 transmit a fitness advantage to the cells. Here, we report a similar dynamic for two mutations in KRAS (G12A and A146T), providing a biological rationale for the high frequency of KRAS and TP53 alterations at MM relapse. Resistance mutations, on the other hand, did not endow MM cells with a general fitness advantage but rather presented a disadvantage compared to the wild‐type. CUL4B KO and IKZF1 A152T transmit resistance against immunomodulatory agents, PSMB5 A20T to proteasome inhibition. However, MM cells harboring such lesions only outcompete the culture in the presence of the respective drug. To better prevent the selection of clones with the potential of inducing relapse, these results argue in favor of treatment‐free breaks or a switch of the drug class given as maintenance therapy. In summary, the fitness benefit of TP53 and KRAS mutations was not treatment‐related, unlike patient‐derived drug resistance alterations that may only induce an advantage under treatment. CCAs are suitable models for the study of clonal evolution and competitive (dis)advantages conveyed by a specific genetic lesion of interest, and their dependence on external factors such as the treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b736cb2bf2240ae0e1d6133faf0f979a9afbbf92" target='_blank'>
              Clonal competition assays identify fitness signatures in cancer progression and resistance in multiple myeloma
              </a>
            </td>
          <td>
            L. Haertle, U. Munawar, Hipólito N. C. Hernández, Andrés Arroyo-Barea, T. Heckel, I. Cuenca, Lucia Martin, Carlotta Höschle, Nicole Müller, C. Vogt, Thorsten Bischler, Paula L. del Campo, Seungbin Han, Natalia Buenache, Xiang Zhou, Florian Bassermann, J. Waldschmidt, Torsten Steinbrunn, L. Rasche, T. Stühmer, J. Martínez-López, K. Martin Kortüm, S. Barrio
          </td>
          <td>2024-07-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23cf92719cd49496400fba609df94fa13b009008" target='_blank'>
              Precision medicine in AML: overcoming resistance.
              </a>
            </td>
          <td>
            Samuel Urrutia, Koichi Takahashi
          </td>
          <td>2024-08-01</td>
          <td>International journal of hematology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer heterogeneity presents a significant challenge in clinical therapy, such as over-treatment and drug resistance. These challenges are largely due to its obscure normal epithelial origins, evolutionary stability, and transitions on the cancer subtypes. This study aims to elucidate the cellular emergence and maintenance of heterogeneous breast cancer via quantitative bio-process modeling, with potential benefit to therapeutic strategies for the disease. An endogenous molecular–cellular hypothesis posits that both pathological and physiological states are phenotypes evolved from and shaped by interactions among a number of conserved modules and cellular factors within a biological network. We hereby developed a model of core endogenous network for breast cancer in accordance with the theory, quantifying its intrinsic dynamic properties with dynamic modeling. The model spontaneously generates cell states that align with molecular classifications at both the molecular and modular level, replicating four widely recognized molecular subtypes of the cancer and validating against data extracted from the TCGA database. Further analysis shows that topologically, a singular progression gateway from normal breast cells to cancerous states is identified as the Luminal A-type breast cancer. Activated positive feedback loops are found to stabilize cellular states, while negative feedback loops facilitate state transitions. Overall, more routes are revealed on the cellular transition between stable states, and a traceable count explains the origin of breast cancer heterogeneity. Ultimately, the research intended to strength the search for therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36bf7724db6a608fda16eecf1282fe4b12434998" target='_blank'>
              Heterogeneous Evolution of Breast Cancer Cells—An Endogenous Molecular-Cellular Network Study
              </a>
            </td>
          <td>
            Tianqi Li, Yong-Cong Chen, Ping Ao
          </td>
          <td>2024-07-26</td>
          <td>Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2e387009a4ed97bf637b9dd6a3591f151035e10" target='_blank'>
              Insights into gemcitabine resistance in pancreatic cancer: association with metabolic reprogramming and TP53 pathogenicity in patient derived xenografts
              </a>
            </td>
          <td>
            Mariam M. Konaté, Julia Krushkal, Ming-Chung Li, Li Chen, Yuri Kotliarov, Alida Palmisano, Rini Pauly, Qian Xie, P. M. Williams, Lisa M. McShane, Yingdong Zhao
          </td>
          <td>2024-08-05</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Epitranscriptomics is a field that delves into post-transcriptional changes. Among these modifications, the conversion of adenosine to inosine, traduced as guanosine (A>I(G)), is one of the known RNA-editing mechanisms, catalyzed by ADARs. This type of RNA editing is the most common type of editing in mammals and contributes to biological diversity. Disruption in the A>I(G) RNA-editing balance has been linked to diseases, including several types of cancer. Drug resistance in patients with cancer represents a significant public health concern, contributing to increased mortality rates resulting from therapy non-responsiveness and disease progression, representing the greatest challenge for researchers in this field. The A>I(G) RNA editing is involved in several mechanisms over the immunotherapy and genotoxic drug response and drug resistance. This review investigates the relationship between ADAR1 and specific A>I(G) RNA-edited sites, focusing particularly on breast cancer, and the impact of these sites on DNA damage repair and the immune response over anti-cancer therapy. We address the underlying mechanisms, bioinformatics, and in vitro strategies for the identification and validation of A>I(G) RNA-edited sites. We gathered databases related to A>I(G) RNA editing and cancer and discussed the potential clinical and research implications of understanding A>I(G) RNA-editing patterns. Understanding the intricate role of ADAR1-mediated A>I(G) RNA editing in breast cancer holds significant promise for the development of personalized treatment approaches tailored to individual patients’ A>I(G) RNA-editing profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e391d01dc14e829f34fcad7c86a78c7ddae70fe2" target='_blank'>
              ADAR-Mediated A>I(G) RNA Editing in the Genotoxic Drug Response of Breast Cancer
              </a>
            </td>
          <td>
            Yanara A. Bernal, Eduardo Durán, Isidora Solar, Eduardo A. Sagredo, R. Armisén
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fba17e91584024423ac0c6e3cc0e42afd48ee9aa" target='_blank'>
              Emerging advances in defining the molecular and therapeutic landscape of small-cell lung cancer.
              </a>
            </td>
          <td>
            Triparna Sen, N. Takahashi, Subhamoy Chakraborty, Naoko Takebe, Amin H. Nassar, Nagla A Karim, S. Puri, Abdul Rafeh Naqash
          </td>
          <td>2024-07-04</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4297de6ca1f2c34995dc4df4f4fa137e566d405" target='_blank'>
              Potent therapeutic strategy in gastric cancer with microsatellite instability-high and/or deficient mismatch repair.
              </a>
            </td>
          <td>
            A. Ooki, H. Osumi, K. Yoshino, Kensei Yamaguchi
          </td>
          <td>2024-06-26</td>
          <td>Gastric cancer : official journal of the International Gastric Cancer Association and the Japanese Gastric Cancer Association</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cf8519f1fd7b9c6b19231a2f05380848e656a1a" target='_blank'>
              Altered ribosomal profile in acquired resistance and reversal associates with pathological response to chemotherapy in inflammatory breast cancer
              </a>
            </td>
          <td>
            G. Devi, Pritha Pai, Seayoung Lee, Matthew W. Foster, Dorababu S. Sannareddy, Francois Bertucci, Naoto Ueno, S. V. van Laere
          </td>
          <td>2024-07-29</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c82722b4065aa54e731d4f283926374f9696da" target='_blank'>
              Chromosomal instability as a driver of cancer progression.
              </a>
            </td>
          <td>
            Xuelan Chen, Albert S. Agustinus, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-29</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Next-generation sequencing (NGS) technology has revolutionized cancer research and treatment by enabling comprehensive analysis of genetic mutations, alterations, and expression profiles. It allows for the identification of cancer-driving mutations, helping in the development of targeted therapies. NGS provides detailed insights into tumor heterogeneity, resistance mechanisms, and clonal evolution. Its high-throughput capacity facilitates large-scale studies, improving our understanding of cancer genomics. By enabling personalized treatment plans based on individual genetic profiles, NGS holds promise for more effective and tailored cancer therapies. Early reviews on cancer genomics often lacked comprehensive coverage of emerging technologies. They missed in-depth analysis of NGS advancements, their impact on cancer research, and clinical applications. The review addresses this gap by reviving a thorough examination of NGS methods, their role in identifying genetic mutations, and their potential in personalized cancer treatment, thus providing essential insights into the evolving landscape of cancer genomics. The article covers the advancements in technology and bioinformatic approaches for NGS data analysis. It delves into NGS applications in research and diagnostics, particularly for solid cancer diagnosis. The review highlights specific cancer types, including hereditary breast cancer, melanoma, prostate cancer, thyroid cancer, lung cancer, and colorectal cancer. It explores NGS contribution in understanding the genetic basis of these cancers and its potential for enhancing personalized diagnosis and treatment strategies. This review rectifies early lacunas by providing a comprehensive and updated examination of NGS technology, addressing gaps in previous analyses and emphasizes bioinformatic approaches for NGS data analysis, crucial for interpreting vast genomic data accurately. The review meets the current need for a thorough understanding of NGS’s role in personalized cancer treatment and research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39fa1bb9c8e82b88cec555956d178d8b56b6683c" target='_blank'>
              Next-generation sequencing technology in cancer
              </a>
            </td>
          <td>
            P. Krubaa, Dr. Anand Mohan Jha, Prof Dr.Ammar A.Razzak Mahmood, Dr. Anil Kumar, D. Abraham
          </td>
          <td>2024-07-18</td>
          <td>International Journal of Trends in OncoScience</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0807e64db184649a4b0df3bd08502c4d34be0a53" target='_blank'>
              Programming tumor evolution with selection gene drives to proactively combat drug resistance.
              </a>
            </td>
          <td>
            Scott M. Leighow, Josh Reynolds, Ivan Sokirniy, Shun Yao, Zeyu Yang, Haider Inam, Dominik Wodarz, Marco Archetti, Justin Pritchard
          </td>
          <td>2024-07-04</td>
          <td>Nature biotechnology</td>
          <td>2</td>
          <td>7</td>
        </tr>

        <tr id="Our research aimed to identify new therapeutic targets for Lung adenocarcinoma (LUAD), a major subtype of non-small cell lung cancer known for its low 5-year survival rate of 22%. By employing a comprehensive methodological approach, we analyzed bulk RNA sequencing data from 513 LUAD and 59 non-tumorous tissues, identifying 2,688 differentially expressed genes. Using Mendelian randomization (MR), we identified 74 genes with strong evidence for a causal effect on risk of LUAD. Survival analysis on these genes revealed significant differences in survival rates for 13 of them. Our pathway enrichment analysis highlighted their roles in immune response and cell communication, deepening our understanding. We also utilized single-cell RNA sequencing (scRNA-seq) to uncover cell type-specific gene expression patterns within LUAD, emphasizing the tumor microenvironment’s heterogeneity. Pseudotime analysis further assisted in assessing the heterogeneity of tumor cell populations. Additionally, protein-protein interaction (PPI) network analysis was conducted to evaluate the potential druggability of these identified genes. The culmination of our efforts led to the identification of five genes (tier 1) with the most compelling evidence, including SECISBP2L, PRCD, SMAD9, C2orf91, and HSD17B13, and eight genes (tier 2) with convincing evidence for their potential as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e174b3e000455b97e2cc57b71e1d6819743b30f" target='_blank'>
              Identifying potential therapeutic targets in lung adenocarcinoma: a multi-omics approach integrating bulk and single-cell RNA sequencing with Mendelian randomization
              </a>
            </td>
          <td>
            Youpeng Chen, Enzhong Li, Zhenglin Chang, Tingting Zhang, Zhenfeng Song, Haojie Wu, Zhangkai J. Cheng, Baoqing Sun
          </td>
          <td>2024-07-18</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dc719f0ae90529603e70fa22df6d2a71e97dd8" target='_blank'>
              In vivo single-cell CRISPR uncovers distinct TNF programmes in tumour evolution.
              </a>
            </td>
          <td>
            Peter F Renz, Umesh Ghoshdastider, Simona Baghai Sain, Fabiola Valdivia-Francia, Ameya Khandekar, Mark Ormiston, Martino Bernasconi, Clara Duré, Jonas A. Kretz, Minkyoung Lee, Katie Hyams, Merima Forny, Marcel Pohly, Xenia Ficht, Stephanie J Ellis, A. Moor, Ataman Sendoel
          </td>
          <td>2024-07-17</td>
          <td>Nature</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2196d99b39ce23824b3c9030fb9688e8f81117c7" target='_blank'>
              Replication timing alterations are associated with mutation acquisition during breast and lung cancer evolution
              </a>
            </td>
          <td>
            M. Dietzen, Haoran Zhai, Olivia Lucas, O. Pich, Christopher Barrington, Wei-Ting Lu, S. Ward, Yanping Guo, R. Hynds, S. Zaccaria, C. Swanton, N. Mcgranahan, N. Kanu
          </td>
          <td>2024-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="PURPOSE
In multiple myeloma (MM), therapy-induced clonal evolution is associated with treatment resistance and is one of the most important hindrances toward a cure for MM. To further understand the molecular mechanisms controlling the clonal evolution of MM, we applied single-cell RNA-sequencing (scRNA-seq) to paired diagnostic and post-treatment bone marrow (BM) samples.


EXPERIMENTAL DESIGN
scRNA-seq was performed on 38 BM samples from patients with monoclonal gammopathy of undetermined significance (MGUS) (n = 1), MM patients at diagnosis (n = 19), MM post-treatment (n = 17), and one healthy donor. The single-cell transcriptome data of malignant plasma cells and the surrounding immune microenvironment were analyzed.


RESULTS
Profiling by scRNA-seq data revealed three primary trajectories of transcriptional evolution after treatment: clonal elimination in patients with undetectable minimal residual disease (MRD-), as well as clonal stabilization and clonal selection in detectable MRD (MRD+) patients. We noted a metabolic shift towards fatty acid oxidation in cycling-resistant plasma cells (PCs), while selective PCs favored the NF-κB pathway. Intriguingly, when comparing the genetic and transcriptional dynamics, we found a significant correlation between genetic and non-genetic factors in driving the clonal evolution. Furthermore, we identified variations in cellular interactions between malignant plasma cells and the tumor microenvironment (TME). Selective PCs showed the most robust cellular interactions with the TME.


CONCLUSIONS
These data suggest that MM cells could rapidly adapt to induction treatment through transcriptional adaptation, metabolic adaptation, and specialized immune evasion. Targeting therapy-induced resistance mechanisms may help to avert refractory disease in multiple myeloma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da4967fd77a9456d257d33b503e573eb358b554d" target='_blank'>
              Identification of therapy-induced clonal evolution and resistance pathways in minimal residual clones in multiple myeloma through single-cell sequencing.
              </a>
            </td>
          <td>
            Jian Cui, Xiaoyun Li, S. Deng, C. Du, H. Fan, W. Yan, Jingyu Xu, Xiaoqing Li, Tengteng Yu, Shuaishuai Zhang, R. Lv, W. Sui, Mu Hao, Xin Du, Yan Xu, S. Yi, D. Zou, Tao Cheng, Lugui Qiu, Xin Gao, Gang An
          </td>
          <td>2024-06-20</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed1698b10d6cf746129577cdea369845de24a960" target='_blank'>
              Signaling dynamics in coexisting monoclonal cell subpopulations unveil mechanisms of resistance to anti-cancer compounds
              </a>
            </td>
          <td>
            Claire E Blanchard, Alison T Gomeiz, Kyle Avery, Emna El Gazzah, Abduljalil M. Alsubaie, M. Sikaroodi, Ylenia Chiari, Chelsea Ward, Jonathan Sanchez, Virginia Espina, Emanuel F. Petricoin, E. Baldelli, M. Pierobon
          </td>
          <td>2024-07-26</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/795e6a16077e23dab48be945dcd0a59b697bb23e" target='_blank'>
              Mechanism of ERBB2 gene overexpression by the formation of super-enhancer with genomic structural abnormalities in lung adenocarcinoma without clinically actionable genetic alterations
              </a>
            </td>
          <td>
            S. Kaneko, Ken Takasawa, Ken Asada, K. Shiraishi, Noriko Ikawa, Hidenori Machino, Norio Shinkai, M. Matsuda, Mari Masuda, Shungo Adachi, Satoshi Takahashi, Kazuma Kobayashi, Nobuji Kouno, Amina Bolatkan, M. Komatsu, M. Yamada, M. Miyake, Hirokazu Watanabe, A. Tateishi, T. Mizuno, Y. Okubo, Masami Mukai, Tatsuya Yoshida, Yukihiro Yoshida, H. Horinouchi, Shun-ichi Watanabe, Y. Ohe, Yasushi Yatabe, V. Saloura, Takashi Kohno, R. Hamamoto
          </td>
          <td>2024-06-11</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Introduction A hallmark of small cell lung cancer (SCLC) is its recalcitrance to therapy. While most SCLCs respond to frontline therapy, resistance inevitably develops. Identifying phenotypes potentiating chemoresistance and immune evasion is a crucial unmet need. Previous reports have linked upregulation of the DNA damage response (DDR) machinery to chemoresistance and immune evasion across cancers. However, it is unknown if SCLCs exhibit distinct DDR phenotypes. Methods To study SCLC DDR phenotypes, we developed a new DDR gene analysis method and applied it to SCLC clinical samples, in vitro, and in vivo model systems. We then investigated how DDR regulation is associated with SCLC biology, chemotherapy response, and tumor evolution following therapy. Results Using multi-omic profiling, we demonstrate that SCLC tumors cluster into three DDR phenotypes with unique molecular features. Hallmarks of these DDR clusters include differential expression of DNA repair genes, increased replication stress, and heightened G2/M cell cycle arrest. SCLCs with elevated DDR phenotypes exhibit increased neuroendocrine features and decreased “inflamed” biomarkers, both within and across SCLC subtypes. Treatment naive DDR status identified SCLC patients with different responses to frontline chemotherapy. Tumors with initial DDR Intermediate and DDR High phenotypes demonstrated greater tendency for subtype switching and emergence of heterogeneous phenotypes following treatment. Conclusions We establish that SCLC can be classified into one of three distinct, clinically relevant DDR clusters. Our data demonstrates that DDR status plays a key role in shaping SCLC phenotypes, chemotherapy response, and patterns of tumor evolution. Future work targeting DDR specific phenotypes will be instrumental in improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d740c08f76c9a3391148ca3278218472f317ff1" target='_blank'>
              DNA damage response signatures are associated with frontline chemotherapy response and routes of tumor evolution in extensive stage small cell lung cancer
              </a>
            </td>
          <td>
            Benjamin B. Morris, Simon Heeke, Yuanxin Xi, L. Diao, Qi Wang, Pedro Rocha, Edurne Arriola, Myung Chang Lee, Darren R. Tyson, K. Concannon, K. Ramkumar, C. Stewart, R. Cardnell, Runsheng Wang, Vito Quaranta, Jing Wang, J. Heymach, B. Nabet, D. Shames, C. Gay, L. Byers
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>103</td>
        </tr>

        <tr id="Simple Summary Diffuse large B-cell lymphoma is the most common type of lymphoma. Despite initial treatment, up to 40% of patients do not achieve a cure and need second-line therapy. For those experiencing late relapse or lacking access to CAR-T cell therapy, platinum-based chemotherapy followed by stem cell transplantation remains the standard of care. In this study, we used genomewide CRISPR/Cas9 screens and single-gene knockout experiments to identify the genes associated with responses to platinum-based drugs. We provide a comprehensive list of genes involved in the response to cisplatin in DLBCL. Our functional experiments highlight the critical roles of the DNA damage response genes XPA and ERCC6, in addition to BTK, in the response to platinum-based drugs. Additionally, we show that inhibition of BTK at lower concentrations sensitizes DLBCL cells to platinum-based drugs. Abstract The recurrence of diffuse large B-cell lymphoma (DLBCL) has been observed in 40% of cases. The standard of care for refractory/relapsed DLBCL (RR-DLBCL) is platinum-based treatment prior to autologous stem cell transplantation; however, the prognosis for RR-DLBCL patients remains poor. Thus, to identify genes affecting the cisplatin response in DLBCL, cisplatin-based whole-genome CRISPR-Cas9 knockout screens were performed in this study. We discovered DNA damage response (DDR) pathways as enriched among identified sensitizing CRISPR-mediated gene knockouts. In line, the knockout of the nucleotide excision repair genes XPA and ERCC6 sensitized DLBCL cells to platinum drugs irrespective of proliferation rate, thus documenting DDR as essential for cisplatin sensitivity in DLBCL. Functional analysis revealed that the loss of XPA and ERCC6 increased DNA damage levels and altered cell cycle distribution. Interestingly, we also identified BTK, which is involved in B-cell receptor signaling, to affect cisplatin response. The knockout of BTK increased cisplatin sensitivity in DLBCL cells, and combinatory drug screens revealed a synergistic effect of the BTK inhibitor, ibrutinib, with platinum drugs at low concentrations. Applying local and external DLBCL cohorts, we addressed the clinical relevance of the genes identified in the CRISPR screens. BTK was among the most frequently mutated genes with a frequency of 3–5%, and XPA and ERCC6 were also mutated, albeit at lower frequencies. Furthermore, 27–54% of diagnostic DLBCL samples had mutations in pathways that can sensitize cells to cisplatin. In conclusion, this study shows that XPA and ERCC6, in addition to BTK, are essential for the response to platinum-based drugs in DLBCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2794e056513b2d174f98b17ca50338733fe72c8" target='_blank'>
              CRISPR-Cas9 Knockout Screens Identify DNA Damage Response Pathways and BTK as Essential for Cisplatin Response in Diffuse Large B-Cell Lymphoma
              </a>
            </td>
          <td>
            I. I. Issa, Hanne Due, R. Brøndum, Vidthdyan Veeravakaran, Hulda Haraldsdóttir, Cathrine Sylvester, Asta Brogaard, Soniya Dhanjal, Bernhard Schmierer, K. Dybkær
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abnormal cell proliferation and growth leading to cancer primarily result from cumulative genome mutations. Single gene mutations alone do not fully explain cancer onset and progression; instead, clustered mutations—simultaneous occurrences of multiple mutations—are considered to be pivotal in cancer development and advancement. These mutations can affect different genes and pathways, resulting in cells undergoing malignant transformation with multiple functional abnormalities. Clustered mutations influence cancer growth rates, metastatic potential, and drug treatment sensitivity. This summary highlights the various types and characteristics of clustered mutations to understand their associations with carcinogenesis and discusses their potential clinical significance in cancer. As a unique mutation type, clustered mutations may involve genomic instability, DNA repair mechanism defects, and environmental exposures, potentially correlating with responsiveness to immunotherapy. Understanding the characteristics and underlying processes of clustered mutations enhances our comprehension of carcinogenesis and cancer progression, providing new diagnostic and therapeutic approaches for cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d085d5b52cda0d463d3c07016cf9320b287c441e" target='_blank'>
              Exploring the Role of Clustered Mutations in Carcinogenesis and Their Potential Clinical Implications in Cancer
              </a>
            </td>
          <td>
            Yi Li, Rui Zhu, Jiaming Jin, Haochuan Guo, Jiaxi Zhang, Zhiheng He, Tingming Liang, Li Guo
          </td>
          <td>2024-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/589af1da53c29b65656184828eec590acfbf3b30" target='_blank'>
              Genetic and epigenetic instability as an underlying driver of progression and aggressive behavior in IDH-mutant astrocytoma
              </a>
            </td>
          <td>
            Timothy E Richardson, Jamie M. Walker, Dolores Hambardzumyan, Steven Brem, K. Hatanpaa, Mariano S Viapiano, Balagopal Pai, M. Umphlett, Oren J. Becher, M. Snuderl, S. McBrayer, Kalil G. Abdullah, N. Tsankova
          </td>
          <td>2024-07-16</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Objective To identify nivolumab resistance-related genes in patients with head and neck squamous cell carcinoma (HNSCC) using single-cell and bulk RNA-sequencing data. Methods The single-cell and bulk RNA-sequencing data downloaded from the Gene Expression Omnibus database were analyzed to screen out differentially expressed genes (DEGs) between the nivolumab resistant and nivolumab sensitive patients using R software. The Least Absolute Shrinkage Selection Operator (LASSO) regression and Recursive Feature Elimination (RFE) algorithm were performed to identify key genes associated with nivolumab resistance. Functional enrichment of DEGs was analyzed with Gene Ontology and Kyoto Encyclopedia of Genes and Genomes analyses. The relationships of key genes with immune cell infiltration, differentation trajectory, dynamic gene expression profiles, and ligand-receptor interaction were explored. Results We found 83 DEGs. They were mainly enriched in T-cell differentiation, PD-1 and PD-L1 checkpoint pathways, and T-cell receptor pathways. In six key genes identified using machine learning algorithms, only PPP1R14A gene was differentially expressed between the nivolumab resistant and nivolumab sensitive groups both before and after immunotherapy (P < 0.05). The high PPP1R14A gene expression group had lower immune score (P < 0.01), higher expression of immunosuppressive factors (such as PDCD1, CTLA4, and PDCD1LG2) (r > 0, P < 0.05), lower differentiation of infiltrated immune cells (P < 0.05), and a higher degree of interaction between HLA and CD4 (P < 0.05). Conclusions PPP1R14A gene is closely associated with resistance to nivolumab in HNSCC patients. Therefore, PPP1R14A may be a target to ameliorate nivolumab resistance of HNSCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e5e0e948e87319aaf4421e185ddbf3b11ef5c14" target='_blank'>
              PPP1R14A is Associated with Immunotherapy Resistance in Head and Neck Squamous Cell Carcinoma Identified by Single-Cell and Bulk RNA-Sequencing.
              </a>
            </td>
          <td>
            Junjie Ma, Lei Zhang, Jin Lu, Hao-Xuan Zhang
          </td>
          <td>2024-06-18</td>
          <td>Chinese medical sciences journal = Chung-kuo i hsueh k'o hsueh tsa chih</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Replication repair deficiency (RRD) is a pan-cancer mechanism characterized by abnormalities in the DNA mismatch repair (MMR) system due to pathogenic variants in the PMS2, MSH6, MSH2 or MLH1 genes, and/ or in the polymerase-proofreading genes, POLE and POLD1. RRD predisposition syndromes [constitutional MMR deficiency (CMMRD), Lynch, polymerase-proofreading associated polyposis (PPAP)] share overlapping phenotypic and biological characteristics. Moreover, cancers stemming from germline defects of one mechanism can acquire somatic defects in another, leading to complete RRD. Here we describe the recent advances in the diagnostics, surveillance, and clinical management for children with RRD syndromes. For patients with CMMRD, new data combining clinical insights and cancer genomics have revealed genotype-phenotype associations, helped in the development of novel functional assays, diagnostic guidelines, and surveillance recommendations. Recognition of non-gastrointestinal/ genitourinary malignancies, particularly aggressive brain tumors, in select children with Lynch and PPAP syndromes harbouring an RRD biology have led to new management considerations. Additionally, universal hypermutation and microsatellite instability have allowed immunotherapy to be a paradigm shift in the treatment of RRD cancers independent of their germline etiology. These advances have also stimulated a need for expert recommendations regarding genetic counselling for these patients and their families. Future collaborative work will focus on newer technologies such as quantitative measurement of circulating tumor DNA and functional genomics to tailor surveillance and clinical care, improving immune surveillance, develop prevention strategies, and deliver these novel discoveries to resource-limited settings to maximize benefits for patients globally.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1f07f0c1be296ec56b40417a06c4f28b0fe8a9c" target='_blank'>
              Clinical updates and surveillance recommendations for DNA replication-repair deficiency syndromes in children and young adults.
              </a>
            </td>
          <td>
            Anirban Das, S. MacFarland, Julia Meade, Jordan R Hansford, Kami Wolfe Schneider, Roland P Kuiper, M. Jongmans, Harry Lesmana, Kris Ann P Schultz, Kim E Nichols, C. Durno, Kristin Zelley, Christopher C Porter, Lisa J States, S. Ben-Shachar, Sharon A Savage, J. Kalish, M. Walsh, Hamish S. Scott, Sharon E. Plon, U. Tabori
          </td>
          <td>2024-06-11</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Simple Summary Lung adenocarcinoma, the most prevalent type of lung cancer, presents a significant treatment challenge owing to drug resistance. This study aimed to investigate the role of long non-coding RNAs (lncRNAs) in promoting intrinsic resistance in three lung adenocarcinoma cell lines from the onset of treatment. Drug-tolerant persister (DTP) cells, a subset of cancer cells with the ability to survive and proliferate after exposure to therapeutic drugs, were generated. RNA sequencing was used to investigate the differential expression of lncRNAs, and the clinical relevance of lncRNAs was assessed in a cohort of lung adenocarcinoma patients from The Cancer Genome Atlas. Knockdown of these lncRNAs increases the sensitivity of the analyzed cell lines to therapeutic drugs. This study provides an opportunity to investigate the role of lncRNAs in the genetic and epigenetic mechanisms underlying intrinsic resistance. Abstract Lung adenocarcinoma is the most prevalent form of lung cancer, and drug resistance poses a significant obstacle in its treatment. This study aimed to investigate the overexpression of long non-coding RNAs (lncRNAs) as a mechanism that promotes intrinsic resistance in tumor cells from the onset of treatment. Drug-tolerant persister (DTP) cells are a subset of cancer cells that survive and proliferate after exposure to therapeutic drugs, making them an essential object of study in cancer treatment. The molecular mechanisms underlying DTP cell survival are not fully understood; however, long non-coding RNAs (lncRNAs) have been proposed to play a crucial role. DTP cells from lung adenocarcinoma cell lines were obtained after single exposure to tyrosine kinase inhibitors (TKIs; erlotinib or osimertinib). After establishing DTP cells, RNA sequencing was performed to investigate the differential expression of the lncRNAs. Some lncRNAs and one mRNA were overexpressed in DTP cells. The clinical relevance of lncRNAs was evaluated in a cohort of patients with lung adenocarcinoma from The Cancer Genome Atlas (TCGA). RT–qPCR validated the overexpression of lncRNAs and mRNA in the residual DTP cells and LUAD biopsies. Knockdown of these lncRNAs increases the sensitivity of DTP cells to therapeutic drugs. This study provides an opportunity to investigate the involvement of lncRNAs in the genetic and epigenetic mechanisms that underlie intrinsic resistance. The identified lncRNAs and CD74 mRNA may serve as potential prognostic markers or therapeutic targets to improve the overall survival (OS) of patients with lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b47f1b958113e3cfa109f6bab13ad3bc4cd8c921" target='_blank'>
              Transcriptomic Analysis Reveals Early Alterations Associated with Intrinsic Resistance to Targeted Therapy in Lung Adenocarcinoma Cell Lines
              </a>
            </td>
          <td>
            Mario Perez-Medina, J. Lopez-Gonzalez, Jesus J. Benito-Lopez, Santiago Ávila-Ríos, Maribel Soto-Nava, M. Matías-Florentino, A. Méndez-Tenorio, M. Galicia-Velasco, R. Chavez-Dominguez, S. Meza-Toledo, D. Aguilar-Cázares
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Recombinant adeno-associated virus (rAAV) is a widely used viral vector for gene therapy. Despite its clinical efficacy, the manufacturing of rAAV faces challenges in productivity and quality, leading to limited availability. To address the growing demand, next-generation process development should be informed by a mechanistic understanding of the cellular response to rAAV. In this study, we performed transcriptomic analysis of 5 cell lines with variable capacities for rAAV production. Using an intersectional approach, we assessed the transcriptional response to rAAV production and compared transcriptional profiles between high and baseline producers to identify possible targets for enhancing production. Modulation of cell cycle and nucleosome components suggested a reduction of proliferative capacity and a shift toward DNA replication to support rAAV production. During rAAV production, we observed upregulation of several core functions including transcription, stress response, and Golgi and endoplasmic reticulum organization. Conversely, inhibitors of DNA-binding proteins and mitochondrial components were consistently downregulated during rAAV production. We next performed a drug connectivity analysis of these results and identified 5 classes of drugs predicted to enhance rAAV production. Validation studies confirmed the efficacy of HDAC and microtubule inhibitors. Our data uncover novel and previously identified pathways that may enhance rAAV productivity, potentially enabling a path to engineer improved processes and cell lines for higher yields and better quality rAAV production. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b270da94a458df57bb4725ecb5dd57f8a71f23d" target='_blank'>
              Transcriptomics-informed pharmacology identifies epigenetic and cell cycle regulators as enhancers of AAV production
              </a>
            </td>
          <td>
            Joshua Tworig, Francis Grafton, Markus Hörer, Christopher A. Reid, M. Mandegar
          </td>
          <td>2024-06-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Lung cancer patients treated with targeted therapies frequently respond well but invariably relapse due to the development of drug resistance. Drug resistance is in part mediated by a subset of cancer cells termed “drug-tolerant persisters” (DTPs), which enter a dormant, slow-cycling state that enables them to survive drug exposure. DTPs also exhibit stem cell-like characteristics, broad epigenetic reprogramming, altered metabolism, and a mutagenic phenotype mediated by adaptive mutability. While several studies have characterised the transcriptional changes that lead to the altered phenotypes exhibited in DTPs, these studies have focused predominantly on protein coding changes. As long non-coding RNAs (lncRNAs) are also implicated in the phenotypes altered in DTPs, it is likely that they play a role in the biology of drug tolerance. In this review, we outline how lncRNAs may contribute to the key characteristics of DTPs, their potential roles in tolerance to targeted therapies, and the emergence of genetic resistance in lung adenocarcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83141f0410de3ec7e33ac04641da48080cc097e5" target='_blank'>
              The Potential Links between lncRNAs and Drug Tolerance in Lung Adenocarcinoma
              </a>
            </td>
          <td>
            William J. H. Davis, Catherine J. Drummond, Sarah D. Diermeier, Glen Reid
          </td>
          <td>2024-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2239c71d70f51fc31d667365569c847abafe0d7a" target='_blank'>
              Novel drug resistance mechanisms and drug targets in BRAF-mutated peritoneal metastasis from colorectal cancer
              </a>
            </td>
          <td>
            Christin Lund-Andersen, A. Torgunrud, Chakravarthi Kanduri, V. Dagenborg, I. S. Frøysnes, Mette M. Larsen, Ben Davidson, Stein G Larsen, K. Flatmark
          </td>
          <td>2024-07-09</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Tumors often show an initial response to chemotherapy, but then develop resistance, leading to relapse and poor prognosis. We hypothesized that a genomic comparison of mutations in pre-treated versus treatment-naive tumors would serve to identify genes that confer resistance. A challenge in such an analysis is that therapy alters mutation burdens and signatures, confounding association studies and complicating identifying causal, selected mutations. We developed DiffInvex, a framework for identifying changes in selection acting on individual genes in somatic genomes. Crucially, DiffInvex draws on a mutation rate baseline that accounts for these shifts in neutral mutagenesis during cancer evolution. We applied DiffInvex to 9,953 cancer whole-genomes from 29 cancer types from 8 studies, containing both WGS of treatment-naive tumors and tumors pre-treated by various drugs, identifying genes where point mutations are under conditional positive or negative selection for a certain chemotherapeutic, suggesting resistance mechanisms occurring via point mutation. DiffInvex confirmed well-known chemoresistance-driver mutations in EGFR, ESR1, KIT and AR genes as being under conditional positive selection, with additional cancer types identified for EGFR and KIT. Additionally, DiffInvex identified 11 genes with treatment-associated selection for different classes of therapeutics. In most cases, these genes were common cancer genes including PIK3CA, APC, MAP2K4 and MAP3K1. This suggests that tumor resistance to therapy via mutation often occurs via selective advantages conferred by known driver genes, rather than via mutations in specialized resistance genes. Various gene-chemotherapy associations were further supported in tests for functional impact of mutations, again implemented in a conditional selection setting, as well as replicating in independent panel or exome sequencing data. In addition to nominating drug resistance genes that could be targeted by future therapeutics, DiffInvex can also be applied to diverse analysis in cancer evolution, such as comparing normal and tumoral tissues, or analyzing subclonal evolution, identifying changes in selection over time.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71b71e5df0edb673b45bd36f564fa0da7eae40ed" target='_blank'>
              The DiffInvex evolutionary model for conditional somatic selection identifies chemotherapy resistance genes in 10,000 cancer genomes
              </a>
            </td>
          <td>
            Ahmed Khalil, F. Supek
          </td>
          <td>2024-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a4fb1e6fb2869d1ffa14cf25654e46c58ec040e" target='_blank'>
              Beyond the code: the role of histone methylation in cancer resistance and therapy
              </a>
            </td>
          <td>
            Daniel Noerenberg, Frederik Damm
          </td>
          <td>2024-06-12</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="RNA modification pathways are mis-regulated in multiple types of human cancer. To comprehensively identify cancer-relevant RNA modifications and their regulators, we screened all 150 annotated human RNA modifying proteins across 18 different normal and cancer cell lines using a CRISPR-based genetic knockout system. Fifty RNA modifying proteins were essential for survival of at least one cell type. A third of these essential genes were amplified in 38 different human primary cancer types and potentially drive cancer growth. Unexpectedly, the number of essential genes per cell line varied considerably, and this variation was not due to tissue of origin. Instead, we found that cancer cell-specific mitochondrial metabolic plasticity was responsible for the unique requirement of certain RNA modifications. For example, leukemia cells with high intrinsic drug tolerance required mitochondrial flexibility to survive treatment with the anti-leukemic drugs cytarabine and venetoclax. Synthetic lethality screens revealed that drug-resistance is abolished by deleting the mitochondrial methyltransferase TRMT5, which is responsible for the formation of N1-methylguanosine (m1G) in the tRNA anticodon loop. In summary, our study identifies cancer-relevant RNA modifying enzymes, and reveals a novel promising drug target for therapy-resistant acute myeloid leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7426bf54469ae857a96fb69b5f2913c6be2bfd40" target='_blank'>
              Unbiased functional genetic screens reveal essential RNA modifications in human cancer and drug resistance
              </a>
            </td>
          <td>
            C. Pauli, M. Kienhöfer, M. Blank, Oguzhan Begik, Christian Rohde, D. Heid, Fu Xu, Sarah Miriam Naomi Zimmermann, Katharina Weidenauer, Sylvain Delaunay, Nadja Krall, Katrin Trunk, Duoduo Zhao, Fengbiao Zhou, Anke Heit-Mondrzyk, U. Platzbecker, Claudia Baldus, H. Serve, M. Bornhäuser, Cathrine Broberg Vågbø, Salvador Aznar Benitah, Jeroen Krijgsveld, E. Novoa, Carsten Müller-Tidow, Michaela Frye
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Background Neoadjuvant chemoradiotherapy (nCRT) is the standard treatment for locally advanced rectal cancer, but only 20–40% of patients completely respond to this treatment. Methods To define the molecular features that are associated with response to nCRT, we generated and collected genomic and transcriptomic data from 712 cancers prior to treatment from our own data and from publicly available data. Results We found that patients with a complete response have decreased risk of both local recurrence and future metastasis. We identified multiple differences in DNA mutations and transcripts between complete and incomplete responders. Complete responder tumors have a higher tumor mutation burden and more significant co-occurring mutations than the incomplete responder tumors. In addition, mutations in DNA repair genes (across multiple mechanisms of repair) were enriched in complete responders and they also had lower expression of these genes indicating that defective DNA repair is associated with complete response to nCRT. Using logistic regression, we identified three significant predictors of complete response: tumor size, mutations within specific network genes, and the existence of three or more specific co-occurrent mutations. In incompletely responder tumors, abnormal cell-cell interaction and increased cancer associated fibroblasts were associated with recurrence. Additionally, gene expression analysis identified a subset of immune hot tumors with worse outcomes and upregulated of immune checkpoint proteins. Conclusions Overall, our study provides a comprehensive understanding of the molecular features associated with response to nCRT and the molecular differences in non-responder tumors that later reoccur. This knowledge may provide critical insight for the development of precision therapy for rectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b58c04acad0e1548042e7825a154a041443bca6e" target='_blank'>
              Molecular differentiation between complete and incomplete responders to neoadjuvant therapy in rectal cancer
              </a>
            </td>
          <td>
            Zechen Chong, Fengyuan Huang, M. McLeod, Regina K. Irwin, Mary Smithson, Zongliang Yue, Min Gao, Karin M. Hardiman
          </td>
          <td>2024-07-01</td>
          <td>Research Square</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id=". The development of targeted therapies offers new hope for patients affected by incurable cancer. However, multiple challenges persist, notably in controlling tumor cell plasticity in patients with refractory and metastatic illness. Neuroblastoma (NB) is an aggressive pediatric malignancy originating from defective differentiation of neural crest– derived progenitors with oncogenic activity due to genetic and epigenetic alterations and remains a clinical challenge for high-risk patients. To identify critical genes driving NB aggressiveness, we performed combined chromatin and transcriptome analyses on matched patient-derived xenografts (PDXs), spheroids, and differentiated adherent cultures derived from metastatic MYCN nonamplified tumors. Bone marrow kinase on chromosome X (BMX) was identified among the most differentially regulated genes in PDXs and spheroids versus adherent models. BMX expression correlated with high tumor stage and poor patient survival and was crucial to the maintenance of the self-renewal and tumorigenic potential of NB spheroids. Moreover, BMX expression positively correlated with the mesenchymal NB cell phenotype, previously associated with increased chemoresistance. Finally, BMX inhibitors readily reversed this cellular state, increased the sensitivity of NB spheroids toward chemotherapy, and partially reduced tumor growth in a preclinical NB model. Altogether, our study identifies BMX as a promising innovative therapeutic target for patients with high-risk MYCN nonamplified NB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/720472ecd3c4d3cbeda17ff249c71decd3c1edd4" target='_blank'>
              Preclinical spheroid models identify BMX as a therapeutic target for metastatic MYCN nonamplified neuroblastoma
              </a>
            </td>
          <td>
            Santhoshkumar Sundaramoorthy, Daniele Filippo Colombo, Rajendran Sanalkumar, Liliane C. Broye, Katia Balmas Bourloud, Gaylor Boulay, L. Cironi, Ivan Stamenkovic, R. Renella, F. Kuttler, Gerardo Turcatti, Miguel N. Rivera, A. Mühlethaler-Mottet, Anaïs Flore Bardet, Nicolò Riggi
          </td>
          <td>2024-07-22</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The genetic integrity of pluripotent stem cells (PSC) is integral to their applications in research and therapy, but it is compromised by frequent development of copy number variations. Little is known about the basis of the variable genomic integrity among different PSC lines. Here we identify aneuploidies using RNA-seq and proteomics data from a panel of mouse embryonic stem cell (mESC) lines derived from 170 Diversity Outbred mice. We identified 62 lines with detectable aneuploid subpopulations and a subset of originally XX lines that lost one Chromosome X (XO). Strikingly, a much lower proportion of XX lines were aneuploid, compared to XY or XO lines. Two single-cell RNA-seq data sets demonstrated that aneuploid XY DO mESC also show lower Chromosome X gene expression and that XY mESC accumulate higher aneuploid proportions in culture than isogenic XX lines. Possible mechanisms for this protective effect of X chromosome dosage include our discovery that the lines with two active X Chromosomes have a higher expression of 2-cell-like state genes, and differential expression of X-linked tumor suppressor genes associated with DNA damage response. Highlights First genetic analysis of predisposition to aneuploidy in pluripotent stem cell cultures Chromosomal regions duplicated in aneuploid mouse embryonic stem cells are syntenic with regions overrepresented in human pluripotent stem cell lines bearing recurrent genetic abnormalities X-Chromosome dosage strongly influences susceptibility to aneuploidy in mouse embryonic stem cells and to a lesser degree in human pluripotent stem cells XX mouse embryonic stem cell lines show a higher proportion of cells in 2 cell-like state and higher expression of tumor suppressor genes associated with DNA damage response">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f853ae6e95a31cb9ad3712a21b0838a6601fe33" target='_blank'>
              Chromosome X Dosage Modulates Development of Aneuploidy in Genetically Diverse Mouse Embryonic Stem Cells
              </a>
            </td>
          <td>
            Alexander Stanton, Selcan Aydin, Daniel A. Skelly, Dylan Stavish, Kim Leonhard, Seth M. Taapken, Erik M. McIntire, Matthew Pankratz, A. Czechanski, T. Ludwig, Ted Choi, Steve P. Gygi, Ivana Barbaric, Steven C. Munger, L. Reinholdt, Martin F. Pera
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Using five complementary short- and long-read sequencing technologies, we phased and assembled >95% of each diploid human genome in a four-generation, 28-member family (CEPH 1463) allowing us to systematically assess de novo mutations (DNMs) and recombination. From this family, we estimate an average of 192 DNMs per generation, including 75.5 de novo single-nucleotide variants (SNVs), 7.4 non-tandem repeat indels, 79.6 de novo indels or structural variants (SVs) originating from tandem repeats, 7.7 centromeric de novo SVs and SNVs, and 12.4 de novo Y chromosome events per generation. STRs and VNTRs are the most mutable with 32 loci exhibiting recurrent mutation through the generations. We accurately assemble 288 centromeres and six Y chromosomes across the generations, documenting de novo SVs, and demonstrate that the DNM rate varies by an order of magnitude depending on repeat content, length, and sequence identity. We show a strong paternal bias (75-81%) for all forms of germline DNM, yet we estimate that 17% of de novo SNVs are postzygotic in origin with no paternal bias. We place all this variation in the context of a high-resolution recombination map (∼3.5 kbp breakpoint resolution). We observe a strong maternal recombination bias (1.36 maternal:paternal ratio) with a consistent reduction in the number of crossovers with increasing paternal (r=0.85) and maternal (r=0.65) age. However, we observe no correlation between meiotic crossover locations and de novo SVs, arguing against non-allelic homologous recombination as a predominant mechanism. The use of multiple orthogonal technologies, near-telomere-to-telomere phased genome assemblies, and a multi-generation family to assess transmission has created the most comprehensive, publicly available “truth set” of all classes of genomic variants. The resource can be used to test and benchmark new algorithms and technologies to understand the most fundamental processes underlying human genetic variation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bcb8df79fd9bfc629f05070b595811f041bfe33" target='_blank'>
              A familial, telomere-to-telomere reference for human de novo mutation and recombination from a four-generation pedigree
              </a>
            </td>
          <td>
            David Porubsky, Harriet Dashnow, T. Sasani, Glennis A. Logsdon, P. Hallast, Michelle D. Noyes, Z. Kronenberg, T. Mokveld, Nidhi Koundinya, Cillian Nolan, C. Steely, A. Guarracino, E. Dolzhenko, William T. Harvey, William J. Rowell, K. Grigorev, T. J. Nicholas, Keisuke K. Oshima, Jiadong Lin, Peter Ebert, W. Watkins, Tiffany Y. Leung, Vincent C. T. Hanlon, Sean Mcgee, Brent S. Pedersen, Michael E. Goldberg, Hannah C. Happ, Hyeonsoo Jeong, Katherine M. Munson, Kendra Hoekzema, Daniel D. Chan, Yanni Wang, Jordan Knuth, Gage H. Garcia, Cairbre Fanslow, Christine Lambert, Charles Lee, Josh Smith, Shawn Levy, C. Mason, Erik Garrison, Peter Lansdorp, D. Neklason, L. Jorde, Aaron R. Quinlan, M. Eberle, Evan E. Eichler
          </td>
          <td>2024-08-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e8868db406f4d297083b3336bdacf7e70f944e3" target='_blank'>
              Characterize molecular signatures and establish a prognostic signature of gastric cancer by integrating single-cell RNA sequencing and bulk RNA sequencing
              </a>
            </td>
          <td>
            Zhiwei Wang, Zhiyan Weng, Luping Lin, Xianyi Wu, Wenju Liu, Yong Zhuang, J. Jian, Changhua Zhuo
          </td>
          <td>2024-07-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c66955c4f03e0dcce369a0bced796c5c5114b438" target='_blank'>
              Transcriptional dysregulation and insights into clinical implications in melanoma
              </a>
            </td>
          <td>
            Chen Shen, Mengjiao Chen, Xuanbingning Nian, Jun Li, Juan Tao
          </td>
          <td>2024-06-24</td>
          <td>Holistic Integrative Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="PURPOSE
Sarcoma encompasses a diverse group of cancers that are typically resistant to current therapies, including immune checkpoint blockade (ICB), and underlying mechanisms are poorly understood. The contexture of sarcomas limits generation of high-quality data using cutting-edge molecular profiling methods, such as single-cell RNA-seq, thus hampering progress in understanding these understudied cancers.


EXPERIMENTAL DESIGN
Here, we demonstrate feasibility of producing multi-modal single-cell genomics and whole-genome sequencing data from frozen tissues, profiling 75,716 cell transcriptomes of five undifferentiated pleomorphic (UPS) and three intimal sarcomas (INS) samples, including paired specimens from two patients treated with ICB.


RESULTS
We find that genomic diversity decreases in patients with response to ICB, and, in unbiased analyses, identify cancer cell programs associated with therapy resistance. Although interactions of tumor-infiltrating T lymphocytes within the tumor ecosystem increase in ICB responders, clonal expansion of CD8+ T cells alone was insufficient to predict drug responses.


CONCLUSION
This study provides a framework for studying rare tumors and identifies salient and treatment-associated cancer cell intrinsic and tumor-microenvironmental features in sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece2adecc484d75c5dbd396e58a992a80003e58" target='_blank'>
              Single-cell profiling of sarcomas from archival tissue reveals programs associated with resistance to immune checkpoint blockade.
              </a>
            </td>
          <td>
            Karan Luthria, Parin Shah, Blake Caldwell, Johannes C. Melms, Sinan Abuzaid, Viktoria Jakubikova, D. Z. Brodtman, S. Bose, A. D. Amin, P. Ho, Jana Biermann, Somnath Tagore, M. Ingham, Gary K Schwartz, Benjamin Izar
          </td>
          <td>2024-07-31</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a critical driver of tumor development and is linked to drug resistance and metastasis in solid malignancies. Here, we demonstrate that WGD is an ongoing mutational process in tumor evolution. Using single-cell whole-genome sequencing, we measured and modeled how WGD events are distributed across cellular populations within tumors and associated WGD dynamics with properties of genome diversification and phenotypic consequences of innate immunity. We studied WGD evolution in 65 high-grade serous ovarian cancer (HGSOC) tissue samples from 40 patients, yielding 29,481 tumor cell genomes. We found near-ubiquitous evidence of WGD as an ongoing mutational process promoting cell-cell diversity, high rates of chromosomal missegregation, and consequent micronucleation. Using a novel mutation-based WGD timing method, doubleTime, we delineated specific modes by which WGD can drive tumor evolution: (i) unitary evolutionary origin followed by significant diversification, (ii) independent WGD events on a pre-existing background of copy number diversity, and (iii) evolutionarily late clonal expansions of WGD populations. Additionally, through integrated single-cell RNA sequencing and high-resolution immunofluorescence microscopy, we found that inflammatory signaling and cGAS-STING pathway activation result from ongoing chromosomal instability and are restricted to tumors that remain predominantly diploid. This contrasted with predominantly WGD tumors, which exhibited significant quiescent and immunosuppressive phenotypic states. Together, these findings establish WGD as an evolutionarily ‘active’ mutational process that promotes evolvability and dysregulated immunity in late stage ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7492c9bf200e273b7b6b57e3677bc99bc004737e" target='_blank'>
              Ongoing genome doubling promotes evolvability and immune dysregulation in ovarian cancer
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Matthew Zatzman, Duaa H Al-Rawi, A. Weiner, Samuel S. Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nick Ceglia, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, G. Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, Rachel Grisham, Ying L. Liu, A. Viale, Nicole Rusk, Y. Lakhman, Lora Hedrick Ellenson, Simon Tavaré, Samuel Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2024-07-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/699986230662d4afba97aea6b14e6128a02adbf7" target='_blank'>
              Molecular mechanisms of DNA lesion and repair during antibody somatic hypermutation.
              </a>
            </td>
          <td>
            Qian Hao, Jinfeng Li, Leng-Siew Yeap
          </td>
          <td>2024-07-23</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Introduction Glioblastomas utilize malignant gene expression pathways to drive growth. Many of these gene pathways are not directly accessible with molecularly targeted pharmacological agents. Chromatin-modifying compounds can alter gene expression and target glioblastoma growth pathways. In this study, we utilize a systematic screen of chromatin-modifying compounds on a panel of patient-derived glioblastoma lines to identify promising compounds and their associated gene targets. Methods Five glioblastoma cell lines were subjected to a drug screen of 106 chromatin-modifying compounds representing 36 unique drug classes to determine the twelve most promising drug classes and the best candidate inhibitors in each class. These twelve drugs were then tested with a panel of twelve patient-derived gliomasphere lines to identify growth inhibition and corresponding gene expression patterns. Overlap analysis and weighted co-expression network analysis (WCGNA) were utilized to determine potential target genes and gene pathways. Results The initial drug screen identified twelve candidate pharmacologic agents for further testing. Drug sensitivity testing indicated an overall high degree of variability between gliomasphere lines. However, CPI203 was the most consistently effective compound, and the BET inhibitor class was the most consistently effective class of compounds across the gliomasphere panel. Correspondingly, most of the compounds tested had highly variable effects on gene expression between gliomasphere lines. CPI203 stood out as the only compound to induce a consistent effect on gene expression across different gliomasphere lines, specifically down-regulation of DNA-synthesis genes. Amongst the twelve tested cell lines, high expression of CDKN2A and CDKN2B distinguished more drug sensitive from more drug resistant lines. WCGNA identified two oncogenic gene modules (FBXO5 and MELK) that were effectively downregulated by CPI203 (FBXO5) and ML228 (FBXO5 and MELK). Conclusions The bromodomain inhibitor CPI203 induced relatively consistent effects on gene expression and growth across a variety of glioblastoma lines, specifically down-regulating genes associated with DNA replication. We propose that clinically effective BET inhibitors have the potential to induce consistent beneficial effects across a spectrum of glioblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1561fa27def3a17181508712df5ccd39c21a9f63" target='_blank'>
              CPI203, a BET inhibitor, down-regulates a consistent set of DNA synthesis genes across a wide array of glioblastoma lines
              </a>
            </td>
          <td>
            Matthew C. Garrett, Troy P Carnwath, Rebecca R Albano, Yonghua Zhuang, Catherine A Behrmann, Merissa Pemberton, Farah Barakat, Robert Lober, Mark Hoeprich, Anthony Paravati, Marilyn Reed, Hailey Spry, Daniel Woo, Eric O’Brien, Brett VanCauwenbergh, John Perentesis, Rani Nasser, Mario Medvedovic, D. Plas
          </td>
          <td>2024-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Transcription and splicing errors in cancer cells generate erroneous transcripts. Since erroneous transcripts are degraded by the nonsense-mediated mRNA decay (NMD) pathway, whether they are junk or could be functional has been overlooked and understudied. We addressed this question by first performing a pan-cancer analysis and identified substantial erroneous noncoding transcripts (ENT) in cancers. Given that RNA/DNA binding domains (DBD) were predicted in ENTs, we deleted predicted DBDs in multiple ENTs in multiple cell lines, with RNA-sequencing and cell experiments before and after DBD deletion. DBD deletion caused significantly changed expression of ENTs’ target genes (whose promoter regions contain ENTs’ DNA binding sites, DBS) and changed cell migration and proliferation ability, indicating that many ENTs can transcriptionally regulate genes. Tightly coupled data analysis and experiments reveal that ENTs’ functions are highly cancer- and cellular-context specific, making ENTs a new class of safe and specific targets for noncoding RNA-based cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2e45af8d1c765a560ff555893aa2ed200e47f3c" target='_blank'>
              Many erroneous noncoding transcripts in cancer cells can highly specifically regulate cancer-related genes and pathways
              </a>
            </td>
          <td>
            Sha He, Wei Xiong, Jianping Huo, Jie Lin, Jianmin Li, Hao Zhu
          </td>
          <td>2024-07-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GBM) is a highly aggressive tumor with unmet therapeutic needs, which can be explained by extensive intra-tumoral heterogeneity and plasticity. In this study, we aimed to investigate the specific metabolic features of Glioblastoma stem cells (GSC), a rare tumor subpopulation involved in tumor growth and therapy resistance.


METHODS
We conducted comprehensive analyses of primary patient-derived GBM cultures and GSC-enriched cultures of human GBM cell lines using state-of-the-art molecular, metabolic and phenotypic studies.


RESULTS
We showed that GSC-enriched cultures display distinct glycolytic profiles compared with differentiated tumor cells. Further analysis revealed that GSC relies on pyruvate carboxylase activity for survival and self-renewal capacity. Interestingly, inhibition of pyruvate carboxylase led to GSC death, particularly when the glutamine pool was low, and increased differentiation. Finally, while GSC displayed resistance to the chemotherapy drug etoposide, genetic or pharmacological inhibition of pyruvate carboxylase restored etoposide sensitivity in GSC, both in vitro and in orthotopic murine models.


CONCLUSION
Our findings demonstrate the critical role of pyruvate carboxylase in GSC metabolism, survival and escape to etoposide. They also highlight pyruvate carboxylase as a therapeutic target to overcome therapy resistance in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd87fc69cc39f932ec1b38c0feef0134502734fd" target='_blank'>
              Metabolic Profiling of Glioblastoma Stem Cells Reveals Pyruvate Carboxylase as a Critical Survival Factor and Potential Therapeutic Target.
              </a>
            </td>
          <td>
            Ophélie Renoult, Mélanie Laurent-Blond, Hala Awada, Lisa Oliver, Noémie Joalland, Mikaël Croyal, François Paris, Catherine Gratas, C. Pecqueur
          </td>
          <td>2024-06-13</td>
          <td>Neuro-oncology</td>
          <td>1</td>
          <td>32</td>
        </tr>

        <tr id="Breast cancer (BCa) is a major global health challenge, characterized by chromosomal instability (CIN) and subsequent acquisition of extensive somatic copy number alterations (CNAs). CNAs including amplifications and deletions, significantly influence intra-tumor heterogeneity and the tumor microenvironment (TME). Among these, the loss of chromosome 13q14.2 emerges as a considerable factor in BCa pathogenesis and treatment responses. We provide evidence that this genomic alteration is under positive selective pressure, correlating with poorer patient survival. Furthermore, through multi-omic and in vitro analyses, we uncover a dual role of 13q14.2 loss: it confers a survival advantage to tumor cells and modulate the cell cycle and pro-apoptotic pathways in cancer cells, affecting macrophages population in the TME, while paradoxically increasing tumor susceptibility to BCL2 inhibitors. These findings suggest that targeting 13q14.2 as a biomarker in BCa could enhance the efficacy of existing treatments and offer a new avenue for improving clinical outcomes in BCa. Figure 1. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e388fe8f21125bbefce3cee2961f295d519ae77" target='_blank'>
              Loss of chromosome cytoband 13q14.2 orchestrates breast cancer pathogenesis and drug response
              </a>
            </td>
          <td>
            Parastoo Shahrouzi, Youness Azimzade, Wioletta Brankiewicz, Sugandha Bhatia, David Kunke, Derek Richard, Xavier Tekpli, Vessela N Kristensen, Pascal H.G. Duijf
          </td>
          <td>2024-06-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Drug resistance is still an imminent issue for melanoma patients even after the use of BRAFi/MEKi combination therapy in clinic. Tumor heterogeneity and heterogeneity in treatment responses make it difficult to find consensus genes and pathways in resistance to therapy. This study used an objective method to analyze published gene expression data from pre-treated tumors and drug-resistant tumors, and identified possible targets and markers for resistant tumors, which centered on the PLXNC1 gene, which promotes a pro-inflammatory tumor microenvironment. Abstract Melanoma tumors exhibit a wide range of heterogeneity in genomics even with shared mutations in the MAPK pathway, including BRAF mutations. Consistently, adaptive drug resistance to BRAF inhibitors and/or BRAF plus MEK inhibitors also exhibits a wide range of heterogeneous responses, which poses an obstacle for discovering common genes and pathways that can be used in clinic for overcoming drug resistance. This study objectively analyzed two sets of previously published tumor genomics data comparing pre-treated melanoma tumors and BRAFi- and/or MEKi-resistant tumors. Heterogeneity in response to BRAFi and BRAFi/MEKi was evident because the pre-treated tumors and resistant tumors did not exhibit a tendency of clustering together. Differentially expressed gene (DEG) analysis revealed eight genes and two related enriched signature gene sets (matrisome and matrisome-associated signature gene sets) shared by both sets of data. The matrisome was closely related to the tumor microenvironment and immune response, and five out of the eight shared genes were also related to immune response. The PLXNC1 gene links the shared gene set and the enriched signature gene sets as it presented in all analysis results. As the PLXNC1 gene was up-regulated in the resistant tumors, we validated the up-regulation of this gene in a laboratory using vemurafenib-resistant cell lines. Given its role in promoting inflammation, this study suggests that resistant tumors exhibit an inflammatory tumor microenvironment. The involvement of the matrisome and the specific set of immune genes identified in this study may provide new opportunities for developing future therapeutic methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fabe8a302d27af7d60acbb39b872cfcbb898453" target='_blank'>
              Unsupervised Analysis Reveals the Involvement of Key Immune Response Genes and the Matrisome in Resistance to BRAF and MEK Inhibitors in Melanoma
              </a>
            </td>
          <td>
            F. Liu-Smith, Jianjian Lin
          </td>
          <td>2024-06-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca87f16129e49df59c97d9b9e83e846766aff915" target='_blank'>
              iPS cell generation-associated point mutations include many C > T substitutions via different cytosine modification mechanisms
              </a>
            </td>
          <td>
            R. Araki, T. Suga, Y. Hoki, Kaori Imadome, Misato Sunayama, Satoshi Kamimura, Mayumi Fujita, M. Abe
          </td>
          <td>2024-06-11</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5073eeabc50bea9fdc09e2d2eeb42d91221fda8" target='_blank'>
              Integrative genomics identifies SHPRH as a tumor suppressor gene in lung adenocarcinoma that regulates DNA damage response
              </a>
            </td>
          <td>
            A. Nagelberg, T. Sihota, Yu-Chi Chuang, Rocky Shi, Justine L M Chow, John English, C. MacAulay, Stephen Lam, Wan L Lam, William W Lockwood
          </td>
          <td>2024-06-18</td>
          <td>British Journal of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e606cab213cb81e6c3f7ec232b67c83ab5b219c" target='_blank'>
              Unveiling novel genetic variants in 370 challenging medically relevant genes using the long read sequencing data of 41 samples from 19 global populations.
              </a>
            </td>
          <td>
            Yanfeng Ji, Junfan Zhao, J. Gong, F. Sedlazeck, Shaohua Fan
          </td>
          <td>2024-07-07</td>
          <td>Molecular genetics and genomics : MGG</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/135134a62bac76cb46dae2f04090c849f2c01172" target='_blank'>
              Targeting chromosomal instability in patients with cancer.
              </a>
            </td>
          <td>
            Duaa H Al-Rawi, Emanuele Lettera, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-11</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>1</td>
          <td>35</td>
        </tr>

        <tr id="Venetoclax is the first example of personalized medicine for multiple myeloma (MM), with meaningful clinical activity as a monotherapy and in combination in myeloma patients harboring the t(11:14) translocation. However, despite the high response rates and prolonged PFS, a significant proportion of patients eventually relapse. Here, we aimed to study adaptive molecular responses after the acquisition of venetoclax resistance in sensitive t(11:14) MM cell models. We therefore generated single-cell venetoclax-resistant t(11:14) MM cell lines and investigated the mechanisms contributing to resistance as well as the cells' sensitivity to other treatments. Our data suggests that acquired resistance to venetoclax is characterized by reduced mitochondrial priming and changes in BCL-2 family proteins' expression in MM cells, conferring broad resistance to standard-of-care anti-myeloma drugs. However, our results show that the resistant cells are still sensitive to immunotherapeutic treatments, highlighting the need to consider appropriate sequencing of these treatments following venetoclax-based regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc29857b46abaf1a033e0c4b6c0e2ae376a185cb" target='_blank'>
              Venetoclax resistance leads to broad resistance to standard-of-care anti-MM agents, but not to immunotherapies.
              </a>
            </td>
          <td>
            Shuhui Deng, Sanika Derebail, Vera Joy Weiler, J. Fong Ng, Elena Maroto-Martin, Madhumouli Chatterjee, Giulia Giorgetti, C. Chakraborty, Poonam Kalhotra, Ting Du, Yao Yao, R. Prabhala, M. Shammas, Annamaria Gullà, Anil Aktas Samur, M. Samur, Lugui Qiu, Kenneth C. Anderson, M. Fulciniti, Nikhil C Munshi
          </td>
          <td>2024-06-11</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="CRISPR gene editing strategies are shaping cell therapies through precise and tunable control over gene expression. However, achieving reliable therapeutic effects with improved safety and efficacy requires informed target gene selection. This depends on a thorough understanding of the involvement of target genes in gene regulatory networks (GRNs) that regulate cell phenotype and function. Machine learning models have been previously used for GRN reconstruction using RNA- seq data, but current techniques are limited to single cell types and focus mainly on transcription factors. This restriction overlooks many potential CRISPR target genes, such as those encoding extracellular matrix components, growth factors, and signaling molecules, thus limiting the applicability of these models for CRISPR strategies. To address these limitations, we have developed CRISPR-GEM, a multi-layer perceptron (MLP)-based synthetic GRN constructed to accurately predict the downstream effects of CRISPR gene editing. First, input and output nodes are identified as differentially expressed genes between defined experimental and target cell/tissue types respectively. Then, MLP training learns regulatory relationships in a black-box approach allowing accurate prediction of output gene expression using only input gene expression. Finally, CRISPR-mimetic perturbations are made to each input gene individually and the resulting model predictions are compared to those for the target group to score and assess each input gene as a CRISPR candidate. The top scoring genes provided by CRISPR-GEM therefore best modulate experimental group GRNs to motivate transcriptomic shifts towards a target group phenotype. This machine learning model is the first of its kind for predicting optimal CRISPR target genes and serves as a powerful tool for enhanced CRISPR strategies across a range of cell therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/486423f8c90770082bb7f4dc4e4e7334da919173" target='_blank'>
              CRISPR-GEM: A Novel Machine Learning Model for CRISPR Genetic Target Discovery and Evaluation
              </a>
            </td>
          <td>
            Josh P. Graham, Yu Zhang, Lifang He, Tomas Gonzalez-Fernandez
          </td>
          <td>2024-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is a genetically and clinically diverse hematological cancer affecting middle-aged and elderly individuals. Novel targeted therapy options are needed for patients who relapse following initial responses or who are intrinsically resistant to current treatments. There is a growing body of investigation currently underway on MDM2 inhibitors in clinical trials, reflecting the increasing interest in including these drugs in cancer treatment regimens. One of the developed compounds, idasanutlin (RG7388), has shown promise in early-stage clinical trials. It is a second-generation MDM2–p53-binding antagonist with enhanced potency, selectivity, and bioavailability. In addition to the TP53 status, which is an important determinant of the response, we have shown in our previous studies that the SF3B1 mutational status is also an independent predictive biomarker of the ex vivo CLL patient sample treatment response to RG7388. The objective of this study was to identify novel biomarkers associated with resistance to RG7388. Gene set enrichment analysis of differentially expressed genes (DEGs) between RG7388-sensitive and -resistant CLL samples showed that the increased p53 activity led to upregulation of pro-apoptosis pathway genes while DNA damage response pathway genes were additionally upregulated in resistant samples. Furthermore, differential expression of certain genes was detected, which could serve as the backbone for novel combination treatment approaches. This research provides preclinical data to guide the exploration of drug combination strategies with MDM2 inhibitors, leading to future clinical trials and associated biomarkers that may improve outcomes for CLL patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4e336986b17d0b66d36dfa25a57ef235e819da3" target='_blank'>
              RNA Sequencing Reveals Candidate Genes and Pathways Associated with Resistance to MDM2 Antagonist Idasanutlin in TP53 Wild-Type Chronic Lymphocytic Leukemia
              </a>
            </td>
          <td>
            E. Aptullahoglu, S. Nakjang, J. Wallis, Helen Marr, Scott Marshall, E. Willmore, John Lunec
          </td>
          <td>2024-06-22</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Single nucleotide variants (SNVs) contribute to cancer by altering the coding and the non-coding regions of the genome. Connecting SNVs to transcriptomic and epigenetic changes at the single-cell level remains challenging. To enable studies of rare cell populations harboring specific point mutations, we developed STAR-FACS, Specific-To-Allele PCR-FACS, to sort cells based on genomic allele alterations. We show that STAR-FACS can separate cells based on TERT promoter mutation status and is compatible with bulk and single-cell transcriptomic and epigenetic profiling. We demonstrate that glioblastoma cell lines derived from the same tumor but harboring distinct TERT promoter SNVs have different transcriptional programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f2ef09f49e055964d96a3768d156e4cf968931" target='_blank'>
              Cell sorting based on single nucleotide variation enables characterization of mutation-dependent transcriptome and chromatin states
              </a>
            </td>
          <td>
            Roberto Salatino, Marianna Franco, Arantxa Romero-Toledo, Yi Wang, Shanel Tsuda, Oszkar Szentirmai, M. Janiszewska
          </td>
          <td>2024-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The burden of cancer is on a rapid rise globally. Deregulated gene expression profiles may lead to the development of cancer. Master regulators are the regulatory checkpoints that define and control the intricate networks of gene and protein interactions that make up cell physiology. The molecular programs that generate particular cellular phenotypes depend on master regulators. SOX4 gene is a master regulator that controls the expression of other genes responsible for tumorigenesis and plays a crucial role in multiple signaling pathways. The expression of SOX4 is upregulated in various malignancies. Increased proliferation of cancer cells, survival, apoptosis, and epithelial-to-mesenchymal transition leading to metastasis have all been linked to SOX4 expression in cancer. Elevated levels of SOX4 also possess a correlation with poor prognosis in various cancer types. Recently, SOX4 has surfaced as a possible target for cancer therapeutics. Furthermore, it has been shown that targeting SOX4 could inhibit tumor growth and enhance the efficacy of conventional cancer therapies. The present review summarizes the current status of SOX4 in the initiation and progression of various human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd13b5840ecb03dba39c075ed71c34056378ac" target='_blank'>
              SOX4 expression in cancer: Insights from developmental regulation and deregulation in tumorigenesis
              </a>
            </td>
          <td>
            Surbhi Arora, Meena Godha, N. K. Lohiya, Nirmala Jagadish
          </td>
          <td>2024-07-29</td>
          <td>Journal of Reproductive Healthcare and Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="BACKGROUND
Metastatic colorectal cancer (mCRC) is increasingly characterized by myriad genomic alterations beyond the well-known factors such as RAS, BRAF, and microsatellite instability (MSI). Novel genomic changes, including ERBB2 amplifications, mutations, and gene fusions, are now recognized as potential targets for precision therapy. This study aims to explore the genomic landscape of a Chinese cohort with mCRC to identify potentially targetable genetic alterations for personalized treatment strategies.


METHODS
A total of 500 mCRC patients in China were enrolled, based on which genomic profiling was performed using capture-based targeted sequencing across a panel of 520 genes on tumor tissues to identify prevalent genomic alterations. The mutations were analyzed by optimized proprietary algorithms. MSI and mismatch repair deficiency status were analyzed using the read-count-distribution approach. Besides, the overall survival (OS) related to these molecular changes was estimated.


RESULTS
The cohort's genomic profiling revealed TP53 mutations in 78%, APC in 60%, and KRAS in 47% of the patients. MSI-High status was confirmed in 5.8% of cases via a next-generation sequencing (NGS)-based algorithm. ERBB2/HER2 amplifications were found in 12% (60/500) of patients, with potential therapeutic implications for those without concurrent KRAS mutations. A subset of patients (1.2%; 6/500) showed fusions and DNA damage response (DDR) gene mutations (except TP53) that could be targeted therapeutically. The KRAS (G12C) variant was detected in 14 patients (2.8%), and 61 (12.2%) had a BRAF V600E mutation. Notably, survival analysis showed no significant differences in OS between KRAS mutant loci and NRAS mutations (p = 0.436). However, BRAF V600E mutations were associated with a poorer prognosis than BRAF wild-type and non-V600E mutations (16.3 months vs. 29.5 and 31.1 months, respectively; p < 0.001).


CONCLUSIONS
This study validates the feasibility of using NGS to detect prognostic and therapeutically actionable genetic variants in Chinese mCRC patients, contributing to understanding the genomic variation within this population and highlighting the potential for personalized medicine in managing mCRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d75b3be37f6d3fb50ba2d4fcbf63055e7d0a5a8b" target='_blank'>
              Genomic Alterations and Clinical Characterization in Chinese Patients with Metastatic Colorectal Cancer.
              </a>
            </td>
          <td>
            Xu-hui Zhang, Jie-Qiong Zhou, Qing Wei, Jian Li, Ting Xu, Chun-mei Bai, Jian-Feng Zhou, Xi-cheng Wang
          </td>
          <td>2024-07-01</td>
          <td>Discovery medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Background. Identifying which patients will benefit from individual therapies remains a general challenge in oncology. Predictive biomarkers of drug sensitivity, when known, can increase the rate of success of trials, and allow stratification of patients for targeted combination therapies, as recent shown by the GUIDANCE-01 trial in Large B-Cell Lymphoma. Peripheral T-Cell Lymphomas (PTCL) could particularly benefit from this approach, being heterogeneous diseases against which many therapies are active in unpredictable subsets of patients. A central challenge of biomarker discovery is finding true determinants of drug response, which may be multiple genes, among thousands of molecular features that may accidentally correlate. Here, we aim to mitigate this challenge by combining functional genetic screens and gene expression profiling in diverse PTCL cultures, to enable the systematic discovery of drug sensitivity biomarkers.
 Methods. To discover genes whose expression directly modifies drug sensitivity, we conducted genome-wide CRISPR screens with PTCL cells under treatment with five therapies with clinical activity in PTCL: Romidepsin, Pralatrexate, Gemcitabine, Oxaliplatin, and Bortezomib. To discover genes whose basal expression significantly correlates with specific drug sensitivities, we measured RNAseq and drug response functions for a diverse panel of 27 PTCL cultures. Genes in the intersection of both experiments are causally related to and correlate with drug responses in PTCL, and have the statistical benefit of detection by independent methods.
 Results. Functional genetic screens on therapy provided mechanistic insights for each drug of interest, and identified genes whose perturbation affects drug sensitivity. Whereas hundreds of genes nominally correlate with drug sensitivity in cell line panels, for each drug a shortlist of only 5 to 20 genes are both CRISPR hits and correlate with response across PTCL cultures. For each drug tested, multi-gene sensitivity scores constructed from hits stratify PTCL cultures into statistically distinct sensitive and non-sensitive subgroups (Mann Whitney P-Values <0.01). These genes’ relationships to drug mechanisms were often supported by prior studies, including functions in DNA repair, cell cycle regulation, and solute carriers required for cellular entry of specific therapies, such as SLC19A1 for Pralatrexate and SLC43A2 for Oxaliplatin.
 Conclusions. Combining functional genetic screens and correlative drug sensitivity measurements narrowed the search for predictive biomarkers of drug sensitivity, and may be a general means of refining biomarker hypotheses for validation in clinical datasets. Although chemotherapy sensitivity is commonly polygenic, multi-gene scores based on few mechanistically-related genes were able to significantly distinguish drug sensitive from resistant PTCLs. These approaches provide novel hypotheses for drug-gene relationships that may serve as the basis for molecular subtyping of individual lymphomas for precision regimens.
 Citation Format: Adam C Palmer, Jacob C Pantazis. Systematic discovery of chemotherapy response biomarkers in Peripheral T Cell Lymphomas [abstract]. In: Proceedings of the Fourth AACR International Meeting on Advances in Malignant Lymphoma: Maximizing the Basic-Translational Interface for Clinical Application; 2024 Jun 19-22; Philadelphia, PA. Philadelphia (PA): AACR; Blood Cancer Discov 2024;5(3_Suppl):Abstract nr PO-031.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d48e5ac52801f4dbaf2b3115a503990ac42f1d08" target='_blank'>
              Abstract PO-031: Systematic discovery of chemotherapy response biomarkers in Peripheral T Cell Lymphomas
              </a>
            </td>
          <td>
            Adam C Palmer, Jacob C Pantazis
          </td>
          <td>2024-06-19</td>
          <td>Blood Cancer Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="A significant limitation of numerous current genetic engineering therapy approaches is their limited control over the strength, timing, or cellular context of their therapeutic effect. Synthetic gene/genetic circuits are synthetic biology approaches that can control the generation, transformation, or depletion of a specific DNA, RNA, or protein and provide precise control over gene expression and cellular behavior. They can be designed to perform logical operations by carefully selecting promoters, repressors, and other genetic components. Patent search was performed in Espacenet, resulting in 38 selected patents with 15 most frequent international classifications. Patent embodiments were categorized into applications for the delivery of therapeutic molecules, treatment of infectious diseases, treatment of cancer, treatment of bleeding, and treatment of metabolic disorders. The logic gates of selected genetic circuits are described to comprehensively demonstrate their therapeutic applications. Synthetic gene circuits can be customized for precise control of therapeutic interventions, leading to personalized therapies that respond specifically to individual patient needs, enhancing treatment efficacy and minimizing side effects. They can be highly sensitive biosensors that provide real-time therapy by accurate monitoring various biomarkers or pathogens and appropriately synthesizing a therapeutic molecule. Synthetic gene circuits may also lead to the development of advanced regenerative therapies and to implantable biodevices that produce on-demand bioactive molecules. However, this technology faces challenges for commercial profitability. The genetic circuit designs need adjustments for specific applications, and may have disadvantages like toxicity from multiple regulators, homologous recombination, context dependency, resource overuse, and environmental variability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a28758d2e1abda7760e2ebff6edf14572cb9b5f" target='_blank'>
              Therapeutic applications of synthetic gene/genetic circuits: a patent review
              </a>
            </td>
          <td>
            Diego C. Carneiro, Vinícius P. C. Rocha, Patrícia K. F. Damasceno, Josiane D. V. Barbosa, Milena B. P. Soares
          </td>
          <td>2024-08-05</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer develops from abnormal cell growth in the body, causing significant mortalities every year. To date, potent therapeutic approaches have been developed to eradicate tumor cells, but intolerable toxicity and drug resistance can occur in treated patients, limiting the efficiency of existing treatment strategies. Therefore, searching for novel genes critical for cancer progression and therapeutic response is urgently needed for successful cancer therapy. Recent advances in bioinformatics and proteomic techniques have allowed the identification of a novel category of peptides encoded by non-canonical open reading frames (ncORFs) from historically non-coding genomic regions. Surprisingly, many ncORFs express functional microproteins that play a vital role in human cancers. In this review, we provide a comprehensive description of different ncORF types with coding capacity and technological methods in discovering ncORFs among human genomes. We also summarize the carcinogenic role of ncORFs such as pTINCR and HOXB-AS3 in regulating hallmarks of cancer, as well as the roles of ncORFs such as HOXB-AS3 and CIP2A-BP in cancer diagnosis and prognosis. We also discuss how ncORFs such as AKT-174aa and DDUP are involved in anti-cancer drug response and the underestimated potential of ncORFs as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d95f26adc077c4e04bfda2c2e30184b8f21906d" target='_blank'>
              Exploring the Dark Matter of Human Proteome: The Emerging Role of Non-Canonical Open Reading Frame (ncORF) in Cancer Diagnosis, Biology, and Therapy
              </a>
            </td>
          <td>
            Anni Ge, Curtis Chan, Xiaolong Yang
          </td>
          <td>2024-07-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cell-cell interactions, which allow cells to communicate with each other through molecules in their microenvironment, are critical for the growth, health, and functions of cells. Previous studies show that drug-resistant cells can interact with drug-sensitive cells to elevate their drug resistance level, which is partially responsible for cancer recurrence. Studying protein targets and pathways involved in cell-cell communication provides essential information for fundamental cell biology studies and therapeutics of human diseases. In the current studies, we performed direct coculture and indirect coculture of drug-resistant and drug-sensitive cell lines, aiming to investigate intracellular proteins responsible for cell communication. Comparative studies were carried out using monoculture cells. Shotgun bottom-up proteomics results indicate that the P53 signaling pathway has a strong association with drug resistance mechanisms, and multiple TP53-related proteins were upregulated in both direct and indirect coculture systems. In addition, cell-cell communication pathways, including the phagosome and the HIF-signaling pathway, contribute to both direct and indirect coculture systems. Consequently, AK3 and H3-3A proteins were identified as potential targets for cell-cell interactions that are relevant to drug resistance mechanisms. We propose that the P53 signaling pathway, in which mitochondrial proteins play an important role, is responsible for inducing drug resistance through communication between drug-resistant and drug-sensitive cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cc9726c157261dbc51a02e3ad073aa8b2f6d16b" target='_blank'>
              Proteomics Analysis of Interactions between Drug-Resistant and Drug-Sensitive Cancer Cells: Comparative Studies of Monoculture and Coculture Cell Systems.
              </a>
            </td>
          <td>
            Zongkai Peng, Nagib Ahsan, Zhibo Yang
          </td>
          <td>2024-06-22</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract DNA replication-repair deficiency (RRD) arises from pathogenic variants in the mismatch repair and/or polymerase-proofreading genes. Multiple germline cancer predisposition syndromes in children and young adults, including constitutional mismatch repair deficiency (CMMRD), Lynch, polymerase-proofreading deficiency, and rare digenic syndromes can lead to RRD cancers. The most frequent brain tumors in these children are high-grade gliomas. Embryonal tumors like medulloblastoma have also been described. Lower-grade tumors are reported from cancer surveillance initiatives. The latter has an extremely high rate of malignant transformation. Novel functional assays quantifying the genomic microsatellite indel load have been demonstrated to be highly sensitive and specific for the diagnosis of RRD cancers and children with germline CMMRD. Importantly, RRD brain tumors uniformly harbor high mutation and microsatellite burden. High T-cell infiltration makes these aggressive cancers amenable to immune checkpoint inhibition, irrespective of their germline genetic background. Synergistic combinations are reported to be successful in patients failing checkpoint inhibitor monotherapy. Future directions include the development of innovative approaches to improve immune surveillance for RRD brain cancers. Additionally, the use of novel tools including circulating tumor DNA and quantifying microsatellite indel load over time can be useful to monitor disease burden and treatment responses in patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1179f2665b8f940bf462f30557e4e5bb2bfd1210" target='_blank'>
              An update on central nervous system tumors in germline replication-repair deficiency syndromes
              </a>
            </td>
          <td>
            Anirban Das, A. B. Ercan, U. Tabori
          </td>
          <td>2024-06-19</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc88953f3695ac0c90c5077d3442247bd93ef70" target='_blank'>
              Mixed responses to targeted therapy driven by chromosomal instability through p53 dysfunction and genome doubling
              </a>
            </td>
          <td>
            S. Hobor, M. Al Bakir, C. Hiley, Marcin Skrzypski, A. Frankell, Bjorn Bakker, T. Watkins, A. Markovets, J. Dry, Andrew P. Brown, Jasper van der Aart, H. van den Bos, D. Spierings, D. Oukrif, Marco Novelli, T. Chakrabarti, Adam H. Rabinowitz, Laila Ait Hassou, Saskia Litière, D. Kerr, L. Tan, Gavin P. Kelly, David A Moore, M. Renshaw, Subramanian Venkatesan, William Hill, A. Huebner, C. Martínez-Ruiz, James R. M. Black, Wei Wu, Mihaela Angelova, N. Mcgranahan, Julian Downward, J. Chmielecki, Carl Barrett, K. Litchfield, S. Chew, Collin M Blakely, Elza C de Bruin, F. Foijer, Karen H. Vousden, T. Bivona, , Jason Lester, A. Bajaj, A. Nakas, Azmina Sodha-Ramdeen, M. Tufail, M. Scotland, R. Boyles, S. Rathinam, C. Wilson, Domenic Marrone, S. Dulloo, D. Fennell, Gurdeep Matharu, J. Shaw, E. Boleti, Heather Cheyne, Mohammed Khalil, S. Richardson, Tracey Cruickshank, Gillian Price, K. M. Kerr, Sarah Benafif, Jack French, Kayleigh Gilbert, B. Naidu, Akshay J. Patel, A. Osman, Carol Enstone, G. Langman, Helen Shackleford, M. Djearaman, S. Kadiri, Gary Middleton, Angela E. Leek, Jack Davies Hodgkinson, Nicola Totton, A. Montero, E. Smith, E. Fontaine, F. Granato, A. Paiva-Correia, Juliette Novasio, K. Rammohan, L. Joseph, P. Bishop, Rajesh Shah, Stuart Moss, Vijay Joshi, Philip A J Crosbie, Katherine D. Brown, Mathew Carter, Anshuman Chaturvedi, Pedro Oliveira, Colin R. Lindsay, Fiona H. Blackhall, Matthew G. Krebs, Yvonne Summers, A. Clipson, J. Tugwood, Alastair Kerr, D. Rothwell, C. Dive, H. Aerts, R. Schwarz, Tom L. Kaufmann, G. Wilson, R. Rosenthal, P. Van Loo, Nicolai J. Birkbak, Zoltán Szállási, J. Kisistók, M. Sokač, R. Salgado, M. Dióssy, J. Demeulemeester, Abigail Bunkum, Angela Dwornik, Alastair Magness, Andrew J. Rowan, Angeliki Karamani, A. Toncheva, B. Chain, C. Castignani, Chris Bailey, C. Abbosh, C. Puttick, C. Weeden, Claudia Lee, Corentin Richard, C. Naceur-Lombardelli, D. Pearce, D. Karagianni, D. Biswas, D. Levi, E. Larose Cadieux, E. Lim, Emma C Colliver, E. Nye, F. Gálvez-Cancino, F. Gimeno-Valiente, G. Kassiotis, Georgia Stavrou, Gerasimos Mastrokalos, Helen L Lowe, Ignacio Matos, I. Noorani, J. Goldman, J. Reading, J. Rane, J. Nicod, John A. Hartley, K. Peggs, Katey S. S. Enfield, Kayalvizhi Selvaraju, K. Thol, Kevin W. Ng, Kezhong Chen, K. Dijkstra, Kristiana Grigoriadis, Krupa Thakkar, L. Ensell, Mansi Shah, Maria Litovchenko, M. Jamal-Hanjani, Mariana Werner Sunderland, M. Huska, M. Hill, M. Dietzen, Michelle Leung, M. Escudero, M. Tanić, Monica Sivakumar, O. Chervova, Olivia Lucas, O. Pich, Othman Al-Sawaf, Paulina Prymas, Philip S Hobson, Piotr Pawlik, R. Stone, R. Bentham, Roberto Vendramin, S. Saghafinia, Samuel Gamble, S. Veeriah, S. Ung, Sergio A Quezada, Sharon P. Vanloo, S. Hessey, S. Ward, Sian Harries, S. Boeing, Stephan Beck, S. Bola, Takahiro Karasaki, Tamara Denner, T. Marafioti, Thomas P. Jones, V. Spanswick, Vittorio Barbè, Wei-Ting Lu, Wing Kin Liu, Yin Wu, Yutaka Naito, Zoe Ramsden, C. Veiga, Gary Royle, C. Collins-Fekete, Francesco Fraioli, Paul Ashford, Martin D. Forster, Siow-Ming Lee, E. Borg, M. Falzon, D. Papadatos-Pastos, James Wilson, T. Ahmad, A. Procter, Asia Ahmed, Magali N. Taylor, Arjun Nair, David S. Lawrence, D. Patrini, Neal Navani, R. Thakrar, Sam M. Janes, Emilie Martinoni Hoogenboom, Fleur Monk, James W. Holding, Junaid Choudhary, Kunal Bhakhri, M. Scarci, Pat Gorman, Reena Khiroya, Robert C. M. Stephens, Y. Wong, Zoltán Káplár, S. Bandula, A. Hackshaw, A. Hacker, A. Sharp, Sean Smith, Harjot Kaur Dhanda, Camilla Pilotti, R. Leslie, Anca-Ioana Grapa, Hanyun Zhang, K. AbdulJabbar, Xiaoxi Pan, Yinyin Yuan, David Chuter, M. Mackenzie, S. Chee, A. Alzetani, Judith Cave, J. Richards, Eric Lim, P. De Sousa, Simon Jordan, Alex Rice, H. Raubenheimer, H. Bhayani, L. Ambrose, A. Devaraj, Hema Chavan, S. Begum, Silviu I. Buderi, Daniel Kaniu, Mpho Malima, S. Booth, Andrew G. Nicholson, Nádia Fernandes, Pratibha Shah, C. Proli, M. Hewish, S. Danson, M. Shackcloth, Lily Robinson, P. Russell, K. Blyth, A. Kidd, C. Dick, J. Le Quesne, Alan Kirk, Mohd. Asif, Rocco Bilancia, N. Kostoulas, Mathew Thomas, R. Hynds, N. Kanu, S. Zaccaria, E. Grönroos, C. Swanton
          </td>
          <td>2024-06-13</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>81</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac81e9d8c69b0fb75e7224cb7e99d720bb7db9c5" target='_blank'>
              KHSRP has oncogenic functions and regulates the expression and alternative splicing of DNA repair genes in breast cancer MDA-MB-231 cells
              </a>
            </td>
          <td>
            Xuelaiti Paizula, Aliya Wulaying, Dong Chen, Jianghua Ou
          </td>
          <td>2024-06-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Osteosarcoma is a rare primary bone tumor for which no significant therapeutic advancement has been made since the late 1980s despite ongoing efforts. Overall, the five-year survival rate remains about 65%, and is much lower in patients with tumors unresponsive to methotrexate, doxorubicin, and cisplatin therapy. Genetic studies have not revealed actionable drug targets, but our group, and others, have reported that epigenomic biomarkers, including regulatory RNAs, may be useful prognostic tools for osteosarcoma. We tested if microRNA (miRNA) transcriptional patterns mark the transition from a chemotherapy sensitive to resistant tumor phenotype. Small RNA sequencing was performed using 14 patient matched pre-chemotherapy biopsy and post-chemotherapy resection high-grade osteosarcoma frozen tumor samples. Independently, small RNA sequencing was performed using 14 patient matched biopsy and resection samples from untreated tumors. Separately, miRNA specific Illumina DASL arrays were used to assay an independent cohort of 65 pre-chemotherapy biopsy and 26 patient matched post-chemotherapy resection formalin fixed paraffin embedded (FFPE) tumor samples. mRNA specific Illumina DASL arrays were used to profile 37 pre-chemotherapy biopsy and five post-chemotherapy resection FFPE samples, all of which were also used for Illumina DASL miRNA profiling. The National Cancer Institute Therapeutically Applicable Research to Generate Effective Treatments dataset, including PCR based miRNA profiling and RNA-seq data for 86 and 93 pre-chemotherapy tumor samples, respectively, was also used. Paired differential expression testing revealed a profile of 17 miRNAs with significantly different transcriptional levels following chemotherapy. Genes targeted by the miRNAs were differentially expressed following chemotherapy, suggesting the miRNAs may regulate transcriptional networks. Finally, an in vitro pharmacogenomic screen using miRNAs and their target transcripts predicted response to a set of candidate small molecule therapeutics which potentially reverse the chemotherapy resistance phenotype and synergize with chemotherapy in otherwise treatment resistant tumors. Importantly, these novel therapeutic targets are distinct from targets identified by a similar pharmacogenomic analysis of previously published prognostic miRNA profiles from pre chemotherapy biopsy specimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/030c30436809f8513c1b8d90a7748f211e900709" target='_blank'>
              A dynamic microRNA profile that tracks a chemotherapy resistance phenotype in osteosarcoma. Implications for novel therapeutics
              </a>
            </td>
          <td>
            Christopher E Lietz, Erik T Newman, Andrew D Kelly, David H Xiang, Ziying Zhang, Nikhil Ramavenkat, Joshua J Bowers, Santiago A Lozano-Calderón, David H Ebb, Kevin A. Raskin, Gregory M Cote, Edwin Choy, G. P. Nielsen, Ioannis S. Vlachos, Benjamin Haibe-Kains, D. Spentzos
          </td>
          <td>2024-06-20</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Resistance to cancer therapy is driven by both cell-intrinsic and microenvironmental factors. Previous work has revealed that multiple resistant cell fates emerge in melanoma following treatment with targeted therapy and that, in vitro, these resistant fates are determined by the transcriptional state of individual cells prior to exposure to treatment. What remains unclear is whether these resistant fates are shared across different genetic backgrounds and how, if at all, these resistant fates interact with the tumor microenvironment. Through spatial transcriptomics and single-cell RNA sequencing, we uncovered distinct resistance programs in melanoma cells shaped by both intrinsic cellular states and the tumor microenvironment. Consensus non-negative matrix factorization revealed shared intrinsic resistance programs across different cell lines, highlighting the presence of universal and unique resistance pathways. In patient samples, we demonstrated that these resistance programs coexist within individual tumors and associate with diverse immune signatures, suggesting that the tumor microenvironment and distribution of resistant fates are closely connected. Single-cell resolution spatial transcriptomics in xenograft models revealed both intrinsically determined and extrinsically influenced resistant fates. Overall, this work demonstrates that each therapy resistant fate coexists with a distinct immune microenvironment in tumors and that, in vivo, tissue features, such as regions of necrosis, can influence which resistant fate is adopted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5979abc00b349ddf0161079fda7fee453bd0ce0a" target='_blank'>
              Spatial transcriptomics reveals influence of microenvironment on intrinsic fates in melanoma therapy resistance
              </a>
            </td>
          <td>
            Ryan H. Boe, Catherine G. Triandafillou, R. Lazcano, J. Wargo, Arjun Raj
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="Liver cancer is a heterogeneous disease and is one of the leading causes of cancer deaths worldwide. Hepatocellular carcinoma, comprising approximately 90% of cases, presents a formidable challenge with a less than 20% 5-year survival rate despite recent treatment advancements. The impediments of drug resistance and off-target effects underscore the critical need for innovative and efficacious therapies. Harnessing the growing understanding of RNA function offers a promising avenue to address previously "undruggable" proteins, transcripts, and genes. Various RNAs demonstrate the potential to selectively act on these targets, expanding the scope of therapeutic interventions. With diverse regulatory roles in cancer pathways, RNAs emerge as valuable targets and tools for anticancer therapy development. This article provides an in-depth exploration of current RNA-based therapies, elucidates their mechanisms of action, and discusses their combinations with chemo-/immunotherapies in clinical trials for hepatocellular carcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16116ad053310e9b38070cc9f4f72620da429655" target='_blank'>
              RNA-based therapies in hepatocellular carcinoma: state of the art and clinical perspectives
              </a>
            </td>
          <td>
            Le Xing, Zhen-Kai Wang, Dongli Li, Jing Li, Ming Liu
          </td>
          <td>2024-06-20</td>
          <td>Hepatoma Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea3f7e6f9fd3216c09d0776dc7d2e0f2d441e9d9" target='_blank'>
              Insights into immune microenvironment and therapeutic targeting in androgen-associated prostate cancer subtypes
              </a>
            </td>
          <td>
            Liang Huang, Yu Xie, Shusuan Jiang, Tao Dai, Zhenzhou Xu, Hong Shan
          </td>
          <td>2024-08-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Synonymous mutations are generally considered neutral, while their roles in the human genome remain largely unexplored. Herein, we employed the PEmax system to create a library of 297,900 epegRNAs and performed extensive screening to identify synonymous mutations that impact cell fitness. While most synonymous mutations are neutral, we found that some can elicit phenotypic effects. By developing a specialized machine learning tool, we uncovered their impact on various biological processes such as mRNA splicing and transcription, supported by multifaceted experimental evidence. Notably, synonymous mutations can alter RNA folding and affect translation, as demonstrated by PLK1_S2. By integrating screening data with our model, we successfully predicted clinically deleterious synonymous mutations. This research deepens our understanding of synonymous mutations and provides insights for clinical disease studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05154aaeac9ce9e07cf3dc417d5a42149dfe9604" target='_blank'>
              Prime editor-based high-throughput screening reveals functional synonymous mutations in the human genome
              </a>
            </td>
          <td>
            Xuran Niu, Wei Tang, Yongshuo Liu, Ying Liu, Binrui Mo, Ying Yu, Wensheng Wei
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c9abec0157ccbe7ffcea6b2ec6d1e2342588925" target='_blank'>
              Sustained inactivation of the Polycomb PRC1 complex induces DNA repair defects and genomic instability in epigenetic tumors
              </a>
            </td>
          <td>
            C. Rawal, V. Loubiere, N. Butova, Juliette Gracia, V. Parreno, C. Merigliano, Anne-Marie Martinez, G. Cavalli, I. Chiolo
          </td>
          <td>2024-06-18</td>
          <td>Histochemistry and Cell Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Accurate detection of homologous recombination deficiency (HRD) in cancer patients is paramount in clinical applications, as HRD confers sensitivity to poly (ADP-ribose) polymerase (PARP) inhibitors. With the advances in genome sequencing technology, mutational profiling on a genome-wide scale has become readily accessible, and our knowledge of the genomic consequences of HRD has been greatly expanded and refined. Here, we review the recent advances in HRD detection methods. We examine the copy number and structural alterations that often accompany the genome instability that results from HRD, describe the advantages of mutational signature-based methods that do not rely on specific gene mutations, and review some of the existing algorithms used for HRD detection. We also discuss the choice of sequencing platforms (panel, exome, or whole-genome) and catalog the HRD detection assays used in key PARP inhibitor trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b04df45a7add0404411b8f4a6e0dd524a6a72b61" target='_blank'>
              Recent Advances in Genomic Approaches for the Detection of Homologous Recombination Deficiency.
              </a>
            </td>
          <td>
            Yoo-Na Kim, D. Gulhan, Hu Jin, Dominik Glodzik, Peter J. Park
          </td>
          <td>2024-07-17</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/155d1a740aed9984732768efd341957b63aa39bb" target='_blank'>
              An integrated single-cell RNA-seq map of human neuroblastoma tumors and preclinical models uncovers divergent mesenchymal-like gene expression programs
              </a>
            </td>
          <td>
            R. Chapple, Xueying Liu, Sivaraman Natarajan, Margaret I. M. Alexander, Yuna Kim, Anand G. Patel, Christy W. LaFlamme, Min Pan, William C. Wright, Hyeong-Min Lee, Yinwen Zhang, Meifen Lu, Selene C. Koo, Courtney Long, John Harper, C. Savage, Melissa D. Johnson, Thomas Confer, Walter J. Akers, M. Dyer, Heather Sheppard, J. Easton, P. Geeleher
          </td>
          <td>2024-06-19</td>
          <td>Genome Biology</td>
          <td>2</td>
          <td>64</td>
        </tr>

        <tr id="BRAF and MEK inhibitors (BRAF/MEKi) have radically changed the treatment landscape of advanced BRAF mutation-positive tumours. However, limited efficacy and emergence of drug resistance are major handicaps for successful treatments. Here, by using relevant preclinical models, we found that Connexin43 (Cx43), a protein that plays a role in cell-to-cell communication, increases effectiveness of BRAF/MEKi by recruiting DNA repair complexes to lamin-associated domains and promoting persistent DNA damage and cellular senescence. The nuclear compartmentalization promoted by Cx43 contributes to genome instability and synthetic lethality caused by excessive DNA damage, which could lead to a novel therapeutic approach for these tumours to overcome drug resistance. Based on these findings, we designed an innovative drug combination using small extracellular vesicles (sEVs) to deliver the full-Cx43 in combination with the BRAF/MEKi. This study reveals Cx43 as a new player on DNA repair and BRAF/MEKi response, underlining the therapeutical potential that this approach could eventually have in the clinic to overcome the limitations of current therapies and improve treatment outcomes for patients with advanced BRAF mutant tumours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2931f384d734ebe11eb6f5f59fbfc1a5a5d9c4e2" target='_blank'>
              Cx43 Enhances Response to BRAF/MEK Inhibitors by Reducing DNA Repair Capacity
              </a>
            </td>
          <td>
            A. Varela-Vázquez, A. Guitián-Caamaño, P. Carpintero-Fernández, Vanesa Álvarez, Alexander Carneiro-Figueira, M. Varela-Eirín, Teresa Calleja-Chuclá, S. B. Bravo-López, Anxo Vidal, Juan Sendón-Lago, M. Rodríguez-Candela Mateos, José R Caeiro, Miguel G. Blanco, Guadalupe Sabio, María Quindós, Carmen Rivas, David Santamaría, Carlos Fernández-Lozano, E. Fonseca, Pablo Huertas, Berta Sanchez-Laorden, Constance Alabert, M. Mayán
          </td>
          <td>2024-07-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/152652d8b9732cb4ce54ecaade267ae36f26fe12" target='_blank'>
              Analysis of 10,478 cancer genomes identifies candidate driver genes and opportunities for precision oncology.
              </a>
            </td>
          <td>
            B. Kinnersley, Amit Sud, Andrew Everall, A. Cornish, D. Chubb, R. Culliford, Andreas J. Gruber, Adrian Lärkeryd, C. Mitsopoulos, David C Wedge, R. Houlston
          </td>
          <td>2024-06-18</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/435232aea775847959662a12da4da5a9a69c73c7" target='_blank'>
              A whole-genome CRISPR screen identifies the spindle accessory checkpoint as a locus of nab-paclitaxel resistance in a pancreatic cancer cell line
              </a>
            </td>
          <td>
            Priya Mondal, George Alyateem, Allison V. Mitchell, Michael M. Gottesman
          </td>
          <td>2024-07-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Acquired chemoresistance remains a significant challenge in the clinics as most of the treated cancers eventually emerge as hard-to-treat phenotypes. Therefore, identifying chemoresistance targets is highly warranted to manage the disease better. In this study, we employed a label-free LC-MS/MS-based quantitative proteomics analysis to identify potential targets and signaling pathways underlying acquired chemoresistance in a sub-cell line (A549DR) derived from the parental lung adenocarcinoma cells (A549) treated with gradually increasing doses of doxorubicin (DOX). Our proteomics analysis identified 146 upregulated and 129 downregulated targets in A549DR cells. The KEGG pathway and Gene ontology (GO) analysis of differentially expressed upregulated and downregulated proteins showed that most abundant upregulated pathways were related to metabolic pathways, cellular senescence, cell cycle, and p53 signaling. Meanwhile, the downregulated pathways were related to spliceosome, nucleotide metabolism, DNA replication, nucleotide excision repair, and nuclear-cytoplasmic transport. Further, STRING analysis of upregulated biological processes showed a protein-protein interaction (PPI) between CDK1, AKT2, SRC, STAT1, HDAC1, FDXR, FDX1, NPC1, ALDH2, GPx1, CDK4, and B2M, proteins. The identified proteins in this study might be the potential therapeutic targets for mitigating DOX resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/974ec76bceeb820f708d654bf77904bfd435f4c2" target='_blank'>
              Identification of chemoresistance targets in doxorubicin-resistant lung adenocarcinoma cells using LC-MS/MS-based proteomics.
              </a>
            </td>
          <td>
            Nuzhat Bano, K. Kainat, Mohammad Imran Ansari, Anjali Pal, Sana Sarkar, Pradeep Kumar Sharma
          </td>
          <td>2024-08-05</td>
          <td>Journal of chemotherapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c7d40b34ce0a1270d80bd0970ece2da4263143d" target='_blank'>
              Integrative single-cell and bulk transcriptomes analyses reveals heterogeneity of serine-glycine-one-carbon metabolism with distinct prognoses and therapeutic vulnerabilities in HNSCC
              </a>
            </td>
          <td>
            Lixuan Wang, Rongchun Yang, Yue Kong, Jing Zhou, Yingyao Chen, Rui Li, Chuwen Chen, Xinran Tang, Xiaobing Chen, J. Xia, Xi-juan Chen, Bing-lin Cheng, Xianyue Ren
          </td>
          <td>2024-06-17</td>
          <td>International Journal of Oral Science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/671f339111b7b478e20a7b9554c095e50b64b22b" target='_blank'>
              Cellular and molecular events in colorectal cancer: biological mechanisms, cell death pathways, drug resistance and signalling network interactions
              </a>
            </td>
          <td>
            Lei Yan, Jia Shi, Jiazuo Zhu
          </td>
          <td>2024-07-20</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Currently, cancer is still a leading cause of human death globally. Tumor deterioration comprises multiple events including metastasis, therapeutic resistance and immune evasion, all of which are tightly related to the phenotypic plasticity especially epithelial–mesenchymal plasticity (EMP). Tumor cells with EMP are manifest in three states as epithelial–mesenchymal transition (EMT), partial EMT, and mesenchymal–epithelial transition, which orchestrate the phenotypic switch and heterogeneity of tumor cells via transcriptional regulation and a series of signaling pathways, including transforming growth factor‐β, Wnt/β‐catenin, and Notch. However, due to the complicated nature of EMP, the diverse process of EMP is still not fully understood. In this review, we systematically conclude the biological background, regulating mechanisms of EMP as well as the role of EMP in therapy response. We also summarize a range of small molecule inhibitors, immune‐related therapeutic approaches, and combination therapies that have been developed to target EMP for the outstanding role of EMP‐driven tumor deterioration. Additionally, we explore the potential technique for EMP‐based tumor mechanistic investigation and therapeutic research, which may burst vigorous prospects. Overall, we elucidate the multifaceted aspects of EMP in tumor progression and suggest a promising direction of cancer treatment based on targeting EMP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32fcd891c5d88b251baf8fc511031710e0fc1629" target='_blank'>
              Epithelial–mesenchymal plasticity in cancer: signaling pathways and therapeutic targets
              </a>
            </td>
          <td>
            Xiangpeng Wang, Xiao-yong Xue, Mingshi Pang, Liuchunyang Yu, Jinxiu Qian, Xiaoyu Li, Meng Tian, Aiping Lyu, Cheng Lu, Yuanyan Liu
          </td>
          <td>2024-08-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Prostate cancer (PCa) poses significant challenges in treatment, particularly when it progresses to a metastatic, castrate-resistant state. Conventional therapies, including chemotherapy, radiotherapy, and hormonal treatments, often fail due to toxicities, off-target effects, and acquired resistance. This research perspective defines an alternative therapeutic strategy focusing on the metabolic vulnerabilities of PCa cells, specifically their reliance on non-essential amino acids such as cysteine. Using an engineered enzyme cyst(e)inase to deplete the cysteine/cystine can induce oxidative stress and DNA damage in cancer cells. This depletion elevates reactive oxygen species (ROS) levels, disrupts glutathione synthesis, and enhances DNA damage, leading to cancer cell death. The combinatorial use of cyst(e)inase with agents targeting antioxidant defenses, such as thioredoxins, further amplifies ROS accumulation and cytotoxicity in PCa cells. Overall, in this perspective provides a compressive overview of the previous work on manipulating amino acid metabolism and redox balance modulate the efficacy of DNA repair-targeted and immune checkpoint blockade therapies in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbf096fee4a52be93875a3cc7398db3ac71384ef" target='_blank'>
              Starving cancer cells to enhance DNA damage and immunotherapy response
              </a>
            </td>
          <td>
            Aashirwad Shahi, Dawit Kidane
          </td>
          <td>2024-06-20</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Melanoma being one of the most common and deadliest skin cancers, has been rising since the past decade. Patients at advanced stages of the disease have very poor prognoses, as opposed to at the earlier stages. Nowadays the standard-of-care of advanced melanoma is resection followed by immune checkpoint inhibition based immunotherapy. However, a substantial proportion of patients either do not respond or develop resistances. This underscores a need for novel approaches and therapeutic targets as well as a better understanding of the mechanisms of melanoma pathogenesis. Long non-coding RNAs (lncRNAs) comprise a poorly characterized class of functional players and promising targets in promoting malignancy. Certain lncRNAs have been identified to play integral roles in melanoma progression and drug resistances, however systematic screens to uncover novel functional lncRNAs are scarce. Here, we profile differentially expressed lncRNAs in patient derived short-term metastatic cultures and BRAF-MEK-inhibition resistant cells. We conduct a focused growth-related CRISPR-inhibition screen of overexpressed lncRNAs, validate and functionally characterize lncRNA hits with respect to cellular growth, invasive capacities and apoptosis in vitro as well as the transcriptomic impact of our lead candidate the novel lncRNA XLOC_030781. In sum, we extend the current knowledge of ncRNAs and their potential relevance on melanoma. Significance Previously considered as transcriptional noise, lncRNAs have emerged as novel players in regulating many cellular aspects in health and disease including melanoma. However, the number and as well as the extent of functional significance of most lncRNAs remains elusive. We provide a comprehensive strategy to identify functionally relevant lncRNAs in melanoma by combining expression profiling with CRISPR-inhibition growths screens lowering the experimental effort. We also provide a larger resource of differentially expressed lncRNAs with potential implications in melanoma growth and invasion. Our results broaden the characterized of lncRNAs as potential targets for future therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a768f16ebbf2c3ec424c154b1baf19ad0e931615" target='_blank'>
              CRISPR-inhibition screen for lncRNAs linked to melanoma growth and metastasis
              </a>
            </td>
          <td>
            Stavroula Petroulia, K. Hockemeyer, Shashank Tiwari, Pietro Berico, Sama Shamloo, Seyedeh Elnaz Banijamali, Eleazar Vega-Saenz de Miera, Yixiao Gong, Palaniraja Thandapani, Eric Wang, Michael Schulz, A. Tsirigos, Iman Osman, Ioannis D. Aifantis, Jochen Imig
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="The genome of a cell is continuously battered by a plethora of exogenous and endogenous processes that can lead to damaged DNA. Repair mechanisms correct this damage most of the time, but failure to do so leaves mutations. Mutations do not occur in random manner, but rather typically follow a more or less specific pattern due to known or imputed mutational processes. Mutational signature analysis is the process by which the predominant mutational process can be inferred for a cancer and can be used in several contexts to study both the genesis of cancer and its response to therapy. Recent pan-cancer genomic efforts such as "The Cancer Genome Atlas" have identified numerous mutational signatures that can be categorized into single base substitutions, doublet base substitutions, or small insertions/deletions. Understanding these mutational signatures as they occur in non-small lung cancer could improve efforts at prevention, predict treatment response to personalized treatments, and guide the development of therapies targeting tumor evolution. For non-small cell lung cancer, several mutational signatures have been identified that correlate with exposures such as tobacco smoking and radon and can also reflect endogenous processes such as aging, APOBEC activity, and loss of mismatch repair. Herein, we provide an overview of the current knowledge of mutational signatures in non-small lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37777d22dfc5a2bd99dc42f300aad4d0a01e922f" target='_blank'>
              Topography of mutational signatures in non-small cell lung cancer: emerging concepts, clinical applications, and limitations.
              </a>
            </td>
          <td>
            Ritujith Jayakrishnan, David J. Kwiatkowski, Michal G Rose, Amin H. Nassar
          </td>
          <td>2024-06-22</td>
          <td>The oncologist</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The fitness effects of new mutations determine key properties of evolutionary processes. Beneficial mutations drive evolution, yet selection is also shaped by the frequency of small-effect deleterious mutations, whose combined effect can burden otherwise adaptive lineages and alter evolutionary trajectories and outcomes in clonally evolving organisms such as viruses, microbes, and tumors. The small effect sizes of these important mutations have made accurate measurements of their rates difficult. In microbes, assessing the effect of mutations on growth can be especially instructive, as this complex phenotype is closely linked to fitness in clonally evolving organisms. Here, we perform high-throughput time-lapse microscopy on cells from mutation-accumulation strains to precisely infer the distribution of mutational effects on growth rate in the budding yeast, Saccharomyces cerevisiae. We show that mutational effects on growth rate are overwhelmingly negative, highly skewed towards very small effect sizes, and frequent enough to suggest that deleterious hitchhikers may impose a significant burden on evolving lineages. By using lines that accumulated mutations in either wild-type or slippage repair-defective backgrounds, we further disentangle the effects of 2 common types of mutations, single-nucleotide substitutions and simple sequence repeat indels, and show that they have distinct effects on yeast growth rate. Although the average effect of a simple sequence repeat mutation is very small (approximately 0.3%), many do alter growth rate, implying that this class of frequent mutations has an important evolutionary impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e2d4d1c86097c9432759848b7286f55d338b931" target='_blank'>
              Spontaneous single-nucleotide substitutions and microsatellite mutations have distinct distributions of fitness effects
              </a>
            </td>
          <td>
            Y. Plavskin, M. S. de Biase, Naomi Ziv, Libuše Janská, Yuan O. Zhu, David W. Hall, Roland F Schwarz, Daniel Tranchina, M. Siegal
          </td>
          <td>2024-07-01</td>
          <td>PLOS Biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Abstract Androgen receptor- (AR-) indifference is a mechanism of resistance to hormonal therapy in prostate cancer (PC). Here we demonstrate that ONECUT2 (OC2) activates resistance through multiple drivers associated with adenocarcinoma, stem-like and neuroendocrine (NE) variants. Direct OC2 gene targets include the glucocorticoid receptor (GR; NR3C1) and the NE splicing factor SRRM4, which are key drivers of lineage plasticity. Thus, OC2, despite its previously described NEPC driver function, can indirectly activate a portion of the AR cistrome through epigenetic activation of GR. Mechanisms by which OC2 regulates gene expression include promoter binding, enhancement of genome-wide chromatin accessibility, and super-enhancer reprogramming. Pharmacologic inhibition of OC2 suppresses lineage plasticity reprogramming induced by the AR signaling inhibitor enzalutamide. These results demonstrate that OC2 activation promotes a range of drug resistance mechanisms associated with treatment-emergent lineage variation in PC and support enhanced efforts to therapeutically target OC2 as a means of suppressing treatment-resistant disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afa57725f4c34a58c8bfb81365718e7515db8a48" target='_blank'>
              ONECUT2 acts as a lineage plasticity driver in adenocarcinoma as well as neuroendocrine variants of prostate cancer
              </a>
            </td>
          <td>
            Chen Qian, Qian Yang, Mirja Rotinen, Rongrong Huang, Hyoyoung Kim, Brad Gallent, Yiwu Yan, Radu M. Cadaneanu, Baohui Zhang, S. Kaochar, Stephen J. Freedland, Edwin M Posadas, Leigh Ellis, Dolores Di Vizio, C. Morrissey, Peter S. Nelson, Lauren Brady, Ramachandran Murali, Moray J. Campbell, Wei Yang, Beatrice S Knudsen, E. Mostaghel, Huihui Ye, I. Garraway, Sungyong You, Michael R Freeman
          </td>
          <td>2024-06-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="A central challenge in the quest for precise gene regulation within mammalian cells is the development of regulatory networks that can achieve perfect adaptation-where outputs consistently return to a set baseline post-stimulus. Here, we present such a system that leverages the CRISPR activation (CRISPRa) and anti-CRISPR proteins as two antithetic elements to establish perfect adaptation in mammalian cells and dynamically regulate gene expression. We demonstrate that this system can maintain stable expression levels of target genes in the face of external perturbations, thus providing a robust platform for biological applications. The versatility of our system is further showcased through its integration with endogenous regulatory mechanisms in T cells, such as the NF-κB-mediated immune response, and its ability to program apoptosis responses for precise spatial and temporal control of cellular growth and death. This study not only advances our understanding of gene regulation in mammalian cells but also opens new avenues for therapeutic intervention, particularly in diseases characterized by dysregulated gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9bdcf363536b5c7b6585a226adb59f423a716ec" target='_blank'>
              CRISPR perfect adaptation for robust control of cellular immune and apoptotic responses.
              </a>
            </td>
          <td>
            Yichi Zhang, Shuyi Zhang
          </td>
          <td>2024-08-01</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Antibiotic resistance is the leading cause of death globally, with a higher possibility of the emergence of highly resistant pathogens, leading to epidemics. Several antibiotic resistance mechanisms have been discovered, such as enhanced efflux of antibiotics, reduced influx of antibiotics, alteration of antibiotics or their targets, and adaptation to antibiotics. However, this mechanism cannot fully explain the development of antibiotic resistance because the genes associated with this mechanism have been elucidated. However, the factors governing their regulation are not yet fully understood. Recent studies have highlighted the epigenetic and epitranscriptomic roles of antibiotic resistance development-associated genes. Epigenetic modification is associated with DNA modification, whereas epitranscriptomic modification is associated with RNA modification to control gene expression by regulating various biological phenomena such as splicing, translation, and stability. Therefore, this review will focus on the discovery of epigenetic modifications, particularly by DNA methyltransferases, such as restriction-modification (R-M) systems associated with methyltransferases, orphan DNA methyltransferases, and nucleoid-associated proteins that contribute to the development of antibiotic resistance. This scrutinization further expands to epitranscriptomic modification of non-coding RNA, which has a role in the regulation of antibiotic resistance. Epitranscriptomic modification of ribosomal RNA (rRNA), which is a major target of antibiotics, has been well explored. while non-coding RNA such as cis and trans small non coding RNA, and riboswitches are poorly explored. This epigenetic and epitranscriptomic modification will help to understand the regulation of antibiotic resistance-associated genes, which will help to identify key regulators of antibiotic resistance, paving the way for new antibiotic discovery, leading to decreased antibiotic mortality globally.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24b16c6a0ffde83445547b9ffde6ff971186ffa3" target='_blank'>
              A comprehensive review on epigenetic and epitranscriptomic-mediated regulation of antibiotic resistance
              </a>
            </td>
          <td>
            P. K. Giri, Shahil Alam, Madhav Dhakal
          </td>
          <td>2024-07-24</td>
          <td>F1000Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background. Lung adenocarcinoma (LUAD) is the most common subtype of non-small cell lung cancer (NSCLC) and frequently affects non-smokers, especially women. It is characterized by a complex genetic profile and interactions with its microenvironment, which contribute to its aggressive and adaptable nature. Early symptoms are often subtle, leading to late diagnoses. Treatment approaches have advanced with targeted therapies and immunotherapy supplementing traditional chemotherapy and radiation. Despite these advancements, the prognosis remains variable, highlighting the need for continued research into new treatment strategies to improve outcomes. Method. In this study, we employed Single-cell RNA Sequencing (scRNA-seq) to comprehensively analyze the impact of chemotherapy on lung adenocarcinoma at the individual cell level. By comparing before and after treatment samples, we assessed the differential expression of genes and pathways, revealing insights into how different cell types within the tumour respond to chemotherapy. This approach enabled us to pinpoint specific mechanisms of drug resistance and highlight potential therapeutic targets for overcoming these challenges. Results. Our analysis uncovered substantial changes in gene expression between primary tumour cells and metastatic cells following chemotherapy. Notably, we observed that 45 pathways were shared between the top 50 upregulated pathways in the primary tumour and the top 50 downregulated pathways in the metastatic tumour post-chemotherapy. Conversely, there was no overlap between the top 50 downregulated pathways in the primary tumour and the top 50 upregulated pathways in the metastatic tumour after chemotherapy. This suggests that chemotherapy effectively downregulated the major upregulated pathways but did not upregulate the key downregulated pathways in metastatic tumours. Conclusions. Integrating single-cell transcriptomics into LUAD research offers detailed insights into the tumour's response to chemotherapy and its interaction with the immune system. This approach enhances our understanding of LUAD and aids in developing targeted and effective treatments. Based on our analysis, we hypothesize that combining chemotherapy with drugs designed to upregulate the downregulated pathways in primary tumour cells could significantly enhance treatment efficacy and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c9748c84cdcd0c997de55e9e674c83813950dc2" target='_blank'>
              Unveiling Chemotherapy's Impact on Lung Cancer through Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            MD Saed Sayad, PhD Mark Hiatt, Mba MS Hazem Mustafa Md
          </td>
          <td>2024-07-10</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33af20a7d0aac05ac83adef2c51ac9ffcee3995c" target='_blank'>
              Enhancer engagement sustains oncogenic transformation and progression of B-cell precursor acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            G. Corleone, C. Sorino, M. Caforio, S. Di Giovenale, F. de Nicola, F. Goeman, V. Bertaina, A. Pitisci, C. Cortile, Franco Locatelli, V. Folgiero, Maurizio Fanciulli
          </td>
          <td>2024-06-27</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Therapy-induced acquired resistance limits the clinical effectiveness of mutation-specific RAS inhibitors in colorectal cancer. It is unknown whether broad-spectrum active-state RAS inhibitors meet similar limitations. Here, we identify and categorize mechanisms of resistance to the broad-spectrum active-state RAS inhibitor RMC-7977 in colorectal cancer cell lines. We found that KRAS-mutant colorectal cancer cell lines are universally sensitive to RMC-7977, inhibiting the RAS-RAF-MEK-ERK axis, halting proliferation and in some cases inducing apoptosis. To monitor KRAS downstream effector pathway activity, we developed a compartment-specific dual-color ERK activity reporter. RMC-7977 treatment reduced reporter activity. However, long-term dose escalation with RMC-7977 revealed multiple patterns of reporter reactivation in emerging resistant cell populations that correlated with phosphorylation states of compartment-specific ERK targets. Cells sorted for high, low, or cytoplasmic reporter activity exhibited distinct patterns of genomic mutations, phospho-protein, and transcriptional activities. Notably, all resistant subpopulations showed dynamic ERK regulation in the presence of the RAS inhibitor, unlike the parental sensitive cell lines. High levels of RAS downstream activities were observed in cells characterized by a KRAS Y71H resistance mutation. In contrast, RAS inhibitor-resistant populations with low, or cytoplasmic ERK reporter reactivation displayed different genetic alterations, among them RAF1 S257L and S259P mutations. Colorectal cancer cells resistant to RMC-7977 and harboring the RAF1 mutation specifically exhibited synergistic sensitivity to concurrent RAS and RAF inhibition. Our findings endorse reporter-assisted screening together with single-cell analyses as a powerful approach for dissecting the complex landscape of therapy resistance. The strategy offers opportunities to develop clinically relevant combinatorial treatments to counteract emergence of resistant cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58f2dfe945929a08a6260606b0df62f40daad49b" target='_blank'>
              Reporter-based screening identifies RAS-RAF mutations as drivers of resistance to active-state RAS inhibition in colorectal cancer
              </a>
            </td>
          <td>
            Oleksandra Aust, Eric Blanc, Mareen Lüthen, Viola Hollek, Rosario Astaburuaga-García, B. Klinger, Alexandra Trinks, D. Beule, Björn Papke, David Horst, Nils Blüthgen, Christine Sers, Channing Der, M. Morkel
          </td>
          <td>2024-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Human pluripotent stem cells (hPSC) are pivotal in regenerative medicine, yet their in vitro expansion often leads to genetic abnormalities, raising concerns about their safety in clinical applications. This study analyzed ten human embryonic stem cell lines across multiple passages to elucidate the dynamics of chromosomal abnormalities and single nucleotide variants (SNVs) in 380 cancer-related genes. Prolonged in vitro culture resulted in 80% of the lines acquiring gains of chromosomes 20q or 1q, both known for conferring in vitro growth advantage. 70% of lines also acquired other copy number variants (CNVs) outside the recurrent set. Additionally, we detected 122 SNVs in 88 genes, with all lines acquiring at least one de novo SNV during culture. Our findings show higher loads of both CNVs and SNVs at later passages which are due to the cumulative acquisition of mutations over a longer time in culture and not to an increased rate of mutagenesis over time. Importantly, we observed that SNVs and rare CNVs follow the acquisition of chromosomal gains in 1q and 20q, while most of the low-passage and genetically balanced samples were devoid cancer-associated mutations. This suggests that the recurrent chromosomal abnormalities are the potential drivers for the acquisition of other mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97f552b60bd87e180c4d6a5dd522428893e4b81b" target='_blank'>
              De novo cancer mutations frequently associate with recurrent chromosomal abnormalities during long-term human pluripotent stem cell culture
              </a>
            </td>
          <td>
            Diana Al Delbany, Manjusha S Ghosh, Nuša Krivec, Anfien Huygebaert, M. Regin, Chi Mai Duong, Yingnan Lei, K. Sermon, Catharina Olsen, C. Spits
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Cancer is a complex and heterogeneous disease, in which a number of genetic and epigenetic changes occur in tumor onset and progression. Recent studies indicate that changes at the RNA level are also involved in tumorigenesis, such as adenosine-to-inosine (A-to-I) RNA editing. Here, we systematically investigate transcriptome-wide A-to-I editing events in a large number of samples from Non-Hodgkin lymphomas (NHLs). Using a computational pipeline that determines significant differences in editing level between NHL and normal samples at known A-to-I editing sites, we identify a number of differentially edited editing sites between NHL subtypes and normal samples. Most of the differentially edited sites are located in non-coding regions, and many such sites show a strong correlation between gene expression level and editing efficiency, indicating that RNA editing might have direct consequences for the cancer cell’s aberrant gene regulation status in these cases. Moreover, we establish a strong link between RNA editing and NHL by demonstrating that NHL and normal samples and even NHL subtypes can be distinguished based on genome-wide RNA editing profiles alone. Our study establishes a strong link between RNA editing, cancer and aberrant gene regulation in NHL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3709fa38f5f1119b1fcb4db3f2e4cd77244ecc08" target='_blank'>
              A-to-I Editing Is Subtype-Specific in Non-Hodgkin Lymphomas
              </a>
            </td>
          <td>
            Cai Chen, Ralf Bundschuh
          </td>
          <td>2024-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Aneuploidy, the presence of an aberrant number of chromosomes, has been associated with tumorigenesis for over a century. More recently, advances in karyotyping techniques have revealed its high prevalence in cancer: About 90% of solid tumors and 50–70% of hematopoietic cancers exhibit chromosome gains or losses. When analyzed at the level of specific chromosomes, there are strong patterns that are observed in cancer karyotypes both pan-cancer and for specific cancer types. These specific aneuploidy patterns correlate strongly with outcomes for tumor initiation, progression, metastasis formation, immune evasion and resistance to therapeutic treatment. Despite their prominence, understanding the basis underlying aneuploidy patterns in cancer has been challenging. Advances in genetic engineering and bioinformatic analyses now offer insights into the genetic determinants of aneuploidy pattern selection. Overall, there is substantial evidence that expression changes of particular genes can act as the positive selective forces for adaptation through aneuploidy. Recent findings suggest that multiple genes contribute to the selection of specific aneuploid chromosomes in cancer; however, further research is necessary to identify the most impactful driver genes. Determining the genetic basis and accompanying vulnerabilities of specific aneuploidy patterns is an essential step in selectively targeting these hallmarks of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0123dd290f56b3193001c1283aea1071e6f411ed" target='_blank'>
              Selection forces underlying aneuploidy patterns in cancer
              </a>
            </td>
          <td>
            Tamara C. Klockner, Christopher S. Campbell
          </td>
          <td>2024-06-24</td>
          <td>Molecular & Cellular Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="We adapted a previously developed targeted single-nucleus DNA sequencing (snDNA-seq) method and constructed a new suite of computational analysis tools to study 137,491 single-nucleus DNA libraries from 24 pancreatic cancers collected under a variety of clinical scenarios including early and late diagnoses, different metastatic sites and before- and after-treatment. We refined the mutational landscape of pancreatic cancer by capturing events missed by bulk sequencing, and validated the evolution pattern of early fixation of driver single-nucleotide variants (SNVs) followed by generation of intratumoral heterogeneity for copy number variations (CNVs). Intertumoral convergent evolution was common, including subclonal inactivation of TGF-β pathway by mutating various components of it; intratumoral convergence was rarely observed, likely due to strong selective force in pancreatic cancer development. Continuous evolution was frequently seen manifesting as CNVs. In the context of non-targeted treatments, no particular pattern was found across metastases or through treatment. In six pancreatic cancers with germline BRCA2 mutation, we discovered varied timing of biallelic inactivation of BRCA2, which sculpted different evolutionary trajectories and could presumably contribute to differential response to treatment. As the first large-scale application of targeted snDNA-seq on pancreatic cancer, this work provides a sample processing and computational analysis pipeline that warrants further clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcb8661871968f256a1ef9033377a1300e9ea64b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A. McIntyre, Akimasa Hayashi, Nicolas Lecomte, Wungki Park, Nan Pang, Eileen M. O'Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41728da5eec57442f344cb37688d002e66c10d7e" target='_blank'>
              Tumor initiation and early tumorigenesis: molecular mechanisms and interventional targets
              </a>
            </td>
          <td>
            Shaosen Zhang, Xinyi Xiao, Yonglin Yi, Xinyu Wang, Lingxuan Zhu, Yanrong Shen, Dongxin Lin, Chen Wu
          </td>
          <td>2024-06-19</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Purpose: Homologous recombination deficiency (HRD) impacts cancer treatment strategies, particularly the effectiveness of PARP inhibitors. However, the variability different HRD assays has hampered the selection of oncology patients who may benefit from these therapies. Our study aims to assess the whole genome landscape to better define HRD in a pan-cancer cohort and to contribute to harmonization of HRD detection. Methods: We employed a whole-genome sequencing WGS HRD classifier that included genome-wide features associated with HRD to analyze 580 tumor/normal paired pan-cancer samples. The HRD results were correlated retrospectively with treatment responses and were compared with commercial HRD tests in a subset of cases. Results: HRD phenotype was identified in 62 samples across various cancers including breast (19%), pancreaticobiliary (17%), gynecological (15%), prostate (8%), upper gastrointestinal (GI) (2%), and other cancers (1%). HRD cases were not confined to BRCA1/2 mutations; 24% of HRD cases were BRCA1/2 wild-type. A diverse range of HRR pathway gene alterations involved in HRD were elucidated, including biallelic mutations in FANCF, XRCC2, and FANCC, and deleterious structural variants. Comparison with results from commercial HRD assays suggests a better performance of WGS to detect HRD, based on treatment response. Conclusion: HRD is a biomarker used to determine which cancer patients would benefit from PARPi and platinum-based chemotherapy. However, a lack of harmonization of tests to determine HRD status makes it challenging to interpret their results. Our study highlights the use of comprehensive WGS analysis to predict HRD in a pan-cancer cohort, elucidates new genomic mechanisms associated with HRD, and enables an accurate identification of this phenotype, paving the way for improved outcomes in oncology care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b80d347a4838be16d195accf2ef681565a4b6929" target='_blank'>
              Whole Genome Landscape Analysis of Homologous Recombination Deficiency in a Pan-Cancer Cohort
              </a>
            </td>
          <td>
            MD Majd Al Assaad, PhD Kevin Hadi, MS Max F. Levine, MD Daniela Guevara, Mph Minal Patel, MS Marvel Tranquille, BS Abigail King, BS John Otilano, MS Alissa Semaan, PhD Gunes Gundem, MS Juan S. Medina-Martínez, BS Michael Sigouros, Mph Jyothi Manohar, BS Hui-Hsuan Kuo, PhD David C. Wilkes, MD Eleni Andreopoulou, MD Eloise Chapman-Davis, MD Scott T. Tagawa, PhD Andrea Sboner, MD Allyson J. Ocean, MD Manish Shah, PhD Elli Papaemmanuil, Cora N. Sternberg, MD Kevin Holcomb, MD David M. Nanus, PhD Olivier Elemento, Juan Miguel Mosquera, J. Medina-Martinez
          </td>
          <td>2024-06-30</td>
          <td>None</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer chimeric, or fusion, transcripts are thought to most frequently appear due to chromosomal aberrations that combine moieties of unrelated normal genes. When being expressed, this results in chimeric RNAs having upstream and downstream parts relatively to the breakpoint position for the 5’- and 3’-fusion components, respectively. As many other types of cancer mutations, fusion genes can be of either driver or passenger type. The driver fusions may have pivotal roles in malignisation by regulating survival, growth, and proliferation of tumor cells, whereas the passenger fusions most likely have no specific function in cancer. The majority of research on fusion gene formation events is concentrated on identifying fusion proteins through chimeric transcripts. However, contemporary studies evidence that fusion events involving non-coding RNA (ncRNA) genes may also have strong oncogenic potential. In this review we highlight most frequent classes of ncRNAs fusions and summarize current understanding of their functional roles. In many cases, cancer ncRNA fusion can result in altered concentration of the non-coding RNA itself, or it can promote protein expression from the protein-coding fusion moiety. Differential splicing, in turn, can enrich the repertoire of cancer chimeric transcripts, e.g. as observed for the fusions of circular RNAs and long non-coding RNAs. These and other ncRNA fusions are being increasingly recognized as cancer biomarkers and even potential therapeutic targets. Finally, we discuss the use of ncRNA fusion genes in the context of cancer detection and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ec4b199299758974fdcbbc48ab668c481680ff0" target='_blank'>
              Cancer fusion transcripts with human non-coding RNAs
              </a>
            </td>
          <td>
            Tharaa Mohammad, Marianna A. Zolotovskaia, Maria V. Suntsova, Anton Buzdin
          </td>
          <td>2024-06-11</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54a1de3c94a3135d7ef2bd2539530b5e0e7ee5f8" target='_blank'>
              DNA mismatch and damage patterns revealed by single-molecule sequencing.
              </a>
            </td>
          <td>
            Mei Hong Liu, Benjamin M Costa, Emilia C Bianchini, Una Choi, Rachel C. Bandler, E. Lassen, Marta Grońska-Pęski, Adam Schwing, Z. R. Murphy, Daniel Rosenkjær, Shany Picciotto, V. Bianchi, L. Stengs, M. Edwards, N. Nunes, Caitlin A. Loh, Tina K Truong, Randall E Brand, Tomi Pastinen, J. R. Wagner, A. Skytte, U. Tabori, J. Shoag, G. Evrony
          </td>
          <td>2024-06-12</td>
          <td>Nature</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="The analysis of DNA sequence outcomes provides molecular insights into double-strand break (DSB) repair mechanisms. By employing parallel in-pool profiling of Cas9-induced indels within a genome-wide knockout library, we present a comprehensive catalog detailing how virtually every human gene influences the DSB repair process. This REPAIRome resource is validated through the identification of novel mechanisms, pathways and factors involved in DSB repair, including unexpected opposing roles for XLF and PAXX in DNA end processing, a molecular explanation for Cas9-induced multi-nucleotide insertions, the identification of HLTF as a DSB-repair factor, the involvement of the SAGA complex in microhomology-mediated end joining, and importantly, an indel mutational signature linked to VHL loss, renal carcinoma and hypoxia. Collectively, these results exemplify the potential of REPAIRome to drive future discoveries in DSB repair, CRISPR-Cas gene editing and the etiology of cancer mutational signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/821f1075e49379e32d3660c5c28ffcf24b062dbf" target='_blank'>
              A comprehensive genetic catalog of human double-strand break repair
              </a>
            </td>
          <td>
            Ernesto López de Alba, Israel Salguero, D. Giménez-Llorente, Ángel Fernández-Sanromán, Ester Casajús-Pelegay, José Terrón-Bautista, Jonathan Barroso-González, J. Bernal, Geoffrey Macintyre, R. Fernández-Leiro, Ana Losada, Felipe Cortés-Ledesma
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/553f4560eb657fca2fe86385e460c71427df430c" target='_blank'>
              GLI1 confers resistance to PARP inhibitors by activating the DNA damage repair pathway.
              </a>
            </td>
          <td>
            Hiroshi Ikeuchi, Yusuke Matsuno, Rika Kusumoto-Matsuo, S. Kojima, Toshihide Ueno, Masachika Ikegami, Rina Kitada, Hitomi Sumiyoshi-Okuma, Yuki Kojima, K. Yonemori, Y. Yatabe, K. Takamochi, Kenji Suzuki, Ken‐ichi Yoshioka, H. Mano, S. Kohsaka
          </td>
          <td>2024-08-03</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Ovarian cancer is the most lethal gynecological malignancy and is often associated with both DNA repair deficiency and extensive metabolic reprogramming. While still emerging, the interplay between these pathways can affect ovarian cancer phenotypes, including therapeutic resistance to the DNA damaging agents that are standard-of-care for this tumor type. In this review, we will discuss what is currently known about cellular metabolic rewiring in ovarian cancer that may impact DNA damage and repair in addition to highlighting how specific DNA repair proteins also promote metabolic changes. We will also discuss relevant data from other cancers that could be used to inform ovarian cancer therapeutic strategies. Changes in the choice of DNA repair mechanism adopted by ovarian cancer are a major factor in promoting therapeutic resistance. Therefore, the impact of metabolic reprogramming on DNA repair mechanisms in ovarian cancer has major clinical implications for targeted combination therapies for the treatment of this devastating disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d7a4f6e6a58cd9429369d6c2946e615d6a364e5" target='_blank'>
              Interplay between altered metabolism and DNA damage and repair in ovarian cancer.
              </a>
            </td>
          <td>
            Apoorva Uboveja, K. Aird
          </td>
          <td>2024-06-14</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="DNA damage-sensing (DDS) and DNA damage-repair (DDR) mechanisms are essential for the fidelity of genetic information transmission. Failure to accomplish an effective DDS/DDR mechanism can lead to cell death or otherwise to cell transformation and cancer development. microRNAs (miRs) are short noncoding RNAs that primarily function as micromanagers of gene expression. Herein, we aimed to investigate the links between miRs and the translation of specific mRNAs encoding proteins involved in genomic DDS and DDR and to screen drugs that have high binding affinity to the selected miRs, which may serve as cancer therapeutics. To accomplish these aims, we used a variety of computational methods spanning data analysis, molecular modeling, and simulation tools (i.e., PyRx, Biopython, ViennaRNA, RNAComposer, AutoDock Vina, OpenBabel, PyMOL, Discovery Studio, MarvinSketch). The genes and miRNAs involved in the DDS and DDR mechanisms were retrieved from either the literature or various online databases (e.g., miRDB). miR data were further cleaned and prepared using scripts, and various libraries were used to obtain their 3-D structures. Genes interacting with miRs were enriched based on multiple database annotations using Enrichr KG. Then, we used docking analyses to virtually screen compounds to serve as ligands for the miRs. Finally, we generated gene-disease-miR-drug networks to study the linkages between the compounds and miR molecules under investigation. For the first time, we were able to identify five compounds that could be repurposed for downregulating miRs that are linked to inhibition of translation of mRNA involved in the DDS and DDR processes. The gathered candidate drugs can be useful for preventing cell transformation and cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04982a355813b15d80368bb235c815920aa1fe9d" target='_blank'>
              Drugs targeting microRNAs that regulate DNA damage sensing and repair mechanisms: A computational drug discovery approach
              </a>
            </td>
          <td>
            S. Ravindranath, Ramirez, Gomez Mejiba S.E, S. Gomez Mejiba, ●. Ravindranath, ●. Ramirez, ●. G. Mejiba
          </td>
          <td>2024-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Alterations in the FMS-like tyrosine kinase 3 (FLT3) gene are the most frequent driver mutations in acute myeloid leukaemia (AML), linked to a high risk of relapse in patients with internal tandem duplications (FLT3-ITD). Tyrosine kinase inhibitors (TKIs) targeting the FLT3 protein are approved for clinical use, yet resistance often emerges. This resistance is mainly seen following the acquisition of additional point mutations in the tyrosine kinase domain (TKD), resulting in a double mutant FLT3-ITD/TKD, which sustains cell signalling and survival despite the presence of FLT3 inhibitors. Here, we developed a FLT3-mutant AML model with adaptive resistance to type II TKIs, sorafenib, and quizartinib by in vitro drug selection. Through global multiomic profiling, we identified upregulation of proteins involved in reactive oxygen species (ROS) production, particularly NADPH-oxidases, driving cellular ’ROS-addiction’, with resistant cells relying on ROS for survival, and genome fidelity preserved by ATM-driven DNA repair. Transcriptomic analysis of adult and paediatric AML (pAML) patients identified high ATM expression as a biomarker for shorter median overall survival in both the de novo and relapsed settings. Inhibition of ATM with clinically relevant therapy WSD-0628 effectively killed TKI- and chemotherapy-resistant AML cells in vitro and significantly extended the survival of mice with sorafenib- and quizartinib-resistant FLT3-ITD AML in vivo. We propose a new treatment strategy to improve survival of patients who develop resistance to sorafenib and quizartinib, as well as relapsed and refractory pAML, exploiting resistance mechanisms to precision therapies and cell-intrinsic features of high-risk cases, highlighting a clinically relevant salvage strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0bcd4e64b15b98143cfc995f3b3d32e9c623b66" target='_blank'>
              Adaptive resistance to FLT3 inhibitors is potentiated by ROS-driven DNA repair signalling
              </a>
            </td>
          <td>
            Dilana E. Staudt, Zacary P. Germon, Abdul Mannan, Tabitha Mclachlan, Heather C Murray, Ryan J. Duchatel, Bryce C. Thomas, Tyrone S Beitaki, Holly P. McEwen, Mika L Persson, Leah Calvert, Izac J. Findlay, Evangeline R. Jackson, Nathan D. Smith, D. Skerrett‐Byrne, David Mossman, Brett Nixon, Geoffry De Iullis, Alicia M. Douglas, A. Enjeti, Jonathan R. Sillar, Janis Chamberlain, Frank Alvaro, Andrew H. Wei, Patrick Connerty, N. Verrills, Matthew D. Dun
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Stable isotope-resolved metabolomics delineates reprogrammed intersecting metabolic networks in human cancers. Knowledge gained from in vivo patient studies provides the "benchmark" for cancer models to recapitulate. It is particularly difficult to model patients' tumor microenvironment (TME) with its complex cell-cell/cell-matrix interactions, which shapes metabolic reprogramming crucial to cancer development/drug resistance. Patient-derived organotypic tissue cultures (PD-OTCs) represent a unique model that retains an individual patient's TME. PD-OTCs of non-small-cell lung cancer better recapitulated the in vivo metabolic reprogramming of patient tumors than the patient-derived tumor xenograft (PDTX), while enabling interrogation of immunometabolic response to modulators and TME-dependent resistance development. Patient-derived organoids (PDOs) are also good models for reconstituting TME-dependent metabolic reprogramming and for evaluating therapeutic responses. Single-cell based 'omics on combinations of PD-OTC and PDO models will afford an unprecedented understanding on TME dependence of human cancer metabolic reprogramming, which should translate into the identification of novel metabolic targets for regulating TME interactions and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6597c82613b1bff5869b53a6356a5e51a2359cc" target='_blank'>
              Metabolic Reprogramming in Human Cancer Patients and Patient-Derived Models.
              </a>
            </td>
          <td>
            T. W. Fan, R. Higashi, A. Lane
          </td>
          <td>2024-07-15</td>
          <td>Cold Spring Harbor perspectives in medicine</td>
          <td>1</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b62bbc42b475a59548fce65b7db966d5dba12012" target='_blank'>
              CRISPRi-TnSeq maps genome-wide interactions between essential and non-essential genes in bacteria.
              </a>
            </td>
          <td>
            Bimal Jana, Xue Liu, Julien Dénéréaz, Hongshik Park, Dmitry Leshchiner, Bruce Liu, Clément Gallay, Junhao Zhu, J. Veening, Tim van Opijnen
          </td>
          <td>2024-07-19</td>
          <td>Nature microbiology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Drug resistance is currently one of the biggest challenges in cancer treatment. With the deepening understanding of drug resistance, various mechanisms have been revealed, including metabolic reprogramming and alterations of redox balance. Notably, metabolic reprogramming mediates the survival of tumor cells in harsh environments, thereby promoting the development of drug resistance. In addition, the changes during metabolic pattern shift trigger reactive oxygen species (ROS) production, which in turn regulates cellular metabolism, DNA repair, cell death, and drug metabolism in direct or indirect ways to influence the sensitivity of tumors to therapies. Therefore, the intersection of metabolism and ROS profoundly affects tumor drug resistance, and clarifying the entangled mechanisms may be beneficial for developing drugs and treatment methods to thwart drug resistance. In this review, we will summarize the regulatory mechanism of redox and metabolism on tumor drug resistance and highlight recent therapeutic strategies targeting metabolic–redox circuits, including dietary interventions, novel chemosynthetic drugs, drug combination regimens, and novel drug delivery systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77cc6698aebc91736fc846603c2b274976902d8b" target='_blank'>
              Targeting Metabolic–Redox Nexus to Regulate Drug Resistance: From Mechanism to Tumor Therapy
              </a>
            </td>
          <td>
            Yuke Wang, Jingqiu He, Shan Lian, Yan Zeng, Sheng He, Jue Xu, Li Luo, Wenyong Yang, Jingwen Jiang
          </td>
          <td>2024-07-01</td>
          <td>Antioxidants</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Cellular senescence in response to ionizing radiation (IR) limits the replication of damaged cells by causing permanent cell cycle arrest. However, IR can induce pro-survival signaling pathways that reduce the extent of radiation-induced cytotoxicity and promote the development of radioresistance. The differential incorporation of histone variant H2A.J has profound effects on higher-order chromatin organization and on establishing the epigenetic state of radiation-induced senescence. However, the precise epigenetic mechanism and function of H2A.J overexpression in response to IR exposure still needs to be elucidated. Methods: Primary (no target, NT) and genetically modified fibroblasts overexpressing H2A.J (H2A.J-OE) were exposed to 20 Gy and analyzed 2 weeks post-IR for radiation-induced senescence by immunohistochemistry and immunofluorescence microscopy. Transcriptome signatures were analyzed in (non-)irradiated NT and H2A.J-OE fibroblasts by RNA sequencing. Since H2A.J plays an important role in the epidermal homeostasis of human skin, the oncogenic potential of H2A.J was investigated in cutaneous squamous cell carcinoma (cSCC). The tissue microarrays of cSCC were analyzed for H2A.J protein expression pattern by automated image analysis. Results: In response to radiation-induced DNA damage, the overexpression of H2A.J impairs the formation of senescence-associated heterochromatin foci (SAHF), thereby inhibiting the SAHF-mediated silencing of proliferation-promoting genes. The dysregulated activation of cyclins and cyclin-dependent kinases disturbs cell cycle arrest in irradiated H2A.J-OE fibroblasts, thereby overcoming radiation-induced senescence. Comparative transcriptome analysis revealed significantly increased WNT16 signaling in H2A.J OE fibroblasts after IR exposure, promoting the fundamental mechanisms of tumor development and progression, including the activation of the epithelial–mesenchymal transition. The quantitative analysis of cSCCs revealed that undifferentiated tumors are associated with high nuclear H2A.J expression, related with greater oncogenic potential. Conclusion: H2A.J overexpression induces radioresistance and promotes oncogenic transformation through the activation of WNT16 signaling pathway functions. H2A.J-associated signatures may improve risk stratification by identifying patients with more aggressive cSCC who may require radiotherapy with increased doses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddd6da202cfd599b89d0d68e33ff36d005d1e9a9" target='_blank'>
              Studies on Human Cultured Fibroblasts and Cutaneous Squamous Cell Carcinomas Suggest That Overexpression of Histone Variant H2A.J Promotes Radioresistance and Oncogenic Transformation
              </a>
            </td>
          <td>
            Benjamin M Freyter, Mutaz A. Abd Al-razaq, Markus Hecht, Christian Rübe, C. Rübe
          </td>
          <td>2024-06-27</td>
          <td>Genes</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Introduction Chemoresistance constitutes a prevalent factor that significantly impacts thesurvival of patients undergoing treatment for smal-cell lung cancer (SCLC). Chemotherapy resistance in SCLC patients is generally classified as primary or acquired resistance, each governedby distinct mechanisms that remain inadequately researched. Methods In this study, we performed transcriptome screening of peripheral blood plasma obtainedfrom 17 patients before and after receiving combined etoposide and platinum treatment. We firs testimated pseudo-single-cell analysis using xCell and ESTIMATE and identified differentially expressed genes (DEGs), then performed network analysis to discover key hub genes involved in chemotherapy resistance. Results Our analysis showed a significant increase in class-switched memory B cell scores acrossboth chemotherapy resistance patterns, indicating their potential crucial role in mediatingresistance. Moreover, network analysis identifed PRICKLE3, TNFSFI0, ACSLl and EP300 as potential contributors to primary resistance, with SNWl, SENP2 and SMNDCl emerging assignificant factors in acquired resistance, providing valuable insights into chemotherapy resistancein SCLC. Discussion These findings offer valuable insights for understanding chemotherapy resistance and related gene signatures in SCLC, which could help further biological validation studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e543ce8522adc904b0c191dca81d34b4464ae11" target='_blank'>
              Integrative analysis of blood transcriptome profiles in small-cell lung cancer patients for identification of novel chemotherapy resistance-related biomarkers
              </a>
            </td>
          <td>
            Fang Yang, Jinhua Fan, Runxiang Yang, Yupeng Cun
          </td>
          <td>2024-06-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Triple‐negative breast cancer (TNBC) is often considered one of the most aggressive subtypes of breast cancer, characterized by a high recurrence rate and low overall survival (OS). It is notorious for posing challenges related to drug resistance. While there has been progress in TNBC research, the mechanisms underlying chemotherapy resistance in TNBC remain largely elusive. We collect single‐cell RNA sequencing (scRNA‐seq) data from five TNBC patients susceptible to chemotherapy and five resistant cases. Comprehensive analyses involving copy number variation (CNV), pseudotime trajectory, cell–cell interactions, pseudospace analysis, as well as transcription factor and functional enrichment are conducted specifically on macrophages and malignant cells. Furthermore, we performed validation experiments on clinical samples using multiplex immunofluorescence. We identified a subset of SPP1+ macrophages that secrete SPP1 signals interacting with CD44 on malignant cell surfaces, potentially activating the PDE3B pathway within malignant cells via the integrin pathway, leading to chemotherapy resistance. The abnormally enhanced SPP1 signal between macrophages and malignant cells may serve as a factor promoting chemotherapy resistance in TNBC patients. Therefore, SPP1+ macrophages could potentially serve as a therapeutic target to reduce chemotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8af0989421bdfb2401a61fa8d2dab5197b650cb7" target='_blank'>
              Single‐cell transcriptome sequencing reveals SPP1‐CD44‐mediated macrophage–tumor cell interactions drive chemoresistance in TNBC
              </a>
            </td>
          <td>
            Fuzhong Liu, Junfeng Zhang, Xiaowei Gu, Qiuyang Guo, Wenjia Guo
          </td>
          <td>2024-07-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA methylation is a key epigenetic regulatory mechanism that plays a critical role in a variety of cellular processes, including the regulation of cell fate during development, maintenance of cell identity, and genome stability. DNA methylation is tightly regulated by enzymatic reactions and its deregulation plays an important role in the development of cancer. Specific DNA methylation alterations have been found in pediatric solid tumors, providing new insights into the development of these tumors. In addition, DNA methylation profiles have greatly contributed to tune the diagnosis of pediatric solid tumors and to define subgroups of patients with different risks of progression, leading to the reduction in unwanted toxicity and the improvement of treatment efficacy. This review highlights the dysregulated DNA methylome in pediatric solid tumors and how this information provides promising targets for epigenetic therapies, particularly inhibitors of DNMT enzymes (DNMTis). Opportunities and limitations are considered, including the ability of DNMTis to induce viral mimicry and immune signaling by tumors. Besides intrinsic action against cancer cells, DNMTis have the potential to sensitize immune-cold tumors to immunotherapies and may represent a remarkable option to improve the treatment of challenging pediatric solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b1eac857dfcc5a5e3b672ddb17d6097e48bc70f" target='_blank'>
              Targeting DNA Methylation Machinery in Pediatric Solid Tumors
              </a>
            </td>
          <td>
            C. Cristalli, Katia Scotlandi
          </td>
          <td>2024-07-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The use of k-mers to capture genetic variation in bacterial genome-wide association studies (bGWAS) has demonstrated its effectiveness in overcoming the plasticity of bacterial genomes by providing a comprehensive array of genetic variants in a genome set that is not confined to a single reference genome. However, little attempt has been made to interpret k-mers in the context of genome rearrangements, partly due to challenges in the exhaustive and high-throughput identification of genome structure and individual rearrangement events. Here, we present GWarrange, a pre- and post-bGWAS processing methodology that leverages the unique properties of k-mers to facilitate bGWAS for genome rearrangements. Repeat sequences are common instigators of genome rearrangements through intragenomic homologous recombination, and they are commonly found at rearrangement boundaries. Using whole-genome sequences, repeat sequences are replaced by short placeholder sequences, allowing the regions flanking repeats to be incorporated into relatively short k-mers. Then, locations of flanking regions in significant k-mers are mapped back to complete genome sequences to visualise genome rearrangements. Four case studies based on two bacterial species (Bordetella pertussis and Enterococcus faecium) and a simulated genome set are presented to demonstrate the ability to identify phenotype-associated rearrangements. GWarrange is available at https://github.com/DorothyTamYiLing/GWarrange.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c42162bc2b2f883e8a6834a9dc952dc10d3392ba" target='_blank'>
              GWarrange: a pre- and post- genome-wide association studies pipeline for detecting phenotype-associated genome rearrangement events.
              </a>
            </td>
          <td>
            Yi Ling Tam, Sarah Cameron, Andrew Preston, Lauren A. Cowley
          </td>
          <td>2024-07-01</td>
          <td>Microbial genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b2c5c7d1cda920e03f84144f36e08d5e8bdd9f" target='_blank'>
              Genomic barcoding for clonal diversity monitoring and control in cell-based complex antibody production
              </a>
            </td>
          <td>
            Niels Bauer, Christoph Oberist, Michaela Poth, Julian Stingele, Oliver Popp, Simon Ausländer
          </td>
          <td>2024-06-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af8a5e84090feb0d13451624ef0b08cb6aa245a3" target='_blank'>
              Epigenetic regulation of DNA repair gene program by Hippo/YAP1-TET1 axis mediates sorafenib resistance in HCC
              </a>
            </td>
          <td>
            Chunli Mo, Weixin You, Yipeng Rao, Zhenping Lin, Shuai Wang, Ting He, Huanming Shen, Xun Li, Rui Zhang, Boan Li
          </td>
          <td>2024-07-05</td>
          <td>Cellular and Molecular Life Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Exploring the molecular mechanisms of PD-1/PDL-1 blockade for non-small cell lung cancer (NSCLC) would facilitate understanding for tumor microenvironment (TME) and development of individualized medicine. To date, biomarkers of response to PD-1 blockade therapy were still limited. In this study, we hypothesize that cell type in the tumor microenvironment can influence the effect of PD-1 blockade immunotherapy through specific genes. Therefore, we re-analyze the single-cell RNA sequencing data and validation in tissue from lung adenocarcinoma patients. Dynamic changes of cellular subpopulation were observed after anti-PD-1 immunotherapy among TMEs between primary/metastasis or good/poor response patients. Non-exhausted CD8 T cells and dysregulated genes were observed in responsing patients from PD-1 blockade therapy. Among all changed genes, JUN, involved in PD-1 blockade immunotherapy pathway, and could be considered as a PD-1 responsing biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98469e74353589b5bc39ceef749a7ebfa0a0d298" target='_blank'>
              Identification of JUN gene and cellular microenvironment in response to PD-1 blockade treatment in lung cancer patients via single-cell RNA sequencing
              </a>
            </td>
          <td>
            Yuxuan Wang, Tao Ran, Yunke Li, Lei Tian, Lifeng Yang, Zhidong Liu, Biao Yao
          </td>
          <td>2024-06-13</td>
          <td>Aging (Albany NY)</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Accurately quantifying the functional consequences of non-coding mosaic variants requires the pairing of DNA sequence with both accessible and closed chromatin architectures along individual DNA molecules—a pairing that cannot be achieved using traditional fragmentation-based chromatin assays. We demonstrate that targeted single-molecule chromatin fiber sequencing (Fiber-seq) achieves this, permitting single-molecule, long-read genomic and epigenomic profiling across targeted >100 kilobase loci with ∼10-fold enrichment over untargeted sequencing. Targeted Fiber-seq reveals that pathogenic expansions of the DMPK CTG repeat that underlie Myotonic Dystrophy 1 are characterized by somatic instability and disruption of multiple nearby regulatory elements, both of which are repeat length-dependent. Furthermore, we reveal that therapeutic adenine base editing of the segmentally duplicated γ-globin (HBG1/HBG2) promoters in primary human hematopoietic cells induced towards an erythroblast lineage increases the accessibility of the HBG1 promoter as well as neighboring regulatory elements. Overall, we find that these non-protein coding mosaic variants can have complex impacts on chromatin architectures, including extending beyond the regulatory element harboring the variant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9447591d5b0bc6ff15f592902f9149ea8d6bbf6" target='_blank'>
              Resolving the chromatin impact of mosaic variants with targeted Fiber-seq
              </a>
            </td>
          <td>
            Stephanie C. Bohaczuk, Zachary J. Amador, Chang Li, Ben Mallory, Elliott G. Swanson, Jane Ranchalis, Mitchell R. Vollger, Katherine M. Munson, Tom Walsh, Morgan O. Hamm, Yizi Mao, Andre Lieber, A. Stergachis
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background and aim. Lung cancer remains one of the most threatening malignancies, ranking as the second most diagnosed cancer, and it continues to be the leading cause of cancer-related deaths worldwide. Challenges persist with late diagnosis and the high mutational burden characteristic of lung cancer. 
Methods. Our study focuses on identifying the mutational spectrum of a cohort of advanced-stage non-small cell lung cancer (NSCLC) patients using a minimally invasive method through blood collection. To analyze the mutational landscape of these patients, we employed plasma DNA for the next-generation sequencing (NGS) cancer panel Ion Torrent, which contains 50 of the most mutated genes in lung cancer. All protocols for extraction, quality and quantity control, and library preparation follow the manufacturer’s rules. Bioinformatics analysis was performed to select pathogenic mutations versus non-pathogenic-benign ones. 
Results. This approach is particularly valuable for patients in advanced stages (III and IV, n=10) of lung adenocarcinoma and lung squamous cell carcinoma, who lack surgical options and limited therapeutic avenues. The comprehensive sequencing analysis revealed that nine of the ten lung cancer patients carried a TP53 mutation. Also, several other mutations exist in various cases, showing heterogeneous profiling. 
Conclusions. Our findings demonstrate the potential of liquid biopsies in providing crucial genetic insights that can guide personalized treatment strategies, improving the management and outcomes for patients with advanced lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6844d155d0da77d29ae773874578536a65b3932" target='_blank'>
              Next-generation sequencing as a valuable tool for mutational spectrum in advanced-stage NSCLC patients
              </a>
            </td>
          <td>
            Ioana Iurca, Ecaterina Isakescu, L. Pop, Liviuţa Budişan, R. Pîrlog, A. Haranguș, T. Ciuleanu, C. Braicu, I. Berindan-Neagoe
          </td>
          <td>2024-07-10</td>
          <td>Medicine and Pharmacy Reports</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Kirsten Rat Sarcoma (KRAS) is the most commonly mutated oncogene in colorectal carcinoma (CRC). We have previously reported the interactions between microsatellite instability (MSI), DNA promoter methylation, and gene expression. In this study, we looked for associations between KRAS mutation, gene expression, and methylation that may help with precision medicine. Genome-wide gene expression and DNA methylation were done in paired CRC tumor and surrounding healthy tissues. The results suggested that (a) the magnitude of dysregulation of many major gene pathways in CRC was significantly greater in patients with the KRAS mutation, (b) the up- and down-regulation of these dysregulated gene pathways could be correlated with the corresponding hypo- and hyper-methylation, and (c) the up-regulation of CDKN2A was more pronounced in tumors with the KRAS mutation. A recent cell line study showed that there were higher CDKN2A levels in 5-FU-resistant CRC cells and that these could be down-regulated by Villosol. Our findings suggest the possibility of a better response to anti-CDKN2A therapy with Villosol in KRAS-mutant CRC. Also, the more marked up-regulation of genes in the proteasome pathway in CRC tissue, especially with the KRAS mutation and MSI, may suggest a potential role of a proteasome inhibitor (bortezomib, carfilzomib, or ixazomib) in selected CRC patients if necessary.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40b3e427194ea41aedea0dfcb0cf1b6c9f955b47" target='_blank'>
              Association of KRAS Mutation and Gene Pathways in Colorectal Carcinoma: A Transcriptome- and Methylome-Wide Study and Potential Implications for Therapy
              </a>
            </td>
          <td>
            F. Jasmine, Armando Almazan, Yuliia Khamkevych, Marc Bissonnette, Habibul Ahsan, M. Kibriya
          </td>
          <td>2024-07-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b517632791bed91e0fed3fb023edfd9ccfe184b7" target='_blank'>
              Deciphering the Role of Exosomal Non-Coding RNA (ncRNA) in Drug Resistance of Gastrointestinal Tumors; an Updated Review.
              </a>
            </td>
          <td>
            Nafiseh Tashakori, Maryam Armanfar, Anahita Mashhadi, Adil Tawfeeq Mohammed, Manal Morad Karim, Abbas Hameed Abdul Hussein, Mohaned Adil, Sajad Ataei Azimi, Fatemeh Abedini
          </td>
          <td>2024-06-15</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Metastasis is the primary cause of cancer-related death, with the dissemination and colonization of primary tumor cells at the metastatic site facilitated by various molecules and complex pathways. Understanding the biological mechanisms underlying the metastatic process is critical for the development of effective interventions. Several epigenetic modifications have been identified that play critical roles in regulating cancer metastasis. This review aims to provide a comprehensive summary of recent advances in understanding the role of epigenetic modifiers, including histone modifications, DNA methylation, non-coding RNAs, enhancer reprogramming, chromatin accessibility, and N6-methyladenosine, in metastasis-associated processes, such as epithelial-mesenchymal transition (EMT), cancer cell migration, and invasion. In particular, this review provides a detailed and in-depth description of the role of crosstalk between epigenetic regulators in tumor metastasis. Additionally, we explored the potential and limitations of epigenetics-related target molecules in the diagnosis, treatment, and prognosis of cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1902f2a43039c2e646ddaffdb896714416a5a63" target='_blank'>
              Epigenetic Modifiers in Cancer Metastasis
              </a>
            </td>
          <td>
            Die Hu, Tianci Zhao, Chenxing Xu, Xinyi Pan, Zhengyu Zhou, Shengjie Wang
          </td>
          <td>2024-07-27</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Precision (individualized) medicine relies on the molecular profiling of tumors’ dysregulated characteristics (genomic, epigenetic, transcriptomic) to identify the reliance on key pathways (including genome stability and epigenetic gene regulation) for viability or growth, and then utilises targeted therapeutics to disrupt these survival-dependent pathways. Non-mutational epigenetic changes alter cells’ transcriptional profile and are a key feature found in many tumors. In contrast to genetic mutations, epigenetic changes are reversable, and restoring a normal epigenetic profile can inhibit tumor growth and progression. Lysine acetyltransferases (KATs or HATs) protect genome stability and integrity, and Tip60 is an essential acetyltransferase due to its roles as an epigenetic and transcriptional regulator, and as master regulator of the DNA double-strand break response. Tip60 is commonly downregulated and mislocalized in many cancers, and the roles that mislocalized Tip60 plays in cancer are not well understood. Here we categorize and discuss Tip60-regulated genes, evaluate Tip60-interacting proteins based on cellular localization, and explore the therapeutic potential of Tip60-targeting compounds as epigenetic inhibitors. Understanding the multiple roles Tip60 plays in tumorigenesis will improve our understanding of tumor progression and will inform therapeutic options, including informing potential combinatorial regimes with current chemotherapeutics, leading to improvements in patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0558cedd75f278d6feed6575b276aad74d694cf" target='_blank'>
              Evaluating the Cellular Roles of the Lysine Acetyltransferase Tip60 in Cancer: A Multi-Action Molecular Target for Precision Oncology
              </a>
            </td>
          <td>
            Nazanin Zohourian, Erin Coll, Muiread Dever, Anna Sheahan, Petra Burns-Lane, James A. L. Brown
          </td>
          <td>2024-07-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc05d0e416d1c5836cac6aabe3bd3a577d1440ba" target='_blank'>
              Enhanced eMAGE applied to identify genetic factors of nuclear hormone receptor dysfunction via combinatorial gene editing
              </a>
            </td>
          <td>
            Peter N Ciaccia, Zhuobin Liang, Anabel Y. Schweitzer, Eli Metzner, Farren J. Isaacs
          </td>
          <td>2024-06-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) is a form of a circular double-stranded DNA that exists independently of conventional chromosomes. eccDNA exhibits a broad and random distribution across eukaryotic cells and has been associated with tumor-related properties due to its ability to harbor the complete gene information of oncogenes. The complex and multifaceted mechanisms underlying eccDNA formation include pathways such as DNA damage repair, breakage–fusion–bridge (BFB) mechanisms, chromothripsis, and cell apoptosis. Of note, eccDNA plays a pivotal role in tumor development, genetic heterogeneity, and therapeutic resistance. The high copy number and transcriptional activity of oncogenes carried by eccDNA contribute to the accelerated growth of tumors. Notably, the amplification of oncogenes on eccDNA is implicated in the malignant progression of cancer cells. The improvement of high-throughput sequencing techniques has greatly enhanced our knowledge of eccDNA by allowing for a detailed examination of its genetic structures and functions. However, we still lack a comprehensive and efficient annotation for eccDNA, while challenges persist in the study and understanding of the functional role of eccDNA, emphasizing the need for the development of robust methodologies. The potential clinical applications of eccDNA, such as its role as a measurable biomarker or therapeutic target in diseases, particularly within the spectrum of human malignancies, is a promising field for future research. In conclusion, eccDNA represents a quite dynamic and multifunctional genetic entity with far-reaching implications in cancer pathogenesis and beyond. Further research is essential to unravel the molecular pathways of eccDNA formation, elucidate its functional roles, and explore its clinical applications. Addressing these aspects is crucial for advancing our understanding of genomic instability and developing novel strategies for tailored therapeutics, especially in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eba9d45916b5521fe06e0dfbffaa25ccf1f330f0" target='_blank'>
              Beyond the Chromosome: Recent Developments in Decoding the Significance of Extrachromosomal Circular DNA (eccDNA) in Human Malignancies
              </a>
            </td>
          <td>
            P. Tsiakanikas, Konstantina Athanasopoulou, Ioanna A. Darioti, Vasiliki Taxiarchoula Agiassoti, Stamatis Theocharis, Andreas Scorilas, P. Adamopoulos
          </td>
          <td>2024-07-24</td>
          <td>Life</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79f8be9a3b2a2fa9867435314cacdcbe80a7eb4d" target='_blank'>
              Multiple mechanisms contribute to acquired TRAIL resistance in multiple myeloma
              </a>
            </td>
          <td>
            Fany V Ticona-Pérez, Xi Chen, A. Pandiella, Elena Díaz-Rodríguez
          </td>
          <td>2024-08-05</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3169b366c8dfc0a14136eb7bf3ab2054f7cc2f88" target='_blank'>
              CRISPR-Cas9 screens reveal common essential miRNAs in human cancer cell lines
              </a>
            </td>
          <td>
            D. Merk, L. Paul, F. Tsiami, H. Hohenthanner, Ghazal Mohseni Kouchesfahani, L. Haeusser, B. Walter, A. Brown, N. Persky, D. Root, G. Tabatabai
          </td>
          <td>2024-06-17</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Multiple Myeloma (MM) is a highly prevalent and incurable form of cancer that arises from malignant plasma cells, with over 35,000 new cases diagnosed annually in the United States. While there are a growing number of approved therapies, MM remains incurable and nearly all patients will relapse and exhaust all available treatment options. Mechanisms for disease progression are unclear and in particular, little is known regarding the role of long non-coding RNAs (lncRNA) in mediating disease progression and response to treatment. In this study, we used transcriptome sequencing to compare newly diagnosed MM patients who had short progression- free survival (PFS) to standard first-line treatment (PFS < 24 months) to patients who had prolonged PFS (PFS > 24 months). We identified 157 differentially upregulated lncRNAs with short PFS and focused our efforts on characterizing the most upregulated lncRNA, LINC01432. We investigated LINC01432 overexpression and CRISPR/Cas9 knockdown in MM cell lines to show that LINC01432 overexpression significantly increases cell viability and reduces apoptosis, while knockdown significantly reduces viability and increases apoptosis, supporting the clinical relevance of this lncRNA. Next, we used individual-nucleotide resolution cross-linking immunoprecipitation with RT-qPCR to show that LINC01432 directly interacts with the RNA binding protein, CELF2. Lastly, we showed that LINC01432-targeted locked nucleic acid antisense oligonucleotides reduce viability and increases apoptosis. In summary, this fundamental study identified lncRNAs associated with short PFS to standard NDMM treatment and further characterized LINC01432, which inhibits apoptosis. Key points: lncRNA expression was found to be dysregulated in patients with short PFS to standard multiple myeloma therapy. LINC01432-bound CELF2 inhibits apoptosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/847cba66ef27ec89828bb55803b54ea81c4c449e" target='_blank'>
              LINC01432 binds to CELF2 in newly diagnosed multiple myeloma promoting short progression-free survival to standard therapy
              </a>
            </td>
          <td>
            Richa Mishra, Prasanth Thunuguntla, Alani Perkin, Dhanusha Duraiyan, Katelyn Bagwill, Savannah Gonzales, Vanessa Brizuela, Steve Daly, Yoon Jae Chang, Mahdote Abebe, Yash Rajana, Kelly Wichmann, Catheryn Bolick, Jaiyana King, M. Fiala, J. Fortier, R. Jayasinghe, Mark Schroeder, Li Ding, Ravi Vij, Jessica M. Silva-Fisher
          </td>
          <td>2024-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Objective The escape from T cell-mediated immune surveillance is an important cause of death for patients with acute myeloid leukemia (AML). This study aims to identify clonal heterogeneity in leukemia progenitor cells and explore molecular or signaling pathways associated with AML immune escape. Methods Single-cell RNA sequencing (scRNA-seq) was performed to identified AML-related cellular subsets, and intercellular communication was analyzed to investigate molecular mechanisms associated with AML immune escape. Bulk RNA sequencing (RNA-seq) was performed to screen differentially expressed genes (DEGs) related to hematopoietic stem cell progenitors (HSC-Prog) in AML, and critical ore signaling pathways and hub genes were found by Gene Set Enrichment Analysis (GSEA), Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) enrichment analysis. The mRNA level of the hub gene was verified using quantitative real-time PCR (qRT-PCR) and the protein level of human leukocyte antigen A (HLA-A) using enzyme-linked immuno sorbent assay (ELISA). Results scRNA-seq analysis revealed a large heterogeneity of HSC-Prog across samples, and the intercellular communication analysis indicated a strong association between HSC-Prog and CD8+-T cells, and HSC-Prog also had an association with HLA-A. Transcriptome analysis identified 1748 DEGs, enrichment analysis results showed that non-classical wnt signaling pathway was associated with AML, and 4 pathway-related genes (RHOA, RYK, CSNK1D, NLK) were obtained. After qRT-PCR and ELISA validation, hub genes and HLA-A were found to be down-regulated in AML and up-regulated after activation of the non-classical Wnt signaling pathway. Conclusion In this study, clonal heterogeneity of HSC-Prog cells in AML was identified, non-classical wnt signaling pathways associated with AML were identified, and it was verified that HLA-A could be upregulated by activation of non-classical wnt signaling, thereby increasing antigen presentation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f50ef877e5f9629d2799ca7878c873b260c4a74" target='_blank'>
              Activation of non-classical Wnt signaling pathway effectively enhances HLA-A presentation in acute myeloid leukemia
              </a>
            </td>
          <td>
            YuHan Ma, Jun-shuai Yue, Ling Gao, Jingxin Zhou, Wei Chen, Jing Su, JinRong Yao, QiaoMei Shi, XiaoDong Zhao, Na Hu
          </td>
          <td>2024-06-19</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Linking disease phenotypes with molecular targets is key to the rational design of treatment interventions. Resistance to the chemotherapeutic cisplatin is one of the major factors limiting the clinical utility of this therapy, which is central to the treatment of a variety of solid malignancies. In this study, we couple the upregulation of a chemoresistant factor, the glutamate-cystine antiporter SLC7A11, with the addiction of cisplatin-resistant cancer cells to extracellular glutamine. In doing so, we thus provide a putative biomarker for this acquired metabolic dependency of chemoresistance. Subsequently, we evaluate various therapeutic strategies to selectively kill SLC7A11high cisplatin-resistant cancer cells, identifying cross-resistance to ferroptosis-inducing compounds and hypersensitivity to glutaminase inhibitor CB-839. We identify enzymatic depletion of extracellular glutamine using the long-standing anti-leukemic therapy asparaginase (ASNase), which possesses glutaminase activity, as a potential approach, and show this can be successfully combined with cisplatin in cell models. In summary, this study mechanistically links an acquired metabolic dependency of chemoresistant cancer cells with a putative biomarker and provides a potentially actionable strategy to target these drug resistant cells warranting further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32c836fb51ec4a12788e118211188bb2a1ae2ab5" target='_blank'>
              Glutamine-addiction in cisplatin resistant cancer cells is mediated by SLC7A11 and can be targeted with asparaginase therapy
              </a>
            </td>
          <td>
            Jiantao Wang, Robert Strauss, Jiri Bartek, S. G. Rudd
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) possess a significant ability to renew themselves, which gives them a strong capacity to form tumors and expand to encompass additional body areas. In addition, they possess inherent resistance to chemotherapy and radiation therapies used to treat many forms of cancer. Scientists have focused on investigating the signaling pathways that are highly linked to the ability of CSCs to renew themselves and maintain their stem cell properties. The pathways encompassed are Notch, Wnt/β-catenin, hedgehog, STAT3, NF-κB, PI-3K/Akt/mTOR, sirtuin, ALDH, MDM2, and ROS. Recent studies indicate that directing efforts towards CSC cells is essential in eradicating the overall cancer cell population and reducing the likelihood of tumor metastasis. As our comprehension of the mechanisms that stimulate CSC activity, growth, and resistance to chemotherapy advances, the discovery of therapeutic drugs specifically targeting CSCs, such as small-molecule compounds, holds the potential to revolutionize cancer therapy. This review article examines and analyzes the novel anti-CSC compounds that have demonstrated effective and selective targeting of pathways associated with the renewal and stemness of CSCs. We also discussed their special drug metabolism and absorption mechanisms. CSCs have been the subject of much study in cancer biology. As a possible treatment for malignancies, small-molecule drugs that target CSCs are gaining more and more attention. This article provides a comprehensive review of the current state of key small-molecule compounds, summarizes their recent developments, and anticipates the future discovery of even more potent and targeted compounds, opening up new avenues for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c2ec47520bdf5746027b1d46298064477df151" target='_blank'>
              Novel Anti-Cancer Stem Cell Compounds: A Comprehensive Review
              </a>
            </td>
          <td>
            Shanchun Guo, Shilong Zheng, Mingli Liu, Guangdi Wang
          </td>
          <td>2024-08-01</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Most human cells contain two non-identical genomes, and differences in their regulation underlie human development and disease. We demonstrate that Fiber-seq Inferred Regulatory Elements (FIREs) enable the accurate quantification of chromatin accessibility across the 6 Gbp diploid human genome with single-molecule and single-nucleotide precision. We find that cells can harbor >1,000 regulatory elements with haplotype-selective chromatin accessibility (HSCA) and show that these elements preferentially localize to genomic loci containing the most human genetic diversity, with the human leukocyte antigen (HLA) locus showing the largest amount of HSCA genome-wide in immune cells. Furthermore, we uncover HSCA elements with sequence non-deterministic chromatin accessibility, representing likely somatic epimutations, and show that productive transcription from the inactive X chromosome is buttressed by clustered promoter-proximal elements that escape X chromosome inactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c0c60a592276d18a9f211d3827524a5d632d729" target='_blank'>
              A haplotype-resolved view of human gene regulation
              </a>
            </td>
          <td>
            Mitchell R. Vollger, Elliott G. Swanson, Shane J. Neph, Jane Ranchalis, Katherine M. Munson, Ching-Huang Ho, Adriana E. Sedeño-Cortés, William E. Fondrie, Stephanie C. Bohaczuk, Yizi Mao, Nancy L. Parmalee, Ben Mallory, William T. Harvey, Younjun Kwon, Gage H. Garcia, Kendra Hoekzema, Jeffrey G. Meyer, Mine Cicek, Evan E. Eichler, W. S. Noble, Daniela M. Witten, James T. Bennett, John P. Ray, A. Stergachis
          </td>
          <td>2024-06-15</td>
          <td>bioRxiv</td>
          <td>3</td>
          <td>29</td>
        </tr>

        <tr id="The mutation rate is a pivotal biological characteristic, intricately governed by natural selection and historically garnering considerable attention. Recent advances in high-throughput sequencing and analytical methodologies have profoundly transformed our understanding in this domain, ushering in an unprecedented era of mutation rate research. This paper aims to provide a comprehensive overview of the key concepts and methodologies frequently employed in the study of mutation rates. It examines various types of mutations, explores the evolutionary dynamics and associated theories, and synthesizes both classical and contemporary hypotheses. Furthermore, this review comprehensively explores recent advances in understanding germline and somatic mutations in animals and offers an overview of experimental methodologies, mutational patterns, molecular mechanisms, and driving forces influencing variations in mutation rates across species and tissues. Finally, it proposes several potential research directions and pressing questions for future investigations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae50b3aef439b195f59303f4114cfc56c351eb28" target='_blank'>
              A new era of mutation rate analyses: Concepts and methods
              </a>
            </td>
          <td>
            Kun Wu, Danqi Qin, Yang Qian, Haoxuan Liu
          </td>
          <td>2024-07-18</td>
          <td>Zoological Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Upregulation of the Wilms' tumour 1 (WT1) gene is common in acute myeloid leukaemia (AML) and is associated with poor prognosis. WT1 generates 12 primary transcripts through different translation initiation sites and alternative splicing. The short WT1 transcripts express abundantly in primary leukaemia samples. We observed that overexpression of short WT1 transcripts lacking exon 5 with and without the KTS motif (sWT1+/- and sWT1-/-) led to reduced cell growth. However, only sWT1+/- overexpression resulted in decreased CD71 expression, G1 arrest, and cytarabine resistance. Primary AML patient cells with low CD71 expression exhibit resistance to cytarabine, suggesting that CD71 may serve as a potential biomarker for chemotherapy. RNAseq differential expressed gene analysis identified two transcription factors, HOXA3 and GATA2, that are specifically upregulated in sWT1+/- cells, whereas CDKN1A is upregulated in sWT1-/- cells. Overexpression of either HOXA3 or GATA2 reproduced the effects of sWT1+/-, including decreased cell growth, G1 arrest, reduced CD71 expression and cytarabine resistance. HOXA3 expression correlates with chemotherapy response and overall survival in NPM1 mutation-negative leukaemia specimens. Overexpression of HOXA3 leads to drug resistance against a broad spectrum of chemotherapeutic agents. Our results suggest that WT1 regulates cell proliferation and drug sensitivity in an isoform-specific manner.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/747999aa3c419c4737033b97641dd45130ab9db4" target='_blank'>
              Upregulation of HOXA3 by isoform-specific Wilms tumour 1 drives chemotherapy resistance in acute myeloid leukaemia.
              </a>
            </td>
          <td>
            Basil Allen, Lindsey Savoy, P. Ryabinin, D. Bottomly, Reid Chen, Bonnie Goff, Anthony Wang, Shannon K McWeeney, Haijiao Zhang
          </td>
          <td>2024-06-12</td>
          <td>British journal of haematology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Introduction Field cancerization is suggested to arise from imbalanced differentiation in individual basal progenitor cells leading to clonal expansion of mutant cells that eventually replace the epithelium, although without evidence. Methods We performed deep sequencing analyses to characterize the genomic and transcriptomic landscapes of field change in two patients with synchronous aerodigestive tract tumors. Results Our data support the emergence of numerous genetic alterations in cancer-associated genes but refutes the hypothesis that founder mutation(s) underpin this phenomenon. Mutational signature analysis identified defective homologous recombination as a common underlying mutational process unique to synchronous tumors. Discussion Our analyses suggest a common etiologic factor defined by mutational signatures and/or transcriptomic convergence, which could provide a therapeutic opportunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae797c96e298d92f976b952e9d1ccd0f24f2a970" target='_blank'>
              Transcriptomic convergence despite genomic divergence drive field cancerization in synchronous squamous tumors
              </a>
            </td>
          <td>
            Qiu Xuan Tan, Nicholas B. Shannon, Weng Khong Lim, J. X. Teo, Daniel R. Y. Yap, Sze Min Lek, Joey W. S. Tan, Shih Jia, Josephine Hendrikson, Ying Liu, Gillian Ng, Clara Y. L. Chong, Wanyu Guo, Kelvin K. N. Koh, C. Ng, Vikneswari Rajasegaran, Jolene S.M. Wong, Chin Jin Seo, C. Ong, Tony K.H. Lim, Bin Tean Teh, Oi Lian Kon, C. Chia, Khee Chee, N. G. Iyer, C. Ong, M. Shammas, Sarah Mazzilli, Prajish Iyer
          </td>
          <td>2024-06-13</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Introduction Acute myeloid leukemia (AML) is an aggressive blood cancer with high heterogeneity and poor prognosis. Although the metabolic reprogramming of nicotinamide adenine dinucleotide (NAD) has been reported to play a pivotal role in the pathogenesis of acute myeloid leukemia (AML), the prognostic value of NAD metabolism and its correlation with the immune microenvironment in AML remains unclear. Methods We utilized our large-scale RNA-seq data on 655 patients with AML and the NAD metabolism-related genes to establish a prognostic NAD metabolism score based on the sparse regression analysis. The signature was validated across three independent datasets including a total of 1,215 AML patients. ssGSEA and ESTIMATE algorithms were employed to dissect the tumor immune microenvironment. Ex vivo drug screening and in vitro experimental validation were performed to identify potential therapeutic approaches for the high-risk patients. In vitro knockdown and functional experiments were employed to investigate the role of SLC25A51, a mitochondrial NAD+ transporter gene implicated in the signature. Results An 8-gene NAD metabolism signature (NADM8) was generated and demonstrated a robust prognostic value in more than 1,800 patients with AML. High NADM8 score could efficiently discriminate AML patients with adverse clinical characteristics and genetic lesions and serve as an independent factor predicting a poor prognosis. Immune microenvironment analysis revealed significant enrichment of distinct tumor-infiltrating immune cells and activation of immune checkpoints in patients with high NADM8 scores, acting as a potential biomarker for immune response evaluation in AML. Furthermore, ex vivo drug screening and in vitro experimental validation in a panel of 9 AML cell lines demonstrated that the patients with high NADM8 scores were more sensitive to the PI3K inhibitor, GDC-0914. Finally, functional experiments also substantiated the critical pathogenic role of the SLC25A51 in AML, which could be a promising therapeutic target. Conclusion Our study demonstrated that NAD metabolism-related signature can facilitate risk stratification and prognosis prediction in AML and guide therapeutic decisions including both immunotherapy and targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966bc9d8486254b1e8db50812645261d71323173" target='_blank'>
              NAD metabolism-related genes provide prognostic value and potential therapeutic insights for acute myeloid leukemia
              </a>
            </td>
          <td>
            Yuncan Cao, Wenjing Shu, Peng Jin, Jianfeng Li, Hong-Ming Zhu, Xin-jie Chen, Yongmei Zhu, Xi Huang, Wenyan Cheng, Yang Shen
          </td>
          <td>2024-06-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) presents a variety of druggable genetic alterations that revolutionized the treatment approaches. However, identifying new alterations may broaden the group of patients benefitting from such novel treatment options. Recently, the interest focused on the neuregulin-1 gene (NRG1), whose fusions may have become a potential predictive factor. To date, the occurrence of NRG1 fusions has been considered a negative prognostic marker in NSCLC treatment; however, many premises remain behind the targetability of signaling pathways affected by the NRG1 gene. The role of NRG1 fusions in ErbB-mediated cell proliferation especially seems to be considered as a main target of treatment. Hence, NSCLC patients harboring NRG1 fusions may benefit from targeted therapies such as pan-HER family inhibitors, which have shown efficacy in previous studies in various cancers, and anti-HER monoclonal antibodies. Considering the increased interest in the NRG1 gene as a potential clinical target, in the following review, we highlight its biology, as well as the potential clinical implications that were evaluated in clinics or remained under consideration in clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/302eaed4f1e7bdac2f2c9a0f86b9a6908f0e39da" target='_blank'>
              NRG1 Gene Fusions—What Promise Remains Behind These Rare Genetic Alterations? A Comprehensive Review of Biology, Diagnostic Approaches, and Clinical Implications
              </a>
            </td>
          <td>
            Tomasz Kucharczyk, M. Nicoś, Marek Kucharczyk, E. Kalinka
          </td>
          <td>2024-08-05</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is an aggressive and lethal form of brain cancer with few effective treatments. In this context, Zika virus has emerged as a promising therapeutic agent due to its ability to selectively infect and kill GBM cells. To elucidate these mechanisms and expand the landscape of oncolytic virotherapy, we pursued a transcriptomic meta-analysis comparing the molecular signatures of Zika infection in GBM and neuroblastoma (NBM). Over-representation analysis of dysregulated coding genes showed significant enrichment of tumor necrosis factor (TNF), NF-κB, and p53 signaling pathways. A refined list of long non-coding RNAs consistently dysregulated in Zika-infected GBMs was also developed. Functional review of these candidates revealed their potential regulatory role in Zika-mediated oncolysis. We performed validation of the less-researched targets in adult and pediatric GBM cell lines and found significant differential regulation, as predicted. Altogether, our results provide novel insights into the molecular mechanisms underlying the effect of Zika on GBM. We highlight potential therapeutic targets that could be further interrogated to improve the efficacy of tumor cell death and the utility of Zika as an adjuvant virotherapy for GBM and other related cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d5fb2f82f19cb82c54f2e078dc5fd330417c5a7" target='_blank'>
              Transcriptomic Meta-analysis Identifies Long Non-Coding RNAs Mediating Zika’s Oncolytic Impact in Glioblastoma Multiforme
              </a>
            </td>
          <td>
            Youssef A. Kousa, Shriya Singh, Allison Horvath, Federica Tomasso, Javad Nazarian, Lisa Henderson, Tamer A. Mansour
          </td>
          <td>2024-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e256414c851623f11f78259f789000632e7ca061" target='_blank'>
              Epigenetic modification of PHLDA2 is associated with tumor microenvironment and unfavorable outcome of immune checkpoint inhibitor-based therapies in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Junjie Zhao, Xiuyi Pan, Zilin Wang, Yuntian Chen, Dingbang Liu, Yu Shen, Xinyuan Wei, Chenhao Xu, Xingming Zhang, Xu Hu, Junru Chen, Jinge Zhao, Bo Tang, G. Sun, P. Shen, Zhenhua Liu, H. Zeng, Jiayu Liang
          </td>
          <td>2024-07-20</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="DNA damage-inducible transcript 3 (DDIT3) is a transcription factor central to apoptosis, differentiation, and stress response. DDIT3 has been extensively studied in cancer biology. However, its precise implications in breast cancer progression and its interaction with the immune microenvironment are unclear. In this study, we utilized a novel multi-omics integration strategy, combining bulk RNA sequencing, single-cell sequencing, spatial transcriptomics and immunohistochemistry, to explore the role of DDIT3 in breast cancer and establish the correlation between DDIT3 and poor prognosis in breast cancer patients. We identified a robust prognostic signature, including six genes (unc-93 homolog B1, TLR signaling regulator, anti-Mullerian hormone, DCTP pyrophosphatase 1, mitochondrial ribosomal protein L36, nuclear factor erythroid 2, and Rho GTPase activating protein 39), associated with DDIT3. This signature stratified the high-risk patient groups, characterized by increased infiltration of the regulatory T cells and M2-like macrophages and fibroblast growth factor (FGF)/FGF receptor signaling activation. Notably, the high-risk patient group demonstrated enhanced sensitivity to immunotherapy, presenting novel therapeutic opportunities. Integrating multi-omics data helped determine the spatial expression pattern of DDIT3 in the tumor microenvironment and its correlation with immune cell infiltration. This multi-dimensional analysis provided a comprehensive understanding of the intricate interplay between DDIT3 and the immune microenvironment in breast cancer. Overall, our study not only facilitates understanding the role of DDIT3 in breast cancer but also offers innovative insights for developing prognostic models and therapeutic strategies. Identifying the DDIT3-related prognostic signature and its association with the immune microenvironment provided a promising avenue for personalized breast cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95de9c824f2784b988fb174126a6ea54d393ac3" target='_blank'>
              DDIT3 is associated with breast cancer prognosis and immune microenvironment: an integrative bioinformatic and immunohistochemical analysis
              </a>
            </td>
          <td>
            Xin Yu, Wenge Li, Shengrong Sun, Juanjuan Li
          </td>
          <td>2024-05-28</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/354d0cada1a73a94d156b70d215246eda0c2a524" target='_blank'>
              T cell dysfunction and therapeutic intervention in cancer.
              </a>
            </td>
          <td>
            Caitlin C. Zebley, D. Zehn, Stephen Gottshalk, Hongbo Chi
          </td>
          <td>2024-07-18</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Cells must adapt to environmental changes to maintain homeostasis. One of the most striking environmental adaptations is entry into hibernation during which core body temperature can decrease from 37°C to as low at 4°C. How mammalian cells, which evolved to optimally function within a narrow range of temperatures, adapt to this profound decrease in temperature remains poorly understood. In this study, we conducted the first genome-scale CRISPR-Cas9 screen in cells derived from Syrian hamster, a facultative hibernator, as well as human cells to investigate the genetic basis of cold tolerance in a hibernator and a non-hibernator in an unbiased manner. Both screens independently revealed glutathione peroxidase 4 (GPX4), a selenium-containing enzyme, and associated proteins as critical for cold tolerance. We utilized genetic and pharmacological approaches to demonstrate that GPX4 is active in the cold and its catalytic activity is required for cold tolerance. Furthermore, we show that the role of GPX4 as a suppressor of cold-induced cell death extends across hibernating species, including 13-lined ground squirrels and greater horseshoe bats, highlighting the evolutionary conservation of this mechanism of cold tolerance. This study identifies GPX4 as a central modulator of mammalian cold tolerance and advances our understanding of the evolved mechanisms by which cells mitigate cold-associated damage—one of the most common challenges faced by cells and organisms in nature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/181b39f570d649216102dc87d265ba81bbf8c6ba" target='_blank'>
              Multi-species genome-wide CRISPR screens identify GPX4 as a conserved suppressor of cold-induced cell death
              </a>
            </td>
          <td>
            Breanna Lam, Kathrin M. Kajderowicz, Heather R. Keys, Julian M. Roessler, Evgeni M. Frenkel, Adina Kirkland, Punam Bisht, Mohamed A. El-Brolosy, Rudolf Jaenisch, George W. Bell, Jonathan S Weissman, Eric C. Griffith, Siniša Hrvatin
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Adenocarcinoma of the lung (LUAD) is a common and highly lethal disease. Clinical grading of disease strongly predicts recurrence and survival after surgery and is determined by morphological assessment of histological growth patterns in resected tumours. The molecular basis of growth pattern is poorly understood at present, as are the mechanisms linking growth pattern to recurrence and death. Interestingly, the two archetypal lethal morphologies, solid and micropapillary patterns, are characterised by their biphasic appearance. Both have an epithelial fraction which is in direct stromal contact, and a fraction which is not. This morphological variance seems likely to represent plasticity, and to be causally linked to mechanisms of virulence. To investigate the gene expression changes related to growth pattern both intra- and intertumoral, we applied spatial transcriptomics (Nanostring GeoMx DSP) to tissue microarray specimens of primary resected human lung adenocarcinoma. Using a variety of region-of-interest (ROI) selection strategies, we sampled 160 pure epithelial ROIs across 7 distinct morphological features of LUAD from 51 patients. Analyses of gene expression reveal fundamental trajectories connecting growth patterns, and crucial modes of plasticity which underly high-risk morphologies. These modes suggest mechanisms for the origins of growth pattern and mechanisms of virulence. Our work highlights dramatic divergence in gene expression programmes between highly lethal but morphologically diverse modes of tumour growth. Furthermore, it provides an explanation for how microscopically localised hypoxia in the primary tumour helps to establish and maintain survival strategies which ultimately determine morphology-specific mechanisms of tumour metastasis, suggesting new therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35a08117cd59bda92ca8c6af39ca3279ca1ec800" target='_blank'>
              Spatial resolution of transcriptomic plasticity states underpinning lethal morphologies in lung adenocarcinoma
              </a>
            </td>
          <td>
            Hannah L Williams, Nicolas Poulain, Ian Powley, Silvia Martinelli, R. Bielik, H. Leslie, Colin Nixon, C. Wilson, M. Sereno, Zhangyi He, Leah Officer-Jones, Fiona Ballantyne, Rachel Pennie, C. Wood, David Y Lewis, Nigel B Jamieson, J. Le Quesne
          </td>
          <td>2024-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="The expression of eukaryotic genes relies on the precise 3’-terminal cleavage and polyadenylation of newly synthesized pre-mRNA transcripts. Defects in these processes have been associated with various diseases, including cancer. While cancer-focused sequencing studies have identified numerous driver mutations in protein-coding sequences, noncoding drivers – particularly those affecting the cis-elements required for pre-mRNA cleavage and polyadenylation – have received less attention. Here, we systematically analysed cancer somatic mutations affecting 3’UTR polyadenylation signals using the Pan-Cancer Analysis of Whole Genomes (PCAWG) dataset. We found a striking enrichment of cancer-specific somatic mutations that disrupt strong and evolutionarily conserved cleavage and polyadenylation signals within tumour suppressor genes. Further bioinformatics and experimental analyses conducted as a part of our study suggest that these mutations have a profound capacity to downregulate the expression of tumour suppressor genes. Thus, this work uncovers a novel class of noncoding somatic mutations with significant potential to drive cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa4b3938c7f84b24956b4223cbd287669eff1caf" target='_blank'>
              Recurrent disruption of tumour suppressor genes in cancer by somatic mutations in cleavage and polyadenylation signals
              </a>
            </td>
          <td>
            Yaroslav A. Kainov, F. Hamid, E. Makeyev
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="NF1 is recurrently mutated in glioblastoma yet the molecular landscape and efficacy of targeted therapies remain unclear. Here, we combine bulk and single cell genomics of human somatic NF1 mutant, IDH-wildtype glioblastomas with functional genomic analysis of cell lines and mouse intracranial tumor models to identify molecular subgroups within NF1 mutant glioblastomas and mechanisms underlying MEK inhibitor response. Targeted DNA sequencing showed homozygous deletion of the cell cycle regulator CDKN2A/B is a poor prognostic marker in somatic NF1 mutant, but not NF1 wildtype, tumors. DNA methylation array profiling revealed three epigenetic groups highlighted by distinct clinical features, co-mutation patterns, and reference methylation classifier identities. Genome-wide CRISPRi screens in glioblastoma cells revealed cell cycle regulators are conserved mediators of cell growth while response to the MEK inhibitor selumetinib converges on Ras/RAF/MEK pathway activation. Repression of the RAF regulator SHOC2 sensitizes glioblastomas to selumetinib in vitro and in vivo in mouse intracranial glioblastoma models. Single cell RNA-sequencing of mouse intracranial glioblastomas treated with the MEK inhibitor selumetinib reveals distinct responses between mesenchymal-like (MES-like) and non MES-like subpopulations suggesting non-MES like cells are intrinsically resistant to MEK inhibition. Finally, single nuclear RNA-sequencing (snRNA-seq) of human NF1 mutant, CDKN2A/B deleted glioblastomas reveals MES-like tumor cells are associated with selumetinib sensitivity signatures while non MES-like cells exhibit increased cell cycle progression and lack selumetinib sensitivity, further supporting the notion that MEK inhibition specifically targets MES-like tumor cell subpopulations. Taken together, our data underscores the heterogeneity between and within somatic NF1 mutant glioblastomas and delineates mechanisms of MEK inhibitor response across distinct tumor subpopulations, guiding the development of future therapeutic strategies that may synergize with MEK inhibition for NF1 mutant tumors. The tumor suppressor NF1 is mutated in 15% of glioblastomas,1–3 the most common malignant brain tumor with poor outcomes and few effective treatments.4 NF1 is a GTPase activating protein (GAP) that negatively regulates Ras, and thus, NF1 loss leads to induction of Ras/RAF/MEK/ERK signaling, driving tumorigenesis and comprising a targetable molecular cascade.5,6 Genomic analysis of glioblastoma demonstrates NF1 mutation is associated with a mesenchymal-like (MES-like) transcriptomic tumor cell subpopulation and altered tumor microenvironment.7,8 More broadly, DNA methylation analysis reveals multiple epigenetic subgroups with overlapping relationships to transcriptomic subtype and DNA alterations, underscoring the complex relationship between genetic drivers and molecular signatures.9 While the updated 2021 Central Nervous System WHO tumor classification incorporates an ever increasing amount of molecular criteria for diffuse astrocytic tumors,10 the existence and clinical significance of molecular subgroups within somatic NF1 mutant, IDH-wildtype glioblastomas based on genetic co-mutations, epigenetic profile, or transcriptomic signatures remain unclear. Preclinical data support the utility of MEK inhibition in NF1 mutant gliomas,11,12 and the MEK inhibitor selumetinib is FDA approved for tumors arising in patients with syndromic neurofibromatosis type 1 (NF-1) harboring a germline NF1 mutation.13,14 In NF-1 associated gliomas, MEK inhibition demonstrates efficacy in a limited case series,15 and combined BRAF/MEK inhibition shows efficacy in BRAF p.V600E mutant gliomas,16 further supporting the translational potential of Ras/Raf/MEK/ERK blockade within genetically defined glioma subtypes. Nevertheless, treatment resistance to molecular monotherapy remains a challenge,17–20 and the mechanisms underlying MEK inhibitor resistance in NF1 mutant glioma are unknown. Here, we integrate targeted DNA sequencing, DNA methylation profiling, and single nuclear RNA-sequencing (snRNA-seq) of human patient somatic NF1 mutant, IDH-wildtype glioblastomas with single cell RNA-sequencing (scRNA-seq), genome-wide clustered regularly interspaced short palindromic repeats interference (CRISPRi) screens, and pharmacologic studies in cell lines and mouse intracranial glioblastoma models to define molecular subgroups and functional mediators of MEK inhibitor response. Targeted DNA sequencing of NF1 mutant, IDH-wildtype glioblastomas (n=186 tumors) revealed CDKN2A/B deletion was associated with poor outcomes in NF1 mutant, but not NF1 wildtype, glioblastomas. DNA methylation profiling (n=129 tumors) demonstrated three epigenetic subgroups with distinct clinical features, co- mutation patterns across cell cycle genes, and reference methylation classifier identities. Genome-wide CRISPRi screens in mouse SB28 and human GBM43 glioblastoma cells identified a conserved cell cycle gene network mediating cell growth, consistent with the clinical importance of additional hits affecting the cell cycle in human somatic NF1 mutant glioblastomas. Moreover, genome-wide mediators of selumetinib response converged upon two Ras pathway effectors mediating selumetinib sensitivity: BRAF and SHOC2. SHOC2 repression in glioblastoma cells significantly improved selumetinib response both in vitro and in intracranial allografts in vivo. Single cell RNA-sequencing (scRNA-seq) of mouse intracranial glioblastomas treated with the MEK inhibitor selumetinib revealed MES-like tumor cells correlated with CDKN2A retention and the CRISPRi screen selumetinib sensitivity signature, with selumetinib resistant cells displaying Ras pathway induction. In contrast, non-MES like tumor cells were CDKN2A deficient and lacked expression of the CRISPRi screen selumetinib sensitivity signature, with selumetinib resistant cells inducing a glial de-differentiation program. Finally, snRNA-seq of NF1 mutant, CDKN2A/B deleted, IDH-wildtype glioblastomas (n=9) showed non MES-like tumor cells exhibit increased cell cycle progression and were not associated with the CRISPRi screen selumetinib sensitivity signature. MES-like tumor cells within newly diagnosed, but not recurrent, tumors retained expression of the CRISPRi screen selumetinib sensitivity signature, suggesting resistance can arise both between and within specific transcriptomic glioblastoma cell tumor cell subpopulations. Taken together, our data identifies clinically important subgroups of NF1 mutant, IDH-wildtype glioblastomas and supports a model in which heterogeneity between tumors and within tumor cell subpopulations underlies MEK inhibitor response, supporting the need for additional synergistic therapeutic approaches beyond maximal Ras pathway blockade for NF1 mutant glioblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71e53f09e755435876ce84b544a3051562b27f2f" target='_blank'>
              Multiplatform molecular profiling and functional genomic screens identify prognostic signatures and mechanisms underlying MEK inhibitor response in somatic NF1 mutant glioblastoma
              </a>
            </td>
          <td>
            Sixuan Pan, Kanish Mirchia, E. Payne, S. J. Liu, Nadeem N Al-Adli, Zain Peeran, Poojan Shukla, Jacob S. Young, Rohit Gupta, Jasper Wu, J. Pak, K. Seo, T. Ozawa, Brian Na, Alyssa T. Reddy, S. Braunstein, Joanna J. Phillips, Susan Chang, D. A. Solomon, Arie Perry, D. Raleigh, Mitchell S. Berger, A. Abate, H. Vasudevan
          </td>
          <td>2024-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="As precision medicine such as targeted therapy and immunotherapy often have limited accessibility, low response rate, and evolved resistance, it is urgent to develop simple, low-cost, and quick-turnaround personalized diagnostic technologies for drug response prediction with high sensitivity, speed, and accuracy. The major challenges of drug response prediction strategies employing digital database modeling are the scarcity of labeled clinical data, applicability only to a few classes of drugs, and losing the resolution at the individual patient level. Although these challenges have been partially addressed by large-scale cancer cell line datasets and more patient-relevant cell-based systems, the integration of different data types and data translation from pre-clinical to clinical utilities are still far-fetched. To overcome the current limitations of precision medicine with a clinically proven drug response prediction assay, we have developed an innovative and proprietary technology based on in vitro patient testing and in silico data analytics. First, a patient-derived gene expression signature was established via the transcriptomic profiling of cell-free mRNA (cfmRNA) from the patient’s blood. Second, a gene-to-drug data fusion and overlaying mechanism to transfer data were performed. Finally, a semi-supervised method was used for the database searching, matching, annotation, and ranking of drug efficacies from a pool of ~700 approved, investigational, or clinical trial drug candidates. A personalized drug response report can be delivered to inform clinical decisions within a week. The PGA (patient-derived gene expression-informed anticancer drug efficacy) test has significantly improved patient outcomes when compared to the treatment plans without PGA support. The implementation of PGA, which combines patient-unique cfmRNA fingerprints with drug mapping power, has the potential to identify treatment options when patients are no longer responding to therapy and when standard-of-care is exhausted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a4d56cd8aea96d0d0aeb07e90e09da931c495cc" target='_blank'>
              A Transformative Technology Linking Patient’s mRNA Expression Profile to Anticancer Drug Efficacy
              </a>
            </td>
          <td>
            Chen Yeh, Shu-Ti Lin, Hung-Chih Lai
          </td>
          <td>2024-07-14</td>
          <td>Onco</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mutually exclusive loss-of-function alterations in gene pairs are those that occur together less frequently than may be expected and may denote a synthetically lethal relationship (SSL) between the genes. SSLs can be exploited therapeutically to selectively kill cancer cells. Here, we analysed mutation, copy number variation, and methylation levels in samples from The Cancer Genome Atlas, using the hypergeometric and the Poisson binomial tests to identify mutually exclusive inactivated genes. We focused on gene pairs where one is an inactivated tumour suppressor and the other a gene whose protein product can be inhibited by known drugs. This provided an abundance of potential targeted therapeutics and repositioning opportunities for several cancers. These data are available on the MexDrugs website, https://bioinformaticslab.sussex.ac.uk/mexdrugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7250b34d6949747e0dee284668cf7df5c1a39253" target='_blank'>
              Identifying actionable synthetically lethal cancer gene pairs using mutual exclusivity.
              </a>
            </td>
          <td>
            Sarah K. Wooller, Laurence H. Pearl, Frances M. G. Pearl
          </td>
          <td>2024-07-08</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="DNA methylation is traditionally associated with gene silencing, but its causal relationship and role in shaping cell fate decisions still need to be fully elucidated. Here, we conducted a genome-wide analysis to investigate the relationship between DNA methylation and gene expression at gene regulatory regions in human immune cells. By utilizing CRISPR-dCas9 DNA methylation editing tools, we successfully established a cause-and-effect relationship between the methylation levels of the promoter of the Interleukin1-receptor antagonist (IL1RN) gene and its expression. Notably, we observed that modifying the DNA methylation status of the IL1RN promoter is sufficient to alter the acquisition of the human myeloid cell fate and change the cellular response to inflammatory stimuli, resulting in abnormal cytokine release and distinctive capacity to support cancer growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0466661c0180cda161896839ec2c88a8975d7da9" target='_blank'>
              Modulating immune cell fate and inflammation through CRISPR-mediated DNA methylation editing
              </a>
            </td>
          <td>
            Gemma Valcárcel, Anna V. López-Rubio, Aleksey Lazarenkov, Clara Berenguer, Josep Calafell, J. Rodríguez-Ubreva, E. Ballestar, J. L. Sardina
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Studies on somatic mutations in cloned animals have revealed slight genetic variances between clones and their originals, but have yet to identify the precise effects of these differences within the organism. Somatic mutations contribute to aging and are implicated in tumor development and other age-related diseases. Thus, we compared whole genome sequencing data from an original dog with that of cloned dogs, identifying candidate somatic mutations that were disproportionately located within genes previously implicated in aging. The substitutional signature of cloning-specific somatic mutations mirrored the uniform distribution characteristic of the signature associated with human aging. Further analysis of genes revealed significant enrichment of traits associated with body size as well as the molecular mechanisms underlying neuronal function and tumorigenesis. Overall, the somatic mutations found in cloned dogs may indicate a conserved mechanism driving aging across species and a broad spectrum of pathway alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f3d14c854be2c4eda75c0ca77517be9684eaeca" target='_blank'>
              Genome-Wide Characterization of Somatic Mutation Patterns in Cloned Dogs Reveals Implications for Neuronal Function, Tumorigenesis, and Aging
              </a>
            </td>
          <td>
            Seung-Wan Woo, Miju Kim, Dayeon Kang, Y. Choe, Seong-Ju Oh, Are-Sun You, S. Lee, Jaemin Kim
          </td>
          <td>2024-06-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Long non-coding RNAs (lncRNAs), defined as RNA molecules exceeding 200 nucleotides in length, have been implicated in the regulation of various biological processes and the progression of tumors. Among them, LINC00518, a recently identified lncRNA encoded by a gene located on chromosome 6p24.3, consists of three exons and is predicted to positively regulate the expression of specific genes. LINC00518 has emerged as a key oncogenic lncRNA in multiple cancer types. It exerts its tumor-promoting effects by modulating the expression of several target genes, primarily through acting as a sponge for microRNAs (miRNAs). Additionally, LINC00518 influences critical signaling pathways, including the Wnt/β-catenin, JAK/STAT, and integrin β3/FAK pathways. Elevated levels of LINC00518 in tumor tissues are associated with increased tumor size, advanced clinical stage, metastasis, and poor survival prognosis. This review provides a comprehensive summary of the genetic characteristics, expression patterns, biological functions, and underlying mechanisms of LINC00518 in human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d93f220783bc0f4834eae797d7fdcb653821aa1b" target='_blank'>
              LINC00518: a key player in tumor progression and clinical outcomes
              </a>
            </td>
          <td>
            Qiang Yi, Gangfeng Zhu, Weijian Zhu, Jiaqi Wang, Xinting Ouyang, Kuan Yang, Jinghua Zhong
          </td>
          <td>2024-07-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA modifications drive aging, neurodegeneration, carcinogenesis, and chemotherapy drug action. To understand the functional genomic roles of DNA modifications, it is critical to accurately map their diverse chemical forms with single-nucleotide precision in complex genomes, but it remains challenging. Click-code-seq is a click-chemistry-aided single-nucleotide-resolution strategy for guanine-oxidation mapping, used in yeast DNA but having poor applicability to human genomes. Here, we upgraded click-code-seq to enable its first application for sequencing DNA oxidation and depurination in human genomes. For this, we developed a companion fluorescence assay, click-fluoro-quant, to rapidly quantify different common DNA modifications, and devised novel adapters to minimize false modification detection and assess modification frequency in cell populations. We uncovered that endogenous DNA oxidation in a human cell line has a highly similar pattern to cancer mutational signatures associated with reactive oxygen species. We established that the DNA-alkylating chemotherapy drug irofulven preferentially induces depurination in ApA dimers and promoter regions. Intriguingly, we revealed that oxidized guanines and apurinic sites, both irofulven-induced and endogenous, are depleted in gene transcribed strands, and the strand bias widens with increasing gene expression. This work substantially advances click-code-seq for deciphering the impacts of key modifications in human DNA on cellular physiology and toxicological responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6b6bbbe3beae3b0b23fbadc72da494f15786faa" target='_blank'>
              Click-chemistry-aided quantitation and sequencing of oxidized guanines and apurinic sites uncovers their transcription-linked strand bias in human cells
              </a>
            </td>
          <td>
            V. Takhaveev, N. Püllen, Navnit K. Singh, Sabrina M. Huber, Stefan Schauer, Hailey L. Gahlon, A. Poetsch, S. Sturla
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8c57c70b2a1bb317c90f2cc0de26bf7a8a7418b" target='_blank'>
              A microdeletion event at 19q13.43 in IDH-mutant astrocytomas is strongly correlated with MYC overexpression
              </a>
            </td>
          <td>
            Ege Ülgen, U. Gerlevik, Sila Gerlevik, Yavuz Oktay, O. U. Sezerman, Şevin Turcan, K. Ozduman
          </td>
          <td>2024-06-14</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="RNAs undergo a complex choreography of metabolic processes in human cells that are regulated by thousands of RNA-associated proteins. While the effects of individual RNA-associated proteins on RNA metabolism have been extensively characterized, the full complement of regulators for most RNA metabolic events remain unknown. Here we present a massively parallel RNA-linked CRISPR (ReLiC) screening approach to measure the responses of diverse RNA metabolic events to knockout of 2,092 human genes encoding all known RNA-associated proteins. ReLiC screens highlight modular interactions between gene networks regulating splicing, translation, and decay of mRNAs. When combined with biochemical fractionation of polysomes, ReLiC reveals striking pathway-specific coupling between growth fitness and mRNA translation. Perturbing different components of the translation and proteostasis machineries have distinct effects on ribosome occupancy, while perturbing mRNA transcription leaves ribosome occupancy largely intact. Isoform-selective ReLiC screens capture differential regulation of intron retention and exon skipping by SF3b complex subunits. Chemogenomic screens using ReLiC decipher translational regulators upstream of mRNA decay and uncover a role for the ribosome collision sensor GCN1 during treatment with the anti-leukemic drug homoharringtonine. Our work demonstrates ReLiC as a versatile platform for discovering and dissecting regulatory principles of human RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9d61d936925a83dd1c8b255f5e6a3f00dce8861" target='_blank'>
              Decoding RNA Metabolism by RNA-linked CRISPR Screening in Human Cells
              </a>
            </td>
          <td>
            Patrick J. Nugent, Heungwon Park, Cynthia L. Wladyka, Katharine Y. Chen, Christine Bynum, Grace Quarterman, Andrew C. Hsieh, A. Subramaniam
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Genomic variations contribute to the phenotypic diversity of individuals. A number of polymorphisms in protein-coding regions that alter drug efficacy or lead to adverse reactions have been characterized; however, noncoding regions that affect drug responses are largely overlooked, except for a limited number of well-studied enhancers. Results We conducted a quantitative assessment of cis-regulatory elements (CREs) based on transcription initiation profiling of mRNAs and noncoding RNAs, including enhancer RNAs, by using CAGE (Cap Analysis of Gene Expression). Candidate CREs identified in a hepatocellular carcinoma HepG2 cell line with stable expression of drug-responsive transcription factor pregnane X receptor (PXR) were further narrowed down by integrating data of PXR-binding sites in human primary hepatocytes and genome-wide association studies. We found more than 100-fold enrichments of the candidates to genetically associated loci with circulating levels of bilirubin and vitamin D, which implicated a link to adverse reactions of PXR ligands. We uncovered novel enhancers of UGT1A1 and TSKU through CRISPR/Cas9 knockout experiments. We identified alleles altering regulatory activities of UGT1A1 and CYP24A1enhancers by using luciferase reporter assay. Furthermore, our siRNA experiments revealed an unexpected impact of TSKU on the expression of vitamin D-metabolizing enzymes. Conclusions Our transcriptome-based assessment of CREs expanded the list of drug-inducible and PXR-mediated enhancers and super-enhancers. We identified regulatory alleles that alter drug-induced gene expressions, and discovered a novel molecular cascade associated with an adverse reaction. Our results contribute a precise understanding of the noncoding elements of the human genome underlying drug responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f12494b9c7c48c33bdacd730ac5e83409d9983a" target='_blank'>
              Drug-induced cis-regulatory elements in human hepatocytes affect molecular phenotypes associated with adverse reactions
              </a>
            </td>
          <td>
            Saki Gotoh-Saito, Ryoko Wada, Hideya Kawaji
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer, due to resistance to standard therapies such as endocrine therapy, anti-HER2 therapy and chemotherapy, continues to pose a major health challenge. A growing body of research emphasizes the heterogeneity and plasticity of metabolism in breast cancer. Because differences in subtypes exhibit a bias toward metabolic pathways, targeting mitochondrial inhibitors shows great potential as stand-alone or adjuvant cancer therapies. Multiple therapeutic candidates are currently in various stages of preclinical studies and clinical openings. However, specific inhibitors have been shown to face multiple challenges (e.g., single metabolic therapies, mitochondrial structure and enzymes, etc.), and combining with standard therapies or targeting multiple metabolic pathways may be necessary. In this paper, we review the critical role of mitochondrial metabolic functions, including oxidative phosphorylation (OXPHOS), the tricarboxylic acid cycle, and fatty acid and amino acid metabolism, in metabolic reprogramming of breast cancer cells. In addition, we outline the impact of mitochondrial dysfunction on metabolic pathways in different subtypes of breast cancer and mitochondrial inhibitors targeting different metabolic pathways, aiming to provide additional ideas for the development of mitochondrial inhibitors and to improve the efficacy of existing therapies for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb2c0f886f8dbb22d140ba74ccba6ea13715c88a" target='_blank'>
              Mitochondrial inhibitors: a new horizon in breast cancer therapy
              </a>
            </td>
          <td>
            Yalan Yan, Sijie Li, Lanqian Su, Xinrui Tang, Xiaoyan Chen, Xiang Gu, Guanhu Yang, Hao Chi, Shangke Huang
          </td>
          <td>2024-07-04</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c785685c384edfe64958536f2b96ef40bece9a9d" target='_blank'>
              Let’s make it personal: CRISPR tools in manipulating cell death pathways for cancer treatment
              </a>
            </td>
          <td>
            Mobina Bayat, J. S. Nahand
          </td>
          <td>2024-07-29</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cce12d4cb261e95e6f4ee6a3b370aa98326787b3" target='_blank'>
              Investigating druggable kinases for targeted therapy in retinoblastoma.
              </a>
            </td>
          <td>
            Kumar Jeyaprakash, M. Kumaran, U. Kim, Radhakrishnan Santhi, V. Muthukkaruppan, B. Devarajan, A. Vanniarajan
          </td>
          <td>2024-07-01</td>
          <td>Journal of human genetics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cellular senescence has been increasingly recognized as a hallmark of cancer, reflecting its association with aging and inflammation, its role as a response to deregulated proliferation and oncogenic stress, and its induction by cancer therapies. While therapy-induced senescence (TIS) has been linked to resistance, recurrence, metastasis, and normal tissue toxicity, TIS also has the potential to enhance therapy response and stimulate anti-tumor immunity. In this review, we examine the Jekyll and Hyde nature of senescent cells (SnCs), focusing on how their persistence while expressing the senescence-associated secretory phenotype (SASP) modulates the tumor microenvironment through autocrine and paracrine mechanisms. Through the SASP, SnCs can mediate both resistance and response to cancer therapies. To fulfill the unmet potential of cancer immunotherapy, we consider how SnCs may influence tumor inflammation and serve as an antigen source to potentiate anti-tumor immune response. This new perspective suggests treatment approaches based on TIS to enhance immune checkpoint blockade. Finally, we describe strategies for mitigating the detrimental effects of senescence, such as modulating the SASP or targeting SnC persistence, which may enhance the overall benefits of cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52dbb8efb9179d8a3a7870f442fb59ca7f4eb573" target='_blank'>
              Therapy-Induced Cellular Senescence: Potentiating Tumor Elimination or Driving Cancer Resistance and Recurrence?
              </a>
            </td>
          <td>
            Yue Liu, Isabelle Lomeli, Stephen J. Kron
          </td>
          <td>2024-07-30</td>
          <td>Cells</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c964458f9642633fba6003bd596ed63245c6ffb5" target='_blank'>
              PARP4 interacts with hnRNPM to regulate splicing during lung cancer progression
              </a>
            </td>
          <td>
            Y. Lee, C. Phua, Ju Yuan, Bin Zhang, May Yin Lee, S. Kannan, Yui Hei Jasper Chiu, Casslynn Wei Qian Koh, Choon Kong Yap, E. Lim, Jianbin Chen, Yuhua Lim, Jane Jia Hui Lee, A. Skanderup, Zhenxun Wang, Weiwei Zhai, Nguan Soon Tan, Chandra S Verma, Yvonne Tay, D. Tan, Wai Leong Tam
          </td>
          <td>2024-07-22</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="ABSTRACT Non-heritable, phenotypic drug resistance toward antibiotics challenges antibiotic therapies. Characteristics of such phenotypic resistance have implications for the evolution of heritable resistance. Diverse forms of phenotypic resistance have been described, but phenotypic resistance characteristics remain less explored than genetic resistance. Here, we add novel combinations of single-cell characteristics of phenotypic resistant E. coli cells and compare those to characteristics of susceptible cells of the parental population by exposure to different levels of recurrent ampicillin antibiotic. Contrasting expectations, we did not find commonly described characteristics of phenotypic resistant cells that arrest growth or near growth. We find that under ampicillin exposure, phenotypic resistant cells reduced their growth rate by about 50% compared to growth rates prior to antibiotic exposure. The growth reduction is a delayed alteration to antibiotic exposure, suggesting an induced response and not a stochastic switch or caused by a predetermined state as frequently described. Phenotypic resistant cells exhibiting constant slowed growth survived best under ampicillin exposure and, contrary to expectations, not only fast-growing cells suffered high mortality triggered by ampicillin but also growth-arrested cells. Our findings support diverse modes of phenotypic resistance, and we revealed resistant cell characteristics that have been associated with enhanced genetically fixed resistance evolution, which supports claims of an underappreciated role of phenotypic resistant cells toward genetic resistance evolution. A better understanding of phenotypic resistance will benefit combatting genetic resistance by developing and engulfing effective anti-phenotypic resistance strategies. IMPORTANCE Antibiotic resistance is a major challenge for modern medicine. Aside from genetic resistance to antibiotics, phenotypic resistance that is not heritable might play a crucial role for the evolution of antibiotic resistance. Using a highly controlled microfluidic system, we characterize single cells under recurrent exposure to antibiotics. Fluctuating antibiotic exposure is likely experienced under common antibiotic therapies. These phenotypic resistant cell characteristics differ from previously described phenotypic resistance, highlighting the diversity of modes of resistance. The phenotypic characteristics of resistant cells we identify also imply that such cells might provide a stepping stone toward genetic resistance, thereby causing treatment failure. Antibiotic resistance is a major challenge for modern medicine. Aside from genetic resistance to antibiotics, phenotypic resistance that is not heritable might play a crucial role for the evolution of antibiotic resistance. Using a highly controlled microfluidic system, we characterize single cells under recurrent exposure to antibiotics. Fluctuating antibiotic exposure is likely experienced under common antibiotic therapies. These phenotypic resistant cell characteristics differ from previously described phenotypic resistance, highlighting the diversity of modes of resistance. The phenotypic characteristics of resistant cells we identify also imply that such cells might provide a stepping stone toward genetic resistance, thereby causing treatment failure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5aacb6bdfb65cb5e5f9765ec92a6f4bb1379df93" target='_blank'>
              Phenotypic resistant single-cell characteristics under recurring ampicillin antibiotic exposure in Escherichia coli
              </a>
            </td>
          <td>
            Silvia Kollerová, Lionel Jouvet, Julia Smelková, Sara Zunk-Parras, A. Rodríguez-Rojas, Ulrich K. Steiner
          </td>
          <td>2024-06-26</td>
          <td>mSystems</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c5f226ca297006300a3de7681b5b8c136da7920" target='_blank'>
              The chromatin accessibility and transcriptomic landscape of the aging mice cochlea and the identification of potential functional super-enhancers in age-related hearing loss
              </a>
            </td>
          <td>
            Chanyuan Zhang, Ting Yang, Xiaoqin Luo, Xiaoqing Zhou, Menglong Feng, Wei Yuan
          </td>
          <td>2024-07-04</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8e1be4efe22c9fbd98a98ceb3e0939364442527" target='_blank'>
              Genome-wide CRISPR screening identifies tyrosylprotein sulfotransferase-2 as a target for augmenting anti-PD1 efficacy
              </a>
            </td>
          <td>
            Yumi Oh, Sujeong Kim, Yunjae Kim, Hyun Kim, Dongjun Jang, Seungjae Shin, Soo Jin Lee, Jiwon Kim, Sang Eun Lee, Jaeik Oh, Yoojin Yang, Dohee Kim, Hae-Rim Jung, Sangjin Kim, Jihui Kim, K. Min, Beomki Cho, Hoseok Seo, Dohyun Han, Hansoo Park, Sung-Yup Cho
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee6169884636c48723540d43c960017df0f71986" target='_blank'>
              Hidden secrets of the cancer genome: unlocking the impact of non-coding mutations in gene regulatory elements
              </a>
            </td>
          <td>
            Sandra Íñiguez-Muñoz, P. Llinàs-Arias, Miquel Ensenyat-Mendez, Andrés F. Bedoya-López, Javier I. J. Orozco, Javier Cortés, Ananya Roy, Karin Forsberg-Nilsson, M. DiNome, D. Marzese
          </td>
          <td>2024-06-20</td>
          <td>Cellular and Molecular Life Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Due to the heterogeneity of cancer, precision medicine has been a major challenge for cancer treatment. Determining medication regimens based on patient genotypes has become a research hotspot in cancer genomics. In this study, we aim to identify key biomarkers for targeted therapies based on single nucleotide variants (SNVs) and copy number variants (CNVs) of genes. The experiment is carried out on 7 cancers on the Encyclopedia of Cancer Cell Lines (CCLE) dataset. Considering the high mutability of driver genes which result in abundant mutated samples, the effect of data sparsity can be eliminated to a large extent. Therefore, we focus on discovering the relationship between driver mutation patterns and three measures of drug response, namely area under the curve (AUC), half maximal effective concentration (EC50), and log2-fold change (LFC). First, multiple statistical methods are applied to assess the significance of difference in drug response between sample groups. Next, for each driver gene, we analyze the extent to which its mutations can affect drug response. Based on the results of multiple hypothesis tests and correlation analyses, our main findings include the validation of several known drug response biomarkers such as BRAF, NRAS, MAP2K1, MAP2K2, and CDKN2A, as well as genes with huge potential to infer drug responses. It is worth emphasizing that we identify a list of genes including SALL4, B2M, BAP1, CCDC6, ERBB4, FOXA1, GRIN2A, and PTPRT, whose impact on drug response spans multiple cancers and should be prioritized as key biomarkers for targeted therapies. Furthermore, based on the statistical p-values and correlation coefficients, we construct gene-drug sensitivity maps for cancer drug recommendation. In this work, we show that driver mutation patterns could be used to tailor therapeutics for precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa993499162c59f7b76c5c6ffab6d2e07409c25b" target='_blank'>
              Single nucleotide and copy number variants of cancer driver genes inform drug response in multiple cancers
              </a>
            </td>
          <td>
            Zeyuan Wang, Hong Gu, Pan Qin, Jia Wang
          </td>
          <td>2024-07-31</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Technological advancements in cell-free DNA (cfDNA) liquid biopsy have triggered exponential growth in numerous clinical applications. While cfDNA-based liquid biopsy has made significant strides in personalizing cancer treatment, the exploration and translation of epigenetics in liquid biopsy to clinical practice is still nascent. This comprehensive review seeks to provide a broad yet in-depth narrative of the present status of epigenetics in cfDNA liquid biopsy and its associated challenges. It highlights the potential of epigenetics in cfDNA liquid biopsy technologies with the hopes of enhancing its clinical translation. The momentum of cfDNA liquid biopsy technologies in recent years has propelled epigenetics to the forefront of molecular biology. We have only begun to reveal the true potential of epigenetics in both our understanding of disease and leveraging epigenetics in the diagnostic and therapeutic domains. Recent clinical applications of epigenetics-based cfDNA liquid biopsy revolve around DNA methylation in screening and early cancer detection, leading to the development of multi-cancer early detection tests and the capability to pinpoint tissues of origin. The clinical application of epigenetics in cfDNA liquid biopsy in minimal residual disease, monitoring, and surveillance are at their initial stages. A notable advancement in fragmentation patterns analysis has created a new avenue for epigenetic biomarkers. However, the widespread application of cfDNA liquid biopsy has many challenges, including biomarker sensitivity, specificity, logistics including infrastructure and personnel, data processing, handling, results interpretation, accessibility, and cost effectiveness. Exploring and translating epigenetics in cfDNA liquid biopsy technology can transform our understanding and perception of cancer prevention and management. cfDNA liquid biopsy has great potential in precision oncology to revolutionize conventional ways of early cancer detection, monitoring residual disease, treatment response, surveillance, and drug development. Adapting the implementation of liquid biopsy workflow to the local policy worldwide and developing point-of-care testing holds great potential to overcome global cancer disparity and improve cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa7f61a66ee2f452017bc6813b88e2fe69622c60" target='_blank'>
              Translation of Epigenetics in Cell-Free DNA Liquid Biopsy Technology and Precision Oncology
              </a>
            </td>
          <td>
            Wan Ying Tan, Snigdha Nagabhyrava, Olivia Ang-Olson, Paromita Das, Luisa Ladel, B. Sailo, Linda He, Anup Sharma, Nita Ahuja
          </td>
          <td>2024-06-27</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="BACKGROUND/AIM
There are no established biomarkers for immune checkpoint inhibitors (ICI) in colorectal cancer (CRC) with microsatellite stability (MSS) or proficient mismatch repair (pMMR). Therefore, this study aimed to identify biomarkers for ICI benefit in patients with pMMR by analyzing the down-regulated DNA repair-related genes involved in highly immunogenic and immune responses, and comparing their expression levels and clinical features.


MATERIALS AND METHODS
Mismatch repair (MMR), tumor-infiltrating lymphocytes (TIL), and tumor mutation burden (TMB) were evaluated in 13 CRC cases and mRNA expression levels of 95 DNA repair-related genes were measured. DNA repair-related genes with reduced mRNA expression in the high immunogenicity and high immune response groups were identified. Then, the mRNA expression levels of the identified DNA repair-related genes were measured in 135 patients with CRC. Hierarchical cluster analysis was performed using the mRNA expression levels to compare the clinicopathological characteristics of each cluster.


RESULTS
ATR, LIG4, and RAD52 mRNA levels were significantly down-regulated in the high immunogenicity group. GADD45B, SMUG1, and XRCC6 mRNA levels were significantly down-regulated in the high immune response group. Cases in the cluster with reduced mRNA expression of the six genes were pMMR cases. CD8 mRNA expression level was higher in this cluster than in the other clusters.


CONCLUSION
Decreased mRNA expression levels of ATR, LIG4, RAD52, GADD45B, SMUG1, and XRCC6 genes were associated with high cytotoxic T cell and TMB levels, suggesting that these genes could serve as biomarkers for ICI efficacy in pMMR cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb38d024ab94655fe817bc6cd4f0bb4ad2d969e2" target='_blank'>
              Identification of DNA Repair-related Genes as Biomarkers Reflecting MSI, TMB, and TIL in Colorectal Cancer.
              </a>
            </td>
          <td>
            Junya Fukuda, Tomoya Sudo, Maako Kikuchi, Akihiko Kawahara, Hirona Shigyo, Yusuke Kawamoto, Satoshi Shimamura, Fumiki Koga, Takasi Noguchi, Hiroyuki Nakane, Takahiro Shigaki, Kenji Fujiyoshi, T. Yomoda, Naohiro Yoshida, Kenichi Koushi, Takefumi Yoshida, Jun Akiba, Fumihiko Fujita
          </td>
          <td>2024-07-26</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia expressing the gamma delta T cell receptor (yo T-ALL) is a poorly understood disease. We studied 200 children with yo T-ALL from 13 clinical study groups to understand the clinical and genetic features of this disease. We found age and genetic drivers were significantly associated with outcome. yo T-ALL diagnosed in children under three years of age was extremely high-risk and enriched for genetic alterations that result in both LMO2 activation and STAG2 inactivation. Mechanistically, using patient samples and isogenic cell lines, we show that inactivation of STAG2 profoundly perturbs chromatin organization by altering enhancer-promoter looping, resulting in deregulation of gene expression associated with T-cell differentiation. High throughput drug screening identified a vulnerability in DNA repair pathways arising from STAG2 inactivation, which can be targeted by Poly(ADP-ribose) polymerase (PARP) inhibition. These data provide a diagnostic framework for classification and risk stratification of pediatric yo T-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fed40523605c31fb96201abbfd6abddb8593a044" target='_blank'>
              Biologic and clinical analysis of childhood gamma delta T-ALL identifies LMO2/STAG2 rearrangements as extremely high-risk.
              </a>
            </td>
          <td>
            Shunsuke Kimura, Chun Shik Park, Lindsey E Montefiori, Ilaria Iacobucci, Petri Polonen, Qingsong Gao, Elizabeth D Arnold, A. Attarbaschi, Anthony Brown, B. Buldini, Kenneth J Caldwell, Yunchao Chang, Chelsey Chen, Cheng Cheng, Zhong-Yi Cheng, John K. Choi, V. Conter, K. Crews, H. D. de Groot-Kruseman, Takao Deguchi, Mariko Eguchi, Hannah Elisa Muhle, S. Elitzur, G. Escherich, Burgess B Freeman, Zhaohui Gu, Katie Han, K. Horibe, Toshihiko Imamura, S. Jeha, Motohiro Kato, Kean-Hui Chiew, Tanya Khan, M. Kicinski, Stefan Kohrer, S. Kornblau, R. Kotecha, Chi-Kong Li, Yen-Chun Liu, F. Locatelli, S. Luger, E. Paietta, Atsushi Manabe, H. Marquart, R. Masetti, Mellissa Maybury, Pauline Mazilier, Jules P P Meijerink, Sharnise Mitchell, T. Miyamura, Andrew S Moore, Koichi Oshima, K. Pawińska-Wąsikowska, Rob Pieters, Mollie S Prater, Shondra M. Pruett-Miller, Ching-Hon Pui, Chunxu Qu, Michaela Reiterova, Noemi Reyes, K. Roberts, J. M. Rowe, A. Sato, Kjeld Schmiegelow, M. Schrappe, Shuhong Shen, S. Skoczeń, O. Spinelli, Jan Stary, M. Svatoň, Masatoshi Takagi, Junko Takita, Yanjing Tang, D. Teachey, Paul G Thomas, D. Tomizawa, J. Trka, Elena Varotto, T. Vincent, Jun J Yang, Allen E J Yeoh, Yinmei Zhou, Martin Zimmermann, H. Inaba, C. Mullighan
          </td>
          <td>2024-06-25</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>102</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4597b2c9168e5b3eb619c4010a4d29dad0a9c28b" target='_blank'>
              mTORC2-driven chromatin cGAS mediates chemoresistance through epigenetic reprogramming in colorectal cancer.
              </a>
            </td>
          <td>
            Guoqing Lv, Qian Wang, Lin Lin, Qiao Ye, Xi Li, Qian Zhou, Xiangzhen Kong, Hongxia Deng, Fuping You, Hebing Chen, Song Wu, Lin Yuan
          </td>
          <td>2024-07-30</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="PLK1 inhibitors are emerging anti-cancer agents being tested in monotherapy and combination therapies for various cancers. Although PLK1 inhibition in experimental models shows potent antitumor effects, translation to the clinic has been hampered by low antitumor activity and tumor relapse. Here, we report the identification of mitochondrial protein signatures that determine sensitivity to approaches targeting PLK1 in human melanoma cell lines. In response to PLK1 inhibition or gene silencing, resistant cells adopt a pro-inflammatory and dedifferentiated phenotype, while sensitive cells engage apoptosis. Mitochondrial DNA depletion and silencing of the ABCD1 transporter sensitize cells to PLK1 inhibition and attenuate the associated pro-inflammatory response. We also found that non-selective inhibitors of the p90 ribosomal S6 kinase (RSK) exert their anti-proliferative and pro-inflammatory effects via PLK1 inhibition. This work reveals overlooked impacts of PLK1 on phenotype switching and suggests that mitochondrial precision medicine can help improve response to targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be11cedd7b84b462b11820d3c974a0336e8915ba" target='_blank'>
              Mitochondrial Signatures Shape Phenotype Switching and Apoptosis in Response to PLK1 and RSK Inhibitors in Melanoma
              </a>
            </td>
          <td>
            Émilie Lavallée, Maëline Roulet-Matton, Viviane Giang, Roxana Cardona Hurtado, Dominic Chaput, Simon-Pierre Gravel
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Bispecific antibodies (bsAbs) that bind to two distinct surface antigens on cancer cells are emerging as an appealing therapeutic strategy in cancer immunotherapy. However, considering the vast number of surface proteins, experimental identification of potential antigen pairs that are selectively expressed in cancer cells and not in normal cells is both costly and time-consuming. Recent studies have utilized large bulk RNA-seq databases to propose bispecific targets for various cancers. However, co-expressed pairs derived from bulk RNA-seq do not necessarily indicate true co-expression of both markers in the malignant cell. Single-cell RNA-seq (scRNA-seq) can circumvent this issue but the issues in low-coverage of transcripts impede the large-scale characterization of co-expressed pairs.



 We present a computational pipeline for bsAbs target identification which combines the advantages of bulk and scRNA-seq while minimizing the issues associated with using these approaches separately. We select Hepatocellular Carcinoma (HCC) as a case study to demonstrate the utility of the approach. First, using the bulk RNA-seq samples in the OCTAD database, we identified target pairs that most distinctly differentiate tumor cases from healthy controls. Next, we confirmed our findings on the scRNA-seq database comprising 39,361 healthy cells from vital organs and 18,000 cells from HCC. The top pair was GPC3∼MUC13, where both genes are co-expressed on the surface of over 30% of malignant HCCs and have very low expression in other cells. Finally, we leveraged the emerging spatial transcriptomic to validate the co-expressed pair in situ.



 A standalone R package (https://github.com/Bin-Chen-Lab/bsAbsFinder)



 Supplementary data are available at Bioinformatics Advances online
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2059ebaa34bf1fd7c83bd94cb6ade0525ae3296b" target='_blank'>
              Computational discovery of co-expressed antigens as dual targeting candidates for cancer therapy through bulk, single-cell, and spatial transcriptomics
              </a>
            </td>
          <td>
            E. Chekalin, Shreya Paithankar, Rama Shankar, Jing Xing, Wenfeng Xu, Bin Chen
          </td>
          <td>2024-06-20</td>
          <td>Bioinformatics Advances</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The genomic analyses of pediatric acute lymphoblastic leukemia (ALL) subtypes, particularly T-cell and B-cell lineages, have been pivotal in identifying potential therapeutic targets. Typical genomic analyses have directed attention toward the most commonly mutated genes. However, assessing the contribution of mutations to cancer phenotypes is crucial. Therefore, we estimated the cancer effects (scaled selection coefficients) for somatic substitutions in T-cell and B-cell cohorts, revealing key insights into mutation contributions. Cancer effects for well-known, frequently mutated genes like NRAS and KRAS in B-ALL were high, which underscores their importance as therapeutic targets. However, less frequently mutated genes IL7R, XBP1, and TOX also demonstrated high cancer effects, suggesting pivotal roles in the development of leukemia when present. In T-ALL, KRAS and NRAS are less frequently mutated than in B-ALL. However, their cancer effects when present are high in both subtypes. Mutations in PIK3R1 and RPL10 were not at high prevalence, yet exhibited some of the highest cancer effects in individual T-cell ALL patients. Even CDKN2A, with a low prevalence and relatively modest cancer effect, is potentially highly relevant for the epistatic effects that its mutated form exerts on other mutations. Prioritizing investigation into these moderately frequent but potentially high-impact targets not only presents novel personalized therapeutic opportunities but also enhances the understanding of disease mechanisms and advances precision therapeutics for pediatric ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ac690cd439c4c5a573dbc1e4f92e6974b5d6430" target='_blank'>
              Rare Drivers at Low Prevalence with High Cancer Effects in T-Cell and B-Cell Pediatric Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Jeff Mandell, Saathvika Diviti, Mina L. Xu, Jeffrey P. Townsend
          </td>
          <td>2024-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Poly(ADP-ribose) polymerase (PARP) inhibitors are targeted therapies that accumulate DNA damage by interfering with DNA repair mechanisms and are approved for treating several cancers with BRCA1/2 mutations. In this study, we utilized CRISPR-dCas9 interference screening to identify genes regulating sensitivity to PARP inhibitors in breast cancer cell lines. Our findings indicated that the interferon (IFN) signaling gene IRF9 was critically involved in modulating sensitivity to these inhibitors. We revealed that the loss of IRF9 leads to increased resistance to the PARP inhibitor in MDA-MB-468 cells, and a similar desensitization was observed in another breast cancer cell line, MDA-MB-231. Further analysis indicated that while the basal expression of IRF9 did not correlate with the response to the PARP inhibitor olaparib, its transcriptional induction was significantly associated with increased sensitivity to the DNA-damaging agent cisplatin in the NCI-60 cell line panel. This finding suggests a mechanistic link between IRF9 induction and cellular responses to DNA damage. Additionally, data from the METABRIC patient tissue study revealed a complex network of IFN-responsive gene expressions postchemotherapy, with seven upregulated genes, including IRF9, and three downregulated genes. These findings underscore the intricate role of IFN signaling in the cellular response to chemotherapy. Collectively, our CRISPR screening data and subsequent bioinformatic analyses suggest that IRF9 is a novel biomarker for sensitivity to DNA-damaging agents, such as olaparib and platinum-based chemotherapeutic agents. Our findings for IRF9 not only enhance our understanding of the genetic basis of drug sensitivity, but also elucidate the role of IRF9 as a critical effector within IFN signaling pathways, potentially influencing the association between the host immune system and chemotherapeutic efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/269c63f48793fead5c71d6f165049d60737829ca" target='_blank'>
              The Role of IRF9 Upregulation in Modulating Sensitivity to Olaparib and Platinum-Based Chemotherapies in Breast Cancer
              </a>
            </td>
          <td>
            SeokGyeong Choi, Han-Gyu Bae, Dong-Kyu Jo, Woo-Young Kim
          </td>
          <td>2024-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cytogenomic characterization is crucial for the classification and risk stratification of acute myeloid leukemia (AML), thereby facilitating therapeutic decision-making. We examined the clinical utility of optical genome mapping (OGM) in 159 AML patients (103 newly diagnosed and 56 refractory/relapsed), all of whom also underwent chromosomal banding analysis (CBA), fluorescence in situ hybridization, and targeted next-generation sequencing. OGM detected nearly all clinically relevant cytogenetic abnormalities that SCG identified with >99% sensitivity, provided the clonal burden was above 20%. OGM identified additional cytogenomic aberrations and/or provided information on fusion genes in 77 (48%) patients, including eight patients with normal karyotypes and four with failed karyotyping. The most common additional alterations identified by OGM included chromoanagenesis (n = 23), KMT2A partial tandem duplication (n = 11), rearrangements involving MECOM (n = 7), NUP98 (n = 2), KMT2A (n = 2), JAK2 (n = 2), and other gene fusions in 17 patients, with 10 showing novel fusion gene partners. OGM also pinpointed fusion genes in 17 (11%) patients where chromosomal rearrangements were concurrently detected by OGM and CBA. Overall, 24 (15%) aberrations were identified exclusively by OGM and had the potential to alter AML classification, risk stratification, and/or clinical trial eligibility. OGM emerges as a powerful tool for identifying fusion genes and detecting subtle or cryptic cytogenomic aberrations that may otherwise remain undetectable by CBA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d116f1be018b74ce05fc5532f2fa6f900955563" target='_blank'>
              Optical genome mapping improves the accuracy of classification, risk stratification, and personalized treatment strategies for patients with acute myeloid leukemia.
              </a>
            </td>
          <td>
            S. Loghavi, Qing Wei, F. Ravandi, Andres E. Quesada, M. Routbort, Shimin Hu, G. Toruner, Sa A Wang, Wei Wang, Roberto N. Miranda, Shaoying Li, Jie Xu, Courtney D. DiNardo, Naval G Daver, T. Kadia, G. Issa, H. Kantarjian, L. J. Medeiros, G. Tang
          </td>
          <td>2024-07-17</td>
          <td>American journal of hematology</td>
          <td>0</td>
          <td>223</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f019d23b5adc09f83ebfe644966cdb6ede88d903" target='_blank'>
              Immune evasion impacts the landscape of driver genes during cancer evolution
              </a>
            </td>
          <td>
            Lucie E. Gourmet, A. Sottoriva, Simon Walker-Samuel, M. Secrier, L. Zapata
          </td>
          <td>2024-06-26</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The Ten-Eleven Translocation (TET) family genes are implicated in a wide array of biological functions across various human cancers. Nonetheless, there is a scarcity of studies that comprehensively analyze the correlation between TET family members and the molecular phenotypes and clinical characteristics of different cancers. Leveraging updated public databases and employing several bioinformatics analysis methods, we assessed the expression levels, somatic variations, methylation levels, and prognostic values of TET family genes. Additionally, we explored the association between the expression of TET family genes and pathway activity, tumor microenvironment (TME), stemness score, immune subtype, clinical staging, and drug sensitivity in pan-cancer. Molecular biology and cytology experiments were conducted to validate the potential role of TET3 in tumor progression. Each TET family gene displayed distinct expression patterns across at least ten detected tumors. The frequency of Single Nucleotide Variant (SNV) in TET genes was found to be 91.24%, primarily comprising missense mutation types, with the main types of copy number variant (CNV) being heterozygous amplifications and deletions. TET1 gene exhibited high methylation levels, whereas TET2 and TET3 genes displayed hypomethylation in most cancers, which correlated closely with patient prognosis. Pathway activity analysis revealed the involvement of TET family genes in multiple signaling pathways, including cell cycle, apoptosis, DNA damage response, hormone AR, PI3K/AKT, and RTK. Furthermore, the expression levels of TET family genes were shown to impact the clinical staging of tumor patients, modulate the sensitivity of chemotherapy drugs, and thereby influence patient prognosis by participating in the regulation of the tumor microenvironment, cellular stemness potential, and immune subtype. Notably, TET3 was identified to promote cancer progression across various tumors, and its silencing was found to inhibit tumor malignancy and enhance chemotherapy sensitivity. These findings shed light on the role of TET family genes in cancer progression and offer insights for further research on TET3 as a potential therapeutic target for pan-cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db4df11d8b0544aeb2264d4da3fba3e8f78e1111" target='_blank'>
              Pan-cancer analyses reveal the molecular and clinical characteristics of TET family members and suggests that TET3 maybe a potential therapeutic target
              </a>
            </td>
          <td>
            Chunyan Zhang, Jie Zheng, Jin Liu, Yanxia Li, G. Xing, Shupeng Zhang, Hekai Chen, Jian Wang, Z. Shao, Yongyuan Li, Zhongmin Jiang, Yingzi Pan, Xiaozhi Liu, Ping Xu, Wenhan Wu
          </td>
          <td>2024-07-22</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Cell immortalization, a hallmark of cancer development, is a process that cells can undergo on their path to carcinogenesis. Spontaneously immortalized mouse embryonic fibroblasts (MEFs) have been used for decades; however, changes in the global transcriptome during this process have been poorly described. In our research, we characterized the poly-A RNA transcriptome changes after spontaneous immortalization. To this end, differentially expressed genes (DEGs) were screened using DESeq2 and characterized by gene ontology enrichment analysis and protein–protein interaction (PPI) network analysis to identify the potential hub genes. In our study, we identified changes in the expression of genes involved in proliferation regulation, cell adhesion, immune response and transcriptional regulation in immortalized MEFs. In addition, we performed a comparative analysis with previously reported MEF immortalization data, where we propose a predicted gene regulatory network model in immortalized MEFs based on the altered expression of Mapk11, Cdh1, Chl1, Zic1, Hoxd10 and the novel hub genes Il6 and Itgb2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edae843236a102c56d19f5e2f20e7de0a5418029" target='_blank'>
              Transcriptome Profiling of Mouse Embryonic Fibroblast Spontaneous Immortalization: A Comparative Analysis
              </a>
            </td>
          <td>
            Jocshan Loaiza-Moss, U. Braun, Michael Leitges
          </td>
          <td>2024-07-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Peripheral T-cell lymphomas (PTCL) are heterogeneous and highly aggressive hematologic malignancies with dismal outcomes, highlighting the need for novel targeted therapies. PTCLs are characterized by complex tumor-microenvironment (TME) ecosystems with low tumor cell content, admixed with rich immune infiltrates and stromal elements correlated with clinical outcomes. Direct targeting of the TME is an attractive strategy for PTCLs; however, the specific mechanisms that regulate the interdependence between lymphoma and TME remain poorly understood, hampering the development of TME-directed therapies. Our previous work based on driver genetic alterations in PTCL patients and genetic mouse models identified an important role of these driver mutations in remodeling the TME (Cortes et al., 2022). To address the specific role and mechanisms of tumor cell-TME interactions in the pathogenesis of PTCL, we developed a novel experimental platform and cutting-edge computational methodologies using single-nucleus RNAseq analysis to comprehensively profile lymphoma cells and their microenvironment from 27 PTCL patient samples of two major nodal pathologic subtypes, including 18 Nodal T follicular helper cell lymphomas (TFHcL) and 9 PTCL, not otherwise specified (PTCL, NOS). Our approach allowed the identification and in-depth characterization of tumor cells, lymphoid and myeloid TME, and vascular endothelium and non-endothelial stromal cells, which were challenging to capture with conventional single-cell RNAseq. The malignant T-cells in PTCL, NOS were grouped into patient-specific clusters demonstrating significant inter-tumor heterogeneity associated with diverse and frequently complex chromosomal abnormalities identified by copy number variation (CNV) analysis. In contrast, malignant TFHcL cells show a low degree to no apparent chromosomal abnormalities and divide into two major transcriptional subclusters, with tumor cells from the same patient falling into one of the two subclusters. Additionally, genomic profiles of TFHcL tumors revealed that the inter-tumor transcriptional heterogeneity of TFHcL tumor cells was strongly associated with the driver RHOA G17V mutation status. To profile oncogenic pathways critical for PTCL growth, we compared PTCL tumor cells to normal CD4 T-cells from reactive lymphoid tissue. PTCL, NOS and TFHcL tumor cells were equally enriched in the TCR signaling pathway, mitosis, and cell cycle related genes. However, exploitation of KRAS and cytokine-driven inflammatory signals was prevalent in TFHcL but not PTCL, NOS tumor cells. Analysis of the immune TME identified an increase in T regulatory cells, CD8 T-cells, immunoblasts, and macrophages in TFHcL. Furthermore, while dysregulated B-cells with heightened cytokine-driven inflammatory signals were unique to TFHcL, inflammatory macrophages and classic dendritic cells (cDC) were prominent in the TME of both subtypes. Our results uncover shared and subtype-specific pathogenic features in PTCL and pave the way for designing novel precision therapies for PTCL patients.
 Citation Format: Wen-Hsuan Wendy Lin, Anqi W Wang, Bobby B Shih, Jean-Baptiste F Reynier, Laura Quevedo Palacio, Craig Soderquist, Ryan D Najac, Cindy Ma, Govind Bhagat, Adolfo A Ferrando, Raul Rabadan, Teresa Palomero. Single-cell transcriptomics reveals shared and subtype-specific vulnerabilities of the tumor-microenvironment ecosystems in peripheral T-cell lymphomas [abstract]. In: Proceedings of the Fourth AACR International Meeting on Advances in Malignant Lymphoma: Maximizing the Basic-Translational Interface for Clinical Application; 2024 Jun 19-22; Philadelphia, PA. Philadelphia (PA): AACR; Blood Cancer Discov 2024;5(3_Suppl):Abstract nr PR05.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d17bf46cac83bb1af31141fb58102ffe6ac218af" target='_blank'>
              Abstract PR05: Single-cell transcriptomics reveals shared and subtype-specific vulnerabilities of the tumor-microenvironment ecosystems in peripheral T-cell lymphomas
              </a>
            </td>
          <td>
            Wen-Hsuan Wendy Lin, Anqi W Wang, Bobby B. Shih, Jean-Baptiste F Reynier, Laura Quevedo Palacio, Craig R. Soderquist, Ryan Najac, Cindy Ma, Govind Bhagat, Adolfo A Ferrando, Raul Rabadan, Teresa Palomero
          </td>
          <td>2024-06-19</td>
          <td>Blood Cancer Discovery</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Introduction: The Euchromatic Histone Methyl Transferase Protein 2 (EHMT2), also known as G9a, deposits transcriptionally repressive chromatin marks that play pivotal roles in the maturation and homeostasis of multiple organs. Recently, we have shown that Ehmt2 inactivation in the mouse pancreas alters growth and immune gene expression networks, antagonizing Kras-mediated pancreatic cancer initiation and promotion. Here, we elucidate the essential role of Ehmt2 in maintaining a transcriptional landscape that protects organs from inflammation. Methods: Comparative RNA-seq studies between normal postnatal and young adult pancreatic tissue from Ehmt2 conditional knockout animals (Ehmt2 fl/fl ) targeted to the exocrine pancreatic epithelial cells (Pdx1-Cre and P48 Cre/+ ), reveal alterations in gene expression networks in the whole organ related to injury-inflammation-repair, suggesting an increased predisposition to damage. Thus, we induced an inflammation repair response in the Ehmt2 fl/fl pancreas and used a data science-based approach to integrate RNA-seq-derived pathways and networks, deconvolution digital cytology, and spatial transcriptomics. We also analyzed the tissue response to damage at the morphological, biochemical, and molecular pathology levels. Results and discussion: The Ehmt2 fl/fl pancreas displays an enhanced injury-inflammation-repair response, offering insights into fundamental molecular and cellular mechanisms involved in this process. More importantly, these data show that conditional Ehmt2 inactivation in exocrine cells reprograms the local environment to recruit mesenchymal and immunological cells needed to mount an increased inflammatory response. Mechanistically, this response is an enhanced injury-inflammation-repair reaction with a small contribution of specific Ehmt2-regulated transcripts. Thus, this new knowledge extends the mechanisms underlying the role of the Ehmt2-mediated pathway in suppressing pancreatic cancer initiation and modulating inflammatory pancreatic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6875f8969140d78af8699723af4ec6fb4942849a" target='_blank'>
              Ehmt2 inactivation in pancreatic epithelial cells shapes the transcriptional landscape and inflammation response of the whole pancreas
              </a>
            </td>
          <td>
            Gareth Pollin, A. Mathison, T. D. de Assunção, Anju Thomas, Atefeh Zeighami, Ann Salmonson, Hongfei Liu, Guillermo Urrutia, Pallavi Vankayala, Stephen J. Pandol, Johnny C. Hong, Michael T. Zimmermann, J. Iovanna, Victor X. Jin, Raul Urrutia, G. Lomberk
          </td>
          <td>2024-06-14</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e81b6b5c5ed75ea0aa53f16f320c4ca6faed6d1d" target='_blank'>
              Intricate effects of post-translational modifications in liver cancer: mechanisms to clinical applications
              </a>
            </td>
          <td>
            Yu Zhang, Weihao Xu, Chuanhui Peng, Shenli Ren, Chengpeng Zhang
          </td>
          <td>2024-07-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d31de7dddff96b795acb0a02ad70f6902f3e8f2" target='_blank'>
              Progress of single-cell RNA sequencing combined with spatial transcriptomics in tumour microenvironment and treatment of pancreatic cancer
              </a>
            </td>
          <td>
            Jie Zhu, Ke Zhang, Yuan Chen, Xinyu Ge, Junqing Wu, Peng Xu, Jie Yao
          </td>
          <td>2024-06-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="microRNAs (miRNAs), tiny, non-coding RNA molecules, fine-tune the expression of target genes through interacting with mRNAs. These miRNAs are involved in a wide range of biological processes, encompassing cell division, death, blood cell production, and tumor development. When these miRNAs become dysfunctional, they can promote the invasion and spread of cancer cells in various human malignancies, including leukemia. Acute lymphoblastic leukemia (ALL), the preeminent malignancy affecting children, is a blood cancer marked by the uncontrollable growth of immature lymphoid cells that displace healthy blood precursors in the bone marrow. Despite a decline in ALL mortality rates over the past two decades, a significant proportion of deaths still results from a lack of effective diagnostic and prognostic markers that can guide treatment decisions and overcome drug resistance. The analysis of miRNA expression patterns in ALL could lead to more precise disease classification, earlier diagnosis, and better prognostic outcomes in the near future. The connection between miRNA dysfunction and the biology of ALL suggests that these molecules could represent promising therapeutic targets. Therefore, this review delves into the regulatory mechanisms of miRNAs in pediatric ALL, exploring how miRNA-based diagnostic, prognostic, and therapeutic strategies offer unique advantages and hold promise for clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b51d3730adcae846d16d92db76f51f153c5c1d0" target='_blank'>
              Advancements in the Regulatory Role of microRNAs in Childhood Acute Lymphoblastic Leukemia: Mechanisms and Clinical Implications
              </a>
            </td>
          <td>
            Wei Deng
          </td>
          <td>2024-01-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69507c41c181ea702ded05e95f253bb2f74c3d09" target='_blank'>
              Breast cancer epigenetics: current and evolving treatment.
              </a>
            </td>
          <td>
            Sujata Purja, Dung Thuy Nguyen, Eunyoung Kim
          </td>
          <td>2024-06-11</td>
          <td>Breast cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd083194686b61ccc53ca0d4c394b67971b4cdb7" target='_blank'>
              Targeted gene therapy for cancer: the impact of microRNA multipotentiality
              </a>
            </td>
          <td>
            Nourhan A. Abou Madawi, Zeinab E Darwish, Enas M Omar
          </td>
          <td>2024-08-01</td>
          <td>Medical Oncology (Northwood, London, England)</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Naïve pluripotent stem cells (nPSC) frequently undergo pathological and not readily reversible loss of DNA methylation marks at imprinted gene loci. This abnormality poses a hurdle for using pluripotent cell lines in biomedical applications and underscores the need to identify the causes of imprint instability in these cells. We show that nPSCs from inbred mouse strains exhibit pronounced strain-specific susceptibility to locus-specific deregulation of imprinting marks during reprogramming to pluripotency and upon culture with MAP kinase inhibitors, a common approach to maintain naïve pluripotency. Analysis of genetically highly diverse nPSCs from the Diversity Outbred (DO) stock confirms that genetic variation is a major determinant of epigenome stability in pluripotent cells. We leverage the variable DNA hypomethylation in DO lines to identify several trans-acting quantitative trait loci (QTLs) that determine epigenome stability at either specific target loci or genome-wide. Candidate factors encoded by two multi-target QTLs on chromosomes 4 and 17 suggest specific transcriptional regulators that contribute to DNA methylation maintenance in nPSCs. We propose that genetic variants represent candidate biomarkers to identify pluripotent cell lines with desirable properties and might serve as entry points for the targeted engineering of nPSCs with stable epigenomes. Highlights Naïve pluripotent stem cells from distinct inbred mouse strains exhibit variable DNA methylation levels at imprinted gene loci. The vulnerability of pluripotent stem cells to loss of genomic imprinting caused by MAP kinase inhibition strongly differs between inbred mouse strains. Genetically diverse pluripotent stem cell lines from Diversity Outbred mouse stock allow the identification of quantitative trait loci controlling DNA methylation stability. Genetic variants may serve as biomarkers to identify naïve pluripotent stem cell lines that are epigenetically stable in specific culture conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fcf4c9c2e254cef424889809e18c41add6362bb" target='_blank'>
              Genetic variation modulates susceptibility to aberrant DNA hypomethylation and imprint deregulation in naïve pluripotent stem cells
              </a>
            </td>
          <td>
            C. Parikh, RA Glenn, Y. Shi, K. Chatterjee, EE Swanzey, S. Singer, SC Do, Y. Zhan, Y. Furuta, M. Tahiliani, E. Apostolou, A. Polyzos, R. Koche, JG Mezey, T. Vierbuchen, M. Stadtfeld
          </td>
          <td>2024-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Annotation of the cis-regulatory elements that drive transcriptional dysregulation in cancer cells is critical to improving our understanding of tumor biology. Herein, we present a compendium of matched chromatin accessibility (scATAC-seq) and transcriptome (scRNA-seq) profiles at single-cell resolution from human breast tumors and healthy mammary tissues processed immediately following surgical resection. We identify the most likely cell-of-origin for luminal breast tumors and basal breast tumors and then introduce a novel methodology that implements linear mixed-effects models to systematically quantify associations between regions of chromatin accessibility (i.e. regulatory elements) and gene expression in malignant cells versus normal mammary epithelial cells. These data unveil regulatory elements with that switch from silencers of gene expression in normal cells to enhancers of gene expression in cancer cells, leading to the upregulation of clinically relevant oncogenes. To translate the utility of this dataset into tractable models, we generated matched scATAC-seq and scRNA-seq profiles for breast cancer cell lines, revealing, for each subtype, a conserved oncogenic gene expression program between in vitro and in vivo cells. Together, this work highlights the importance of non-coding regulatory mechanisms that underlie oncogenic processes and the ability of single-cell multi-omics to define the regulatory logic of BC cells at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08bf4929898d9a3caa32f9bca65e96287e05306d" target='_blank'>
              Defining the Regulatory Logic of Breast Cancer Using Single-Cell Epigenetic and Transcriptome Profiling
              </a>
            </td>
          <td>
            Matthew J. Regner, Susana Garcia-Recio, A. Thennavan, K. Wiśniewska, Raul Mendez-Giraldez, Brooke Felsheim, Philip M. Spanheimer, Joel S. Parker, C. Perou, Hector L. Franco
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>160</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4781c7d248ebd4c7b1d88e8451d7cdc1606f06e6" target='_blank'>
              Genome-wide enhancer RNA profiling adds molecular links between genetic variation and human cancers
              </a>
            </td>
          <td>
            Yimin Cai, Zequn Lu, Bin Li, Jinyu Huang, Ming Zhang, Can Chen, Linyun Fan, Qian-Ying Ma, Chunyi He, Shuo Chen, Yuan Jiang, Yan-Min Li, Caibo Ning, Fu-Wei Zhang, Wenzhuo Wang, Yizhuo Liu, Heng Zhang, Meng Jin, Xiao-Yang Wang, Jinxin Han, Zhen Xiong, Ming Cai, Chaoqun Huang, Xiaojun Yang, Xu Zhu, Ying Zhu, Xiaoping Miao, Shao-Kai Zhang, Yongchang Wei, Jianbo Tian
          </td>
          <td>2024-06-11</td>
          <td>Military Medical Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b139e901b81d232b7592efe9dd25a1948b27ddfe" target='_blank'>
              IκBα kinase inhibitor BAY 11-7082 promotes anti-tumor effect in RAS-driven cancers
              </a>
            </td>
          <td>
            P. Guruvaiah, Romi Gupta
          </td>
          <td>2024-07-09</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Transfer RNA (tRNA) modifications have emerged as critical posttranscriptional regulators of gene expression affecting diverse biological and disease processes. While there is extensive knowledge about the enzymes installing the dozens of post-transcriptional tRNA modifications – the tRNA epitranscriptome – very little is known about how metabolic, signaling, and other networks integrate to regulate tRNA modification levels. Here we took a comprehensive first step at understanding epitranscriptome regulatory networks by developing a high-throughput tRNA isolation and mass spectrometry-based modification profiling platform and applying it to a Pseudomonas aeruginosa transposon insertion mutant library comprising 5,746 strains. Analysis of >200,000 tRNA modification data points validated the annotations of predicted tRNA modification genes, uncovered novel tRNA-modifying enzymes, and revealed tRNA modification regulatory networks in P. aeruginosa. Platform adaptation for RNA-seq library preparation would complement epitranscriptome studies, while application to human cell and mouse tissue demonstrates its utility for biomarker and drug discovery and development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cd74a5ca2ea04241d0e2f28ffe633a7113d027f" target='_blank'>
              tRNA modification profiling reveals epitranscriptome regulatory networks in Pseudomonas aeruginosa
              </a>
            </td>
          <td>
            Jingjing Sun, Junzhou Wu, Yifeng Yuan, Leon Fan, Wei Lin Patrina Chua, Yan Han Sharon Ling, S. Balamkundu, Dwija priya, Hazel Chay Suen Suen, V. de Crécy-Lagard, Agnieszka Dziergowska, Peter C. Dedon
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Cancer genomic studies have identified frequent alterations in components of the SWI/SNF (SWItch/Sucrose Non- Fermenting) chromatin remodeling complex including SMARCA4 and ARID1A. Importantly, clinical reports indicate that SMARCA4-mutant lung cancers respond poorly to immunotherapy and have dismal prognosis. However, the mechanistic basis of immunotherapy resistance is unknown. Here, we corroborated the clinical findings by using immune-humanized, syngeneic, and genetically engineered mouse models of lung cancer harboring SMARCA4 deficiency. Specifically, we show that SMARCA4 loss caused decreased response to anti-PD1 immunotherapy associated with significantly reduced infiltration of dendritic cells (DCs) and CD4+ T cells into the tumor microenvironment (TME). Mechanistically, we show that SMARCA4 loss in tumor cells led to profound downregulation of STING, IL1β and other components of the innate immune system as well as inflammatory cytokines that are required for efficient recruitment and activity of immune cells. We establish that this deregulation of gene expression is caused by cancer cell-intrinsic reprogramming of the enhancer landscape with marked loss of chromatin accessibility at enhancers of genes involved in innate immune response such as STING, IL1β, type I IFN and inflammatory cytokines. Interestingly, we observed that transcription factor NF-κB binding motif was highly enriched in enhancers that lose accessibility upon SMARCA4 deficiency. Finally, we confirmed that SMARCA4 and NF-κB co-occupy the same genomic loci on enhancers associated with STING and IL1β, indicating a functional interplay between SMARCA4 and NF-κB. Taken together, our findings provide the mechanistic basis for the poor response of SMARCA4-mutant tumors to anti-PD1 immunotherapy and establish a functional link between SMARCA4 and NF-κB on innate immune and inflammatory gene expression regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3fdeb7ebdd844e1757256a995b264456319e406" target='_blank'>
              SMARCA4 mutation induces tumor cell-intrinsic defects in enhancer landscape and resistance to immunotherapy
              </a>
            </td>
          <td>
            Yawen Wang, Ismail M. Meraz, Md Qudratullah, Sasi Kotagiri, Yanyan Han, Yuanxin Xi, Jing Wang, Yonathan Lissanu
          </td>
          <td>2024-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="BACKGROUND
Prenatal folate supplementation has been consistently associated with a reduced risk of childhood lymphoblastic leukemia (ALL). Previous germline genetic studies examining the one carbon (folate) metabolism pathway were limited in sample size, scope, and population diversity, and led to inconclusive results.


METHODS
We evaluated whether ~2,900 single nucleotide polymorphisms (SNPs) within 46 candidate genes involved in the folate metabolism pathway influence the risk of childhood ALL, using genome-wide data from nine case-control-studies in the Childhood Cancer and Leukemia International Consortium (n=9,058 cases including 4,510 children of European ancestry, 3,018 Latinx, and 1,406 Asians, and 92,364 controls). Each study followed a standardized protocol for quality control and imputation of genome-wide data, and summary statistics were meta-analyzed for all children combined and by major ancestry group using METAL software.


RESULTS
None of the selected SNPs reached statistical significance, overall and for major ancestry groups (using adjusted Bonferroni p-value of 5x10-6 and less stringent p-value of 3.5x10-5 accounting for the number of "independent" SNPs). None of the 10 top (non-significant) SNPs and corresponding genes overlapped across ancestry groups.


CONCLUSION
This large meta-analysis of original data does not reveal associations between many common genetic variants in the folate metabolism pathway and childhood ALL in various ancestry groups.


IMPACT
Genetic variants in the folate pathway alone do not appear to substantially influence childhood ALL risk. Other mechanisms such as gene-folate interaction, DNA methylation or maternal genetic effects may explain the observed associations with self-reported prenatal folate intake.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a898e87257022dcda41ec075ff464556084a0e7a" target='_blank'>
              Folate metabolism and risk of childhood acute lymphoblastic leukemia: a genetic pathway analysis from the Childhood Cancer and Leukemia International Consortium.
              </a>
            </td>
          <td>
            C. Metayer, Logan G Spector, Michael E Scheurer, Soyoung Jeon, Rodney J. Scott, M. Takagi, J. Clavel, Atsushi Manabe, Xiaomei Ma, E. Hailu, Philip J Lupo, K. Urayama, A. Bonaventure, Motohiro Kato, Aline Meirhaeghe, Charleston W. K. Chiang, L. Morimoto, Joseph L Wiemels
          </td>
          <td>2024-06-21</td>
          <td>Cancer epidemiology, biomarkers & prevention : a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="The DNA damage response is critical for maintaining genome integrity and is commonly disrupted in the development of cancer. PPM1D (protein phosphatase Mg2+/Mn2+-dependent 1D) is a master negative regulator of the response; gain-of-function mutations and amplifications of PPM1D are found across several human cancers making it a relevant pharmacological target. Here, we used CRISPR/Cas9 screening to identify synthetic-lethal dependencies of PPM1D, uncovering superoxide dismutase-1 (SOD1) as a potential target for PPM1D-mutant cells. We revealed a dysregulated redox landscape characterized by elevated levels of reactive oxygen species and a compromised response to oxidative stress in PPM1D-mutant cells. Altogether, our results demonstrate a role for SOD1 in the survival of PPM1D-mutant leukemia cells and highlight a new potential therapeutic strategy against PPM1D-mutant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d387f2219bf67ae899383074a68bd8679c1df671" target='_blank'>
              SOD1 is a synthetic-lethal target in PPM1D-mutant leukemia cells
              </a>
            </td>
          <td>
            Linda Zhang, Joanne I Hsu, E. Braekeleer, Chun-Wei Chen, Tajhal D. Patel, Alejandra G Martell, A. Guzman, Katharina Wohlan, Sarah M Waldvogel, H. Uryu, A. Tovy, E. Callen, Rebecca L. Murdaugh, Rosemary Richard, Sandra Jansen, L. Vissers, Bert B A de Vries, André Nussenzweig, Shixia Huang, Cristian Coarfa, Jamie Anastas, Koichi Takahashi, George Vassiliou, Margaret A Goodell
          </td>
          <td>2024-06-18</td>
          <td>eLife</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Chimeric antigen receptor T (CAR-T) cells that are obtained from the specific patient, modified genetically ex vivo, and possess the remarkable capability to identify and eradicate targeted cancer cells. These modified cells are subsequently reintroduced into the patient, effectively treating blood cancer. CAR-T therapy is approved to be applied in leukemia due to its great clinical therapeutic effect on B cell hematological malignancies. However, solid tumors are more resistant to this therapy for many reasons. The abnormal vascular structure of solid tumors hampers CAR-T cell trafficking. Various kinds of immunosuppressive cells and chemicals in the tumor microenvironment (TME) accelerates CAR-T cell exhaustion, showing poor persistence in vivo. More and more researches have demonstrated that T cell fate is strongly associated with epigenetic regulation. Epigenetic modification is not a direct addition or deletion of DNA, but a reversible method including modifications on DNA and histones, and non-coding RNA (ncRNA)-mediated regulations. The change of epigenetic landscape in CAR-T cells largely determines the therapeutic performance in vivo. This research outlines three major barriers in CAR-T therapy, including T cell exhaustion, differentiation and infiltration. Additionally, the research elucidates several promising epigenetic reprogramming strategies to reduce CAR-T cells exhaustion, modulate the cell differentiation process, and enhance their infiltration into solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4165f9901246426b44e74fa80485927b8d025d28" target='_blank'>
              Epigenetic Strategies to Optimize CAR-T Therapy
              </a>
            </td>
          <td>
            Danyang Song
          </td>
          <td>2024-07-11</td>
          <td>Highlights in Science, Engineering and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29e5e59bd894b6791e1caba61c558e842d1e8aab" target='_blank'>
              Mitoferrin2 is a synthetic lethal target for chromosome 8p deleted cancers
              </a>
            </td>
          <td>
            Stephan Krieg, Thomas Rohde, Tobias Rausch, Luise Butthof, Lena Wendler-Link, Christoph Eckert, K. Breuhahn, Bruno Galy, J. Korbel, Maximilian Billmann, Marco Breinig, Darjus F. Tschaharganeh
          </td>
          <td>2024-06-17</td>
          <td>Genome Medicine</td>
          <td>1</td>
          <td>95</td>
        </tr>

        <tr id="Genome editing is poised to revolutionize treatment of genetic diseases, but poor understanding and control of DNA repair outcomes hinders its therapeutic potential. DNA repair is especially understudied in nondividing cells like neurons, which must withstand decades of DNA damage without replicating. This lack of knowledge limits the efficiency and precision of genome editing in clinically relevant cells. To address this, we used induced pluripotent stem cells (iPSCs) and iPSC-derived neurons to examine how postmitotic human neurons repair Cas9-induced DNA damage. We discovered that neurons can take weeks to fully resolve this damage, compared to just days in isogenic iPSCs. Furthermore, Cas9-treated neurons upregulated unexpected DNA repair genes, including factors canonically associated with replication. Manipulating this response with chemical or genetic perturbations allowed us to direct neuronal repair toward desired editing outcomes. By studying DNA repair in postmitotic human cells, we uncovered unforeseen challenges and opportunities for precise therapeutic editing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6434cf3661015c386f770a07ae2b52c8bf820f87" target='_blank'>
              Neuronal DNA repair reveals strategies to influence CRISPR editing outcomes
              </a>
            </td>
          <td>
            Gokul N. Ramadoss, Samali J Namaganda, Jennifer R Hamilton, Rohit Sharma, Karena G Chow, Bria L Macklin, Mengyuan Sun, Jia-Cheng Liu, Christof Fellmann, Hannah L. Watry, Julianne Jin, Barbara S Perez, Cindy R Sandoval Espinoza, Madeline Matia, Serena H Lu, Luke M. Judge, A. Nussenzweig, Britt Adamson, Niren Murthy, Jennifer A. Doudna, Martin Kampmann, Bruce R. Conklin
          </td>
          <td>2024-06-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Multidrug resistance (MDR) frequently occurs during cancer therapy and remains a major obstacle for the cure of most cancers(1, 2). Drug resistance could exist intrinsically or be acquired by drug treatment(3–5), yet factors that regulate the resistance remain elusive. Here, we show that most anticancer drugs damage neosynthesized proteins prior to reaching their canonic targets and elicit profound cytotoxicity, which is largely compensated by protein damage response (PDR). We demonstrate that the PDR includes damage recognition and clearance that are mainly mediated by ubiquitin and proteasome systems, although some other factors, including cellular ATP levels and proliferation status, are also involved. We show that cancer stem cells (CSCs), which have lower protein synthesis, and drug resistance acquired cells (DRAC), which have higher proteasome activity, are more resistant than other cells. We further demonstrate that ATP promotes protein synthesis and suppresses proteasome activity, thus, increasing mitochondrial ATP production by PDK1 inhibition and using proteasome inhibitor to block protein damage clearance render CSCs and DRACs more vulnerable to anticancer drugs. Thus, patients with drug-resistant cancers and treatment-naïve patients with low ATP levels and/or high proteasome activity can be identified and subtyped, and therapies containing PDK1-I and/or proteasome-I may be effective options for these patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e9626795c580f97c2519d1342441e9145cbcfc" target='_blank'>
              A critical role of protein damage response in mediating cancer drug resistance
              </a>
            </td>
          <td>
            Fangyuan Shao, Jie Li, Hao Xiao, Ling Li, Bo Li, YuJun Chen, Xiangpeng Chu, Maoxin Ran, Dongyang Tang, Yuzhong Peng, Yujian Huang, Lijian Wang, Yanxia Shi, Nan Shao, Kai Miao, Changhua Zhang, Ying Lin, Jun Yan, Kin Yip Tam, Xiaoling Xu, Chu-Xia Deng
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Uncovering the stimulus-response histories that give rise to cell fates and behaviors is an area of great interest in developmental biology, tissue engineering, and regenerative medicine. A comprehensive accounting of cell experiences that lead to the development of organs and tissues can help us to understand developmental anomalies that may underly disease. Perhaps more provocatively, such a record can also reveal clues as to how to drive cell collective decision-making processes, which may yield predictable cell-based therapies or facilitate production of tissue substitutes for transplantation or in vitro screening of prospective therapies to mitigate disease. Toward this end, various methods have been applied to molecularly trace developmental trajectories and record interaction histories of cells. Typical methods involve artificial gene circuits based on recombinases that activate a suite of fluorescent reporters or CRISPR-Cas9 genome writing technologies whose nucleic acid-based record keeping serves to chronicle cell-cell interactions or past exposure to stimuli of interests. Exciting expansions of the synthetic biology toolkit with artificial receptors that permit establishment of defined input-to-output linkages of cell decision-making processes opens the door to not only record cell-cell interactions, but to also potentiate directed manipulation of the outcomes of such interactions via regulation of carefully selected transgenes. Here, we combine CRISPR-based strategies to genetically and epigenetically manipulate cells to express components of the synthetic Notch receptor platform, a widely used artificial cell signaling module. Our approach gives rise to the ability to conditionally record interactions between human cells, where the record of engagement depends on expression of a state-specific marker of a subset of cells in a population. Further, such signal-competent interactions can be used to direct differentiation of human embryonic stem cells toward pre-selected fates based on assigned synNotch outputs. We also implemented CRISPR-based manipulation of native gene expression profiles to bias outcomes of cell engagement histories in a targeted manner. Thus, we present a useful strategy that gives rise to both state-specific recording of cell-cell interactions as well as methods to intentionally influence products of such cell-cell exchanges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02ae9053a3cbc425922aa6f2eeba4d8e0f7318c4" target='_blank'>
              Use of CRISPRoff and synthetic Notch to modulate and relay endogenous gene expression programs in engineered cells
              </a>
            </td>
          <td>
            Shuqun Shi, Catherine A. Hamann, Joanne C. Lee, Jonathan M. Brunger
          </td>
          <td>2024-06-18</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Homologous recombination rearranges genetic information during meiosis to generate new combinations of variants. Recombination also causes new mutations, affects the GC content of the genome and reduces selective interference. Here, we use HiFi long-read sequencing to directly detect crossover and gene conversion events from switches between the two haplotypes along single HiFi-reads from testis tissue of humans, chimpanzees and gorillas as well as human sperm samples. Furthermore, based on DNA methylation calls, we classify the cellular origin of reads to either somatic or germline cells in the testis tissue. We identify 1692 crossovers and 1032 gene conversions in nine samples and investigate their chromosomal distribution. Crossovers are more telomeric and correlate better with recombination maps than gene conversions. We show a strong concordance between a human double-strand break map and the human samples, but not for the other species, supporting different PRDM9-programmed double-strand break loci. We estimate the average gene conversion tract lengths to be similar and very short in all three species (means 40-100 bp, fitted well by a geometric distribution) and that 95-98% of non-crossover events do not involve tracts intersecting with polymorphism and are therefore not detectable. Finally, we detect a GC bias in the gene conversion of both single and multiple SNVs and show that the GC-biased gene conversion affects SNVs flanking crossover events. This implies that gene conversion events associated with crossover events are much longer (estimated above 500 bp) than those associated with non-crossover events. Highly accurate long-read sequencing combined with the classification of reads to specific cell types provides a new, powerful way to make individual, detailed maps of gene conversion and crossovers for any species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e343940af051e78f3a88674b4f23d5e3391f5fb" target='_blank'>
              Insights into gene conversion and crossing-over processes from long-read sequencing of human, chimpanzee and gorilla testes and sperm
              </a>
            </td>
          <td>
            Peter Soerud Porsborg, A. P. Charmouh, Vinod Kumar Singh, S. B. Winge, C. Hvilsom, Marta Pelizzola, S. Laurentino, N. Neuhaus, A. Hobolth, Thomas Bataillon, Kristian Almstrup, Søren Besenbacher, M. Schierup
          </td>
          <td>2024-07-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="CRISPR-Cas9 has revolutionized genome engineering by allowing precise introductions of DNA double-strand breaks (DSBs). However, genome engineering in bacteria is still a complex, multi-step process requiring a donor DNA template for homology-directed repair of DSBs. Prime editing circumvents this need as the repair template is provided within the prime editing guide RNA (pegRNA). Here, we developed make-or-break Prime Editing (mbPE) that allows for precise and effective genetic engineering in the opportunistic human pathogen Streptococcus pneumoniae. In contrast to traditional prime editing in which a nicking Cas9 is employed, mbPE harnesses wild type Cas9 in combination with a pegRNA that destroys the seed region or protospacer adjacent motif. Since most bacteria poorly perform template-independent end joining, correctly genome-edited clones are selectively enriched during mbPE. We show that mbPE can be used to introduce point mutations, deletions and targeted insertions, including protein tags such as a split luciferase, at selection efficiencies of over 93%. mbPE enables sequential genome editing, is scalable, and can be used to generate pools of mutants in a high-throughput manner. The mbPE system and pegRNA design guidelines described here will ameliorate future bacterial genome editing endeavors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e8cb2090cc4138a6d6d7cd040330c66ebea98de" target='_blank'>
              Make-or-break prime editing for bacterial genome engineering
              </a>
            </td>
          <td>
            Monica Rengifo-Gonzalez, Maria-Vittoria Mazzuoli, Axel B. Janssen, Anne-Stéphanie Rueff, Xue Liu, J. Veening
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Cancer, a multifactorial disease characterized by uncontrolled cellular proliferation, remains a global health challenge with significant morbidity and mortality. Genomic and molecular aberrations, coupled with environmental factors, contribute to its heterogeneity and complexity. Chemotherapeutic agents like doxorubicin (Dox) have shown efficacy against various cancers but are hindered by dose-dependent cytotoxicity, particularly on vital organs like the heart and brain. Autophagy, a cellular process involved in self-degradation and recycling, emerges as a promising therapeutic target in cancer therapy and neurodegenerative diseases. Dysregulation of autophagy contributes to cancer progression and drug resistance, while its modulation holds the potential to enhance treatment outcomes and mitigate adverse effects. Additionally, emerging evidence suggests a potential link between autophagy, DNA damage, and caretaker breast cancer genes BRCA1/2, highlighting the interplay between DNA repair mechanisms and cellular homeostasis. This review explores the intricate relationship between cancer, Dox-induced cytotoxicity, autophagy modulation, and the potential implications of autophagy in DNA damage repair pathways, particularly in the context of BRCA1/2 mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f00613c7c992a22afe5ec78107b8d19206649c8" target='_blank'>
              Complex Interplay between DNA Damage and Autophagy in Disease and Therapy
              </a>
            </td>
          <td>
            Aman Singh, Naresh Ravendranathan, Jefferson C. Frisbee, Krishna K. Singh
          </td>
          <td>2024-07-29</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1731e1b468345dc4e9d5ee9c2101e06de967b49" target='_blank'>
              Timing of whole genome duplication is associated with tumor-specific MHC-II depletion in serous ovarian cancer
              </a>
            </td>
          <td>
            Nikki L Burdett, Madelynne O. Willis, A. Pandey, Laura Twomey, Sara Alaei, D. Bowtell, E. Christie
          </td>
          <td>2024-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Ras genes are important oncogenes that are frequently mutated in cancer. Human oncogenic variants exhibit functional distinctions in terms of their representation in different cancer types, impact on cellular targets, and sensitivity to pharmacological treatments. However, how these distinct variants influence and respond to the cellular networks in which they are embedded is poorly understood. To identify novel participants in the complex interplay between Ras genotype and cell interaction networks in vivo, we have developed and tested an experimental framework using a simple vulva development assay in the nematode C. elegans. Using this system, we evaluate a set of Ras oncogenic substitution changes at G12, G13 and Q61. We find that these variants fall into distinct groups based on phenotypic differences, sensitivity to gene dose and inhibition of the downstream kinase MEK, and their response to genetic modulators that influence Ras activity in a non-autonomous manner. Together, our results demonstrate oncogenic C. elegans Ras variants exhibit clear distinctions in how they interface with the vulva development network, and show that extracellular modulators yield variant-restricted effects in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/675fab343bc71b75d1eb1f6e179c6e3a20afa2a0" target='_blank'>
              Functional distinction in oncogenic Ras variant activity in Caenorhabditis elegans.
              </a>
            </td>
          <td>
            Haimeng Lyu, H. Chamberlin
          </td>
          <td>2024-07-01</td>
          <td>Disease models & mechanisms</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Testing for homologous recombination deficiency (HRD) mutations is pivotal to assess individual risk, to proact preventive measures in healthy carriers and to tailor treatments for cancer patients. Increasing prominence of poly(ADP-ribose) polymerase (PARP) inhibitors with remarkable impact on molecular-selected patient survival across diverse nosologies, ingrains testing for BRCA genes and beyond in clinical practice. Nevertheless, testing strategies remain a question of debate. While several pathogenic BRCA1/2 gene variants have been described as founder pathogenic mutations frequently found in patients from Russia, other homologous recombination repair (HRR) genes have not been sufficiently explored. In this study, we present real-world data of routine HRR gene testing in Russia. Methods We evaluated clinical and sequencing data from cancer patients who had germline/somatic next-generation sequencing (NGS) HRR gene testing in Russia (BRCA1/2/ATM/CHEK2, or 15 HRR genes). The primary objectives of this study were to evaluate the frequency of BRCA1/2 and non-BRCA gene mutations in real-world unselected patients from Russia, and to determine whether testing beyond BRCA1/2 is feasible. Results Data of 2,032 patients were collected from February 2021 to February 2023. Most had breast (n = 715, 35.2%), ovarian (n = 259, 12.7%), pancreatic (n = 85, 4.2%), or prostate cancer (n = 58, 2.9%). We observed 586 variants of uncertain significance (VUS) and 372 deleterious variants (DVs) across 487 patients, with 17.6% HRR-mutation positivity. HRR testing identified 120 (11.8%) BRCA1/2-positive, and 172 (16.9%) HRR-positive patients. With 51 DVs identified in 242 formalin-fixed paraffin-embedded (FFPE), testing for variant origin clarification was required in one case (0.4%). Most BRCA1/2 germline variants were DV (121 DVs, 26 VUS); in non-BRCA1/2 genes, VUS were ubiquitous (53 DVs, 132 VUS). In silico prediction identified additional 4.9% HRR and 1.2% BRCA1/2/ATM/CHEK2 mutation patients. Conclusions Our study represents one of the first reports about the incidence of DV and VUS in HRR genes, including genes beyond BRCA1/2, identified in cancer patients from Russia, assessed by NGS. In silico predictions of the observed HRR gene variants suggest that non-BRCA gene testing is likely to result in higher frequency of patients who are candidates for PARP inhibitor therapy. Continuing sequencing efforts should clarify interpretation of frequently observed non-BRCA VUS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6572f58fdbd3054cbf460314d4d8cc666a9bdfe" target='_blank'>
              Untapped Potential of Poly(ADP-Ribose) Polymerase Inhibitors: Lessons Learned From the Real-World Clinical Homologous Recombination Repair Mutation Testing
              </a>
            </td>
          <td>
            A. Lebedeva, E. Veselovsky, A. Kavun, Ekaterina Belova, T. Grigoreva, Pavel Orlov, Anna Subbotovskaya, Maksim Shipunov, Oleg Mashkov, Fanil Bilalov, Peter Shatalov, Andrey Kaprin, Peter Shegai, Zhan Diuzhev, Ochir Migiaev, Natalya Vytnova, V. Mileyko, Maxim Ivanov
          </td>
          <td>2024-06-11</td>
          <td>World Journal of Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Down syndrome is a genetic-based disorder that results from the triplication of chromosome 21, leading to an overexpression of many triplicated genes, including the gene encoding Dual-Specificity Tyrosine Phosphorylation-Regulated Kinase 1A (DYRK1A). This protein has been observed to regulate numerous cellular processes, including cell proliferation, cell functioning, differentiation, and apoptosis. Consequently, an overexpression of DYRK1A has been reported to result in cognitive impairment, a key phenotype of individuals with Down syndrome. Therefore, downregulating DYRK1A has been explored as a potential therapeutic strategy for Down syndrome, with promising results observed from in vivo mouse models and human clinical trials that administered epigallocatechin gallate. Current DYRK1A inhibitors target the protein function directly, which tends to exhibit low specificity and selectivity, making them unfeasible for clinical or research purposes. On the other hand, antisense oligonucleotides (ASOs) offer a more selective therapeutic strategy to downregulate DYRK1A expression at the gene transcript level. Advances in ASO research have led to the discovery of numerous chemical modifications that increase ASO potency, specificity, and stability. Recently, several ASOs have been approved by the U.S. Food and Drug Administration to address neuromuscular and neurological conditions, laying the foundation for future ASO therapeutics. The limitations of ASOs, including their high production cost and difficulty delivering to target tissues can be overcome by further advances in ASO design. DYRK1A targeted ASOs could be a viable therapeutic approach to improve the quality of life for individuals with Down syndrome and their families.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6325ca7451f96825cdc01a440a3d532d3e9fb0b3" target='_blank'>
              Down syndrome and DYRK1A overexpression: relationships and future therapeutic directions
              </a>
            </td>
          <td>
            Aidan J. Murphy, Steve D. Wilton, M. Aung-Htut, Craig S McIntosh
          </td>
          <td>2024-07-24</td>
          <td>Frontiers in Molecular Neuroscience</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="EPAS1HIF2α is the primary gene implicated in systemic hypoxia adaptation. Conversely, aberrantly activated EPAS1HIF2α acts as a tumor driver against which anti-tumor therapeutics are proven effective. We elucidated connections between adaptation to systemic hypoxia in high-altitude populations, such as Tibetans and Sherpas, and human tumors. Similar to the accelerated adaptability observed in high-altitude populations via genetic introgression, tumors from patients with hypoxia since birth exhibited impaired DNA repair and increased mutation burden. As in high-altitude dwellers, EPAS1HIF2α genetic variants were positively selected within sympathetic tumors developed under hypoxia, with a consistently high frequency of 90%. Bulk and single-cell RNA sequencing followed by in vitro studies have shown that hypoxia and EPAS1HIF2α gain-of-function tumor mutations induce COX4i2 expression and impair mitochondrial respiration, indicating that decreased cellular oxygen consumption may confer a proliferative advantage in hypoxia. Analyzing medical data from a patient cohort with hypoxia since birth who developed/did not develop tumors revealed tissue-specific and time-dependent tumorigenic effects of systemic hypoxia, which is limited to oxygen-sensitive and responsive cells, particularly during the postnatal period. This study supports connections between the EPAS1HIF2α genetic adaptation in human tumors developed under systemic hypoxia to populations living in high altitudes. The genetic adaptations in populations to different stressors can be explored further to understand tumorigenesis and tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fb7c10df0e202cb735340e16f27a752b3b7b5d8" target='_blank'>
              Convergent genetic adaptation in human tumors developed under systemic hypoxia and in populations living at high altitudes
              </a>
            </td>
          <td>
            Carlota Arenillas, José Ruiz-Cantador, Lucía Celada, Bruna Calsina, Eduardo García-Galea, Debayan Datta, Roberta Fasani, Ana Belén Moreno-Cárdenas, J. J. Alba-Linares, Berta Miranda, Á. Martínez-Montes, Cristina Álvarez-Escolá, Beatriz Lecumberri, Elvira Ana González García, Shahida K. Flores, Emmanuel Esquivel, Yanli Ding, Rita Maria Regojo Zapata, J. Pozo-Kreilinger, Carmela Iglesias, Trisha Dwight, Christopher Muir, A. O. Alday, M. E. Garrido-Lestache Rodríguez-Monte, M. D. del Cerro, Isaac Martínez-Bendayán, D. M. Lourenço, Maria Adelaide A. Pereira, N. Burnichon, A. Buffet, Craig Broberg, P. Dickson, M. F. Fraga, J. L. Llorente Pendás, Joaquín Rueda Soriano, Francisco Buendía Fuentes, Sergio P. A. Toledo, R. Clifton-Bligh, R. Dienstmann, Jaume Capdevila, A. Gimenez-Roqueplo, J. Favier, Donate Weghorn, P. Nuciforo, William F Young, A. Opotowsky, Anand Vaidya, Irina Bancos, Mercedes Robledo, A. Casterás, L. Dos-Subirà, María Dolores Chiara, Igor Adameyko, P. Dahia, R. Toledo
          </td>
          <td>2024-06-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Abstract Radiotherapy is a key treatment option for a wide variety of human tumors, employed either alone or alongside with other therapeutic interventions. Radiotherapy uses high-energy particles to destroy tumor cells, blocking their ability to divide and proliferate. The effectiveness of radiotherapy is due to genetic and epigenetic factors that determine how tumor cells respond to ionizing radiation. These factors contribute to the establishment of resistance to radiotherapy, which increases the risk of poor clinical prognosis of patients. Although the mechanisms by which tumor cells induce radioresistance are unclear, evidence points out several contributing factors including the overexpression of DNA repair systems, increased levels of reactive oxygen species, alterations in the tumor microenvironment, and enrichment of cancer stem cell populations. In this context, dysregulation of microRNAs or miRNAs, critical regulators of gene expression, may influence how tumors respond to radiation. There is increasing evidence that miRNAs may act as sensitizers or enhancers of radioresistance, regulating key processes such as the DNA damage response and the cell death signaling pathway. Furthermore, expression and activity of miRNAs have shown informative value in overcoming radiotherapy and long-term radiotoxicity, revealing their potential as biomarkers. In this review, we will discuss the molecular mechanisms associated with the response to radiotherapy and highlight the central role of miRNAs in regulating the molecular mechanisms responsible for cellular radioresistance. We will also review radio-miRs, radiotherapy-related miRNAs, either as sensitizers or enhancers of radioresistance that hold promise as biomarkers or pharmacological targets to sensitize radioresistant cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f11e14f281c48ffc0ef7d72af9b361e4a40e0c9" target='_blank'>
              Radio-miRs: a comprehensive view of radioresistance-related microRNAs
              </a>
            </td>
          <td>
            Abraham Pedroza-Torres, Sandra L Romero-Córdoba, S. Montaño, O. Peralta-Zaragoza, Dora Emma Vélez-Uriza, Cristian Arriaga-Canon, Xiadani Guajardo-Barreto, Diana Bautista-Sánchez, Rodrigo Sosa-León, Olivia Hernández-González, J. Díaz-Chávez, R. M. Álvarez-Gómez, Luis A Herrera
          </td>
          <td>2024-07-04</td>
          <td>Genetics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae3ef3973d270e32f48aaa19a00bef4e18bca5be" target='_blank'>
              Research Progress of Long Non-coding RNA-ZFAS1 in Malignant Tumors.
              </a>
            </td>
          <td>
            Xin Liu, Zhong Ma, Xianxu Zhang, Shicheng Li, Jiangdong An, Zhiqiang Luo
          </td>
          <td>2024-07-26</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="While the triggers for the metastatic transformation of breast cancer (BC) cells remain unknown, recent evidence suggests that intrinsic cellular metabolism could be a crucial driver of migratory disposition and chemoresistance. Aiming to decode the molecular mechanisms involved in BC cell metabolic maneuvering, we study how a ketomimetic (ketone body-rich, low glucose) medium affects Doxorubicin (DOX) susceptibility and invasive disposition of BC cells. We quantified glycocalyx sialylation and found an inverse correlation with DOX-induced cytotoxicity and DOX internalization. These measurements were coupled with single-cell metabolic imaging, bulk migration studies, along with transcriptomic and metabolomic analyses. Our findings revealed that a ketomimetic medium enhances chemoresistance and invasive disposition of BC cells via two main oncogenic pathways: hypersialylation and lipid synthesis. We propose that the crosstalk between these pathways, juxtaposed at the synthesis of the glycan precursor UDP-GlcNAc, furthers advancement of a metastatic phenotype in BC cells under ketomimetic conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fa6f2b5143747f6467f4d23175994fac910eade" target='_blank'>
              Ketomimetic Nutrients Trigger a Dual Metabolic Defense in Breast Cancer Cells
              </a>
            </td>
          <td>
            Mohini Kamra, Yuan-I Chen, Paula C. Delgado, Erin H. Seeley, S. Seidlits, Hsin-Chin Yeh, Amy Brock, S. Parekh
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Mutations in polymerases Pold1 and Pole exonuclease domains in humans are associated with increased cancer incidence, elevated tumor mutation burden (TMB) and response to immune checkpoint blockade (ICB). Although ICB is approved for treatment of several cancers, not all tumors with elevated TMB respond. Here we generated Pold1 and Pole proofreading mutator mice and show that ICB treatment of mice with high TMB tumors did not improve survival as only a subset of tumors responded. Similarly, introducing the mutator alleles into mice with Kras/p53 lung cancer did not improve survival, however, passaging mutator tumor cells in vitro without immune editing caused rejection in immune-competent hosts, demonstrating the efficiency by which cells with antigenic mutations are eliminated. Finally, ICB treatment of mutator mice earlier, before observable tumors delayed cancer onset, improved survival, and selected for tumors without aneuploidy, suggesting the use of ICB in individuals at high risk for cancer prevention. Highlights Germline somatic and conditional Pold1 and Pole exonuclease domain mutations in mice produce a mutator phenotype. Spontaneous cancers arise in mutator mice that have genomic features comparable to human tumors with these mutations. ICB treatment of mutator mice with tumors did not improve survival as only a subset of tumors respond. Introduction of the mutator alleles into an autochthonous mouse lung cancer model also did not produce immunogenic tumors, whereas passaging mutator tumor cells in vitro caused immune rejection indicating efficient selection against antigenic mutations in vivo. Prophylactic ICB treatment delayed cancer onset, improved survival, and selected for tumors with no aneuploidy. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9e6ba5a717c7dd3412aa2d3806c5dc666fb03f9" target='_blank'>
              Immune Checkpoint Blockade Delays Cancer and Extends Survival in Murine DNA Polymerase Mutator Syndromes
              </a>
            </td>
          <td>
            Akshada Sawant, Fuqian Shi, E. C. Lopes, Zhixian Hu, Somer Abdelfattah, Jennele Baul, Jesse Powers, Christian S. Hinrichs, Joshua D. Rabinowitz, Chang S. Chan, Edmund C. Lattime, Shridar Ganesan, Eileen White
          </td>
          <td>2024-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53c24f0b42670eaa7d910971701aacf3a8d5213b" target='_blank'>
              Targeting RBM39 through indisulam induced mis-splicing of mRNA to exert anti-cancer effects in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Tongting Ji, Yang Yang, Juanjuan Yu, Hongli Yin, Xinran Chu, Pengju Yang, Ling Xu, Xiaodong Wang, Shaoyan Hu, Yizhen Li, Xiaochen Wu, Wengyuan Liu, Bi Zhou, Wenjuan Wang, Shuqi Zhang, Wei Cheng, Yanling Chen, Lei Shi, Zhiheng Li, Ran Zhuo, Yongping Zhang, Yanfang Tao, Di Wu, Xiao-Lu Li, Zimu Zhang, Jun-Jie Fan, Jian Pan, Jun Lu
          </td>
          <td>2024-07-24</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Despite rapid advances in the field of immunotherapy, including the success of immune checkpoint inhibition in treating multiple cancer types, clinical response in high-grade gliomas (HGGs) has been disappointing. This has been in part attributed to the low tumor mutational burden (TMB) of the majority of HGGs. Hypermutation is a recently characterized glioma signature that occurs in a small subset of cases, which may open an avenue to immunotherapy. The substantially elevated TMB of these tumors most commonly results from alterations in the DNA mismatch repair pathway in the setting of extensive exposure to temozolomide or, less frequently, from inherited cancer predisposition syndromes. In this review, we discuss the genetics and etiology of hypermutation in HGGs, with an emphasis on the resulting genomic signatures, and the state and future directions of immuno-oncology research in these patient populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a2f2c2e78fe8478861279397510414edbdfa255" target='_blank'>
              Pathways to hypermutation in high-grade gliomas: Mechanisms, syndromes, and opportunities for immunotherapy.
              </a>
            </td>
          <td>
            Tuesday Haynes, Mark R Gilbert, Kevin Breen, Chunzhang Yang
          </td>
          <td>2024-06-24</td>
          <td>Neuro-oncology advances</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Introduction. It has been established that the presence of homologous recombination deficiency in a breast tumor is associated with the effectiveness of treatment. But despite the high chemosensitivity of the tumor to DNA-damaging agents, complete pathological responses to treatment are very rare. And this process may be based on a change in the somatic status of BRCA1, that is, a reversion and return of the wild-type allele occurs and the DNA repair function is restored.Aim. To evaluate changes in the presence of chromosomal aberrations and the expression profile of the main genes of homologous recombination in cell models of breast cancer under the influence of cisplatin and docetaxel.Materials and methods. The study was conducted on breast cancer tumor cell cultures: MCF-7, MDA-MB-231 and MDA-MB-468. A cell model of drug resistance was obtained for two drugs: cisplatin and docetaxel. RNA and DNA were isolated from cell suspension using the RNeasy Plus Mini Kit and QIAamp DNA Mini Kit (Qiagen, Germany), respectively. The expression level of homologous recombination genes was assessed using reverse transcription polymerase chain reaction. To assess the presence of chromosomal aberrations, microarray analysis was performed on DNA chips.Results. Restoration of normal copy number for the BRCA1, CDK12, CHEK1 and RAD51D genes in MCF-7 under the influence of cisplatin was shown. For BRCA2 and PALB2, amplifications were detected. A statistically significant increase in the expression of the BRCA1 (p = 0.04), BRCA2 (p = 0.02), PALB2 (p = 0.01) and RAD51D (p = 0.05) genes was also shown. MDAMB-231 shows that all identified loci with deletions, where the BRCA2, BARD1, CHEK2, PALB2 and RAD54L genes are localized, are restored to normal copy number by cisplatin. The appearance of amplifications was registered for BRCA1, BRIP1, FANCL, RAD51B, PARP1. A similar result was shown for docetaxel. An increase in the expression level is typical for the genes BRCA1 (p = 0.02), BRCA2 (p = 0.02), CHEK2 (p = 0.05), FANCL (p = 0.04), PALB2 (p = 0.05), RAD51C (p = 0.02), PARP1 (p = 0.02), which corresponds to the appearance of amplifications. In the MDA-MB-468 cell culture, an increase in the copy number of only the BRCA1 gene is observed. The effect of docetaxel has no effect on this cell culture. The level of BRCA1 expression increases in direct proportion to the duration of drug action.Conclusion. Thus, the study showed that under the influence of cisplatin, reversion of not only homologous recombination gene mutations, but also other disorders can occur.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/767d30042e565f8e61dafca9671915ee6d03ecf7" target='_blank'>
              Modulation of homologous recombination gene activity in breast tumor cells in an in vitro model
              </a>
            </td>
          <td>
            M. Tsyganov, A. Frolova, E. A. Kravtsova, I. A. Tsydenova, M. Ibragimova
          </td>
          <td>2024-06-28</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The treatment of unresectable metastatic colorectal cancer has evolved over the last two decades, as knowledge of cancer biology has broadened and new targets have emerged. ‘The Hallmarks of Cancer’ illustrate the crucial capabilities acquired by cells to become malignant and represent the evolution of knowledge of tumor biology. This review integrates these novel targets and therapies into selected hallmarks: sustaining proliferative signaling, inducing vasculature, avoiding immune destruction, genome instability and mutation, reprogramming cellular metabolism, and resisting cell death. The different strategies and combinations under study are based on treatments with anti-EGFR, anti-VEGF, and anti-HER2 agents, KRAS G12C inhibitors, BRAF and MEK inhibitors, and immune checkpoint inhibitors. However, new approaches are emerging, including vaccines, WEE1 inhibitors, and PARP inhibitors, among others. The further deciphering of cancer biology will unravel new targets, develop novel therapies, and improve patients’ outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37457f69d4f7628ddd7a4be9f921aa65a4201370" target='_blank'>
              Current and Emerging Treatment Paradigms in Colorectal Cancer: Integrating Hallmarks of Cancer
              </a>
            </td>
          <td>
            Clara Salva de Torres, I. Baraibar, N. Saoudi González, J. Ros, F. Salvà, Marta Rodríguez-Castells, Adriana Alcaráz, Ariadna García, J. Tabernero, E. Élez
          </td>
          <td>2024-06-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Caffeine is a natural compound that inhibits the major cellular signaling regulator TOR, leading to widespread effects including growth inhibition. S. cerevisiae yeast can adapt to tolerate high concentrations of caffeine in coffee and cacao fermentations and in experimental systems. While many factors affecting caffeine tolerance and TOR signaling have been identified, further characterization of their interactions and regulation remain to be studied. We used experimental evolution of S. cerevisiae to study the genetic contributions to caffeine tolerance in yeast, through a collaboration between high school students evolving yeast populations coupled with further research exploration in university labs. We identified multiple evolved yeast populations with mutations in PDR1 and PDR5, which contribute to multidrug resistance, and showed that gain-of-function mutations in multidrug resistance family transcription factors Pdr1, Pdr3, and Yrr1 differentially contribute to caffeine tolerance. We also identified loss-of-function mutations in TOR effectors Sit4, Sky1, and Tip41, and show that these mutations contribute to caffeine tolerance. These findings support the importance of both the multidrug resistance family and TOR signaling in caffeine tolerance, and can inform future exploration of networks affected by caffeine and other TOR inhibitors in model systems and industrial applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b45a6e64e528f9cc1b6535255dfd5db15ac79e7" target='_blank'>
              Experimental evolution of S. cerevisiae for caffeine tolerance alters multidrug resistance and TOR signaling pathways.
              </a>
            </td>
          <td>
            Renee C. Geck, Naomi G. Moresi, Leah M. Anderson, Isabel Addepalli, Deepti Aggarwal, Prisha Agnihotri, Ahlaam A Ali, Clara J. Amorosi, Abhinav Anand, Ashna Atukuri, Thang Awi, Insiya Basrai, Hitha Bathala, Sarang Bhide, Benjamin B Cantor, Jocelyn Cervantes, Tridib Chakraborty, James Champlin, Ameen Chbihi, Felicia Chen, Hayley Chenfang, Reagan Choi, S. Chokka, Julian D'Souza, Vivek Dandu, Arkesh Das, Margrette Dawoud, Victoria Dong, Riya Dutta, Graeme Edoff, Cecelia Fan, Rena Foo, Liam T Glanville, Cristian Golat, Suhavi Grewal, Faye Guan, Aarya Gurav, Aranav Gupta, Krish Gupta, Siya Gupta, Osman Hameed, Rhea Hede-Sakhardande, Nushaba Hossain, Youssef Ibrahim, Jemi Isaac, Udayvir Jalf, Medha Jasti, Amar Jazvin, Okichy Johnny, Vidhi Kamat, Venya Kandula, Lekhana Katuri, Keabe E Kebede, Om Khuperkar, Emily Kim, R. Konduru, Salimah Kyaw, Daniel Lee, Tian Syun Lin, Karen Luo, Jwan Magsoosi, Mlahat Mahmood, Ronald Brent F Marzan, Noyonima Masud, Jessica Mathew, Ava Miciuda, Trevor Morey, Anagha Nair, Naveen Natarajan, Aahil Abdul Nazeer, Usoatua Levei P Noa, Shashank Pagadala, Hamin Paik, John Palomino, Kush Parikh, Naisha Phadke, Michelle V Phan, Britta Pingree, Neal Podhuturi, Arya Prasad, Sonia Puri, Sanjini Rajkumar, Ananya Ramanan, Elliot M Russell, Zachary L Saad, Magdalena Sabalsa Gaytan, Francis L Salazar, Anjali Sanil, Neespruha Shah, Mustafa Sharba, Prihensha Sharma, Sophia Showman, S. Showman, Heejin Shyn, Aryan Singh, Saakshi Sovani, Shreya Srugaram, Rachel Stroia, Sanjana Sunilkumar, Nihil Suthy, Asma Syed, Ruthesh Thavamani, Nitya Upadhye, Rebecca Varghese, Annie Wang, Cynthia Wang, Roger Wang, Miya A Watson, Theresa Wei, Myra L Woody, Nancy Yao, Tyler Yee, Chiann-Ling Cindy Yeh, Jungbin Yoon, Jiaying Zhou, Tianhui Zhu, Noah Fredstrom, Sandra Pennington, Scarlett Counihan, Owen Burris, Marisol Jimenez Garcia, Dennis Godin, Rebecca Brewer, Timothy R. Renz, M. Taylor, Maitreya J. Dunham
          </td>
          <td>2024-07-11</td>
          <td>G3</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Breast cancer is a heterogeneous disease that arises as a multi-stage process involving multiple cell types. Patients diagnosed with the same clinical stage and pathological classification may have different prognoses and therapeutic responses due to alterations in molecular genetics. As an essential marker for the molecular subtyping of breast cancer, long non-coding RNAs (lncRNAs) play a crucial role in gene expression regulation, cell differentiation, and the maintenance of genomic stability. Here, we developed a modular framework for lncRNA identification and applied it to a breast cancer cohort to identify novel lncRNAs not previously annotated. To investigate the potential biological function, regulatory mechanisms, and clinical relevance of the novel lncRNAs, we elucidated the genomic and chromatin features of these lncRNAs, along with the associated protein-coding genes and putative enhancers involved in the breast cancer regulatory networks. Furthermore, we uncovered that the expression patterns of novel and annotated lncRNAs identified in breast cancer were related to the hormone response in the PAM50 subtyping criterion, as well as the immune response and progression states of breast cancer across different immune cells and immune checkpoint genes. Collectively, the comprehensive identification and functional analysis of lncRNAs revealed that these lncRNAs play an essential role in breast cancer by altering gene expression and participating in the regulatory networks, contributing to a better insight into breast cancer heterogeneity and potential avenues for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b6bf72d9207f000ec46937ff3a3ba19091f1e49" target='_blank'>
              Comprehensive Transcriptome Analysis Expands lncRNA Functional Profiles in Breast Cancer
              </a>
            </td>
          <td>
            Wenyong Zhu, Hao Huang, Zixi Hu, Yu Gu, Rong Zhang, Huiling Shu, Hongjia Liu, Xiao Sun
          </td>
          <td>2024-08-02</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="OBJECTIVE
Acute Lymphoblastic Leukemia (ALL) is the most common malignancy occurring in children. Copy number alterations (CNA) like PAX5, CDKN2A/2B, PAR1 Region, ETV6, IKZF1, BTG1, and RB1 gene deletion are important genetic events that define and prognosticate B-cell ALL. Thus, this study aimed to evaluate associations of CNA with induction phase remission status in childhood B-cell ALL.


METHODS
This study was observational with a cross-sectional design at the Dharmais Cancer Hospital, Harapan Kita Mother and Children Hospital, and Tangerang Regional Public Hospital. We evaluated 74 pediatric B-cell ALL cases with 1-18-year-olds. Genomic DNA was analyzed by Multiplex Ligation Dependent Probe Amplification Assay (MLPA). This study used the P335 ALL-IKZF1 panel kit, which contains several ALL-related genes. The patient's clinical and laboratory characteristics were collected from medical records from January to December 2019.


RESULT
We observed gene copy number alteration in children with B-Cell ALL. PAX5 was the most commonly observed gene deletion, followed by CDKN21/2B, ETV6, IKZF1, BTG1, RB1, and PAR1 Region. Based on gene mutations, only the PAX5 had a significant association with the remission status of pediatric B-cell ALL (p-value <0.05; OR = 3.91). It showed that patients with PAX5 gene mutations have 3.9 times the risk of no remission and/or relapse compared to those without PAX5 gene mutations.


CONCLUSION
Patients with mutations in the PAX5 gene have a higher chance of not achieving remission and/or experiencing relapse than those without such mutations. The MLPA method can be utilized for examining copy number alterations, which is valuable for achieving more precise stratification in diagnosis.. Further research is needed to expand upon this finding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bd752c14237c4bf075eb8efe30658a5f3ec960c" target='_blank'>
              Unraveling Copy Number Alterations in Pediatric B-Cell Acute Lymphoblastic Leukemia: Correlation with Induction Phase Remission Using MLPA.
              </a>
            </td>
          <td>
            M. Aisyi, Murti Andriastuti, A. Kosasih, Ahmad Rusdan Handoyo Utomo, Fahreza Saputra, Teny Tjitra Sari, H. Sjakti, Fifi Dwijayanti, K. Harimurti, D. Gatot
          </td>
          <td>2024-07-01</td>
          <td>Asian Pacific journal of cancer prevention : APJCP</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/321126b0cb5f073f8273850a71ea2ffc1df84ce8" target='_blank'>
              Deciphering the role of post-translational modifications in fanconi anemia proteins and their influence on tumorigenesis.
              </a>
            </td>
          <td>
            Rui Ma, Xinlin Xu
          </td>
          <td>2024-06-15</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND
The DNA mismatch repair (MMR) system serves as a sophisticated guardian of the precise functioning of the human genome. Dysregulation within this system is linked to the oncogenesis process. Reduced expression of MMR system proteins identified in salivary gland tumors (SGTs) suggests an increased risk of tumoral occurrence. This study aims to analyze the expression of MMR proteins in SGTs and discuss the relevance of this association to the development of these neoplasms.


MATERIAL AND METHODS
This review was conducted following the PRISMA guidelines and was registered in PROSPERO (CRD42023465590). A comprehensive search of the PubMed/MEDLINE, Web of Science, Scopus, Embase, and ProQuest (non-peer reviewed platform) was performed to answer the question "Do DNA MMR system proteins exhibit expression in SGTs?". The methodological quality of the selected studies was assessed using the JBI's Critical Appraisal Tool.


RESULTS
A total of 142 patients with benign SGTs and 84 with malignant SGTs were included in this review. The literature analysis showed a notable reduction in the expression of DNA MMR system proteins (hHMS2, hMLH1, hMSH3 and hMSH6) in the percentage of marked cells.


CONCLUSIONS
The reduction in the expression of the DNA MMR system proteins suggests an interesting correlation with the development of malignant and benign SGTs. Nevertheless, further investigations are warranted to better clarify the precision of measuring biomarker protein expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfbcf6ccd15abbdea0ae296578920af07be7eb3d" target='_blank'>
              DNA mismatch repair system expression in salivary gland tumors: A Systematic Review.
              </a>
            </td>
          <td>
            G-M-T-T Alves, R-J Severino-Lazo, G-K Amaral-Silva, F-M Silveira, M-D Carvalho
          </td>
          <td>2024-06-22</td>
          <td>Medicina oral, patologia oral y cirugia bucal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="INTRODUCTION
Defining the chromosomal and molecular changes associated with myeloid neoplasms (MNs) optimizes clinical care through improved diagnosis, prognosis, treatment planning, and patient monitoring. This review will concisely describe the techniques used to profile MNs clinically today, with descriptions of challenges and emerging approaches that may soon become standard-of-care.


AREAS COVERED
In this review, the authors discuss molecular assessment of MNs using non-sequencing techniques, including conventional cytogenetic analysis, fluorescence in situ hybridization, chromosomal genomic microarray testing; as well as DNA- or RNA-based next-generation sequencing (NGS) assays; and sequential monitoring via digital PCR or measurable residual disease assays. The authors explain why distinguishing somatic from germline alleles is critical for optimal management. Finally, they introduce emerging technologies, such as long-read, whole exome/genome, and single-cell sequencing, which are reserved for research purposes currently but will become clinical tests soon.


EXPERT OPINION
The authors describe challenges to the adoption of comprehensive genomic tests for those in resource-constrained environments and for inclusion into clinical trials. In the future, all aspects of patient care will likely be influenced by the adaptation of artificial intelligence and mathematical modeling, fueled by rapid advances in telecommunications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce5821258a38d2a15a1cce4cdbe2844d4e8f9b60" target='_blank'>
              The impact of next-generation sequencing for diagnosis and disease understanding of myeloid malignancies.
              </a>
            </td>
          <td>
            Erica Vormittag-Nocito, M. Sukhanova, Lucy A. Godley
          </td>
          <td>2024-07-25</td>
          <td>Expert review of molecular diagnostics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/081cebd6c13932575bbecec80f6933eed0eb38d1" target='_blank'>
              Genetic drivers and cellular selection of female mosaic X chromosome loss.
              </a>
            </td>
          <td>
            Aoxing Liu, G. Genovese, Yajie Zhao, M. Pirinen, S. Zekavat, K. Kentistou, Zhiyu Yang, K. Yu, C. Vlasschaert, Xiaoxi Liu, Derek W. Brown, G. Hudjashov, Bryan R. Gorman, J. Dennis, Weiyin Zhou, Y. Momozawa, S. Pyarajan, Valdislav Tuzov, Fanny-Dhelia Pajuste, M. Aavikko, Timo P. Sipilä, A. Ghazal, Wenbin Huang, Neal D. Freedman, L. Song, Eugene J. Gardner, V. Sankaran, A. Palotie, Hanna M. Ollila, T. Tukiainen, S. Chanock, R. Mägi, Pradeep Natarajan, M. Daly, A. Bick, S. Mccarroll, Chikashi Terao, Po-Ru Loh, A. Ganna, J. R. Perry, M. Machiela
          </td>
          <td>2024-06-12</td>
          <td>Nature</td>
          <td>0</td>
          <td>138</td>
        </tr>

        <tr id="Emerging immunotherapies such as immune checkpoint blockade (ICB) and chimeric antigen receptor T-cell (CAR-T) therapy have revolutionized cancer treatment and have improved the survival of patients with multiple cancer types. Despite this success many patients are unresponsive to these treatments or relapse following treatment. CRISPR activation and knockout (KO) screens have been used to identify novel single gene targets that can enhance effector T cell function and promote immune cell targeting and eradication of tumors. However, cancer cells often employ multiple genes to promote an immunosuppressive pathway and thus modulating individual genes often has a limited effect. Paralogs are genes that originate from common ancestors and retain similar functions. They often have complex effects on a particular phenotype depending on factors like gene family similarity, each individual gene’s expression and the physiological or pathological context. Some paralogs exhibit synthetic lethal interactions in cancer cell survival; however, a thorough investigation of paralog pairs that could enhance the efficacy of cancer immunotherapy is lacking. Here we introduce a sensitive computational approach that uses sgRNA sets enrichment analysis to identify cancer-intrinsic paralog pairs which have the potential to synergistically enhance T cell-mediated tumor destruction. We have further developed an ensemble learning model that uses an XGBoost classifier and incorporates features such as gene characteristics, sequence and structural similarities, protein-protein interaction (PPI) networks, and gene coevolution data to predict paralog pairs that are likely to enhance immunotherapy efficacy. We experimentally validated the functional significance of these predicted paralog pairs using double knockout (DKO) of identified paralog gene pairs as compared to single gene knockouts (SKOs). These data and analyses collectively provide a sensitive approach to identify previously undetected paralog pairs that can enhance cancer immunotherapy even when individual genes within the pair has a limited effect.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9398918df5088c04d21b0123727e4ddeb607a57" target='_blank'>
              Sensitive detection of synthetic response to cancer immunotherapy driven by gene paralog pairs
              </a>
            </td>
          <td>
            Chuanpeng Dong, Feifei Zhang, Emily He, Ping Ren, Nipun Verma, Xinxin Zhu, Di Feng, Hongyu Zhao, Sidi Chen
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Ovarian cancer is a deadly female cancer with high rates of recurrence. The primary treatment strategy for patients is platinum-based therapy regimens that almost universally develop resistance. Consequently, new therapeutic avenues are needed to overcome the plateau that current therapies have on patient outcomes. We describe a gene amplification involving both HSF1 and MYC, wherein these two genes on chromosome 8q are co-amplified in over 7% of human tumors that is enriched to over 30% of patients with ovarian cancer. We further found that HSF1 and MYC transcriptional activity is correlated in human tumors and ovarian cancer cell lines, suggesting they may cooperate in ovarian cancer cells. CUT&RUN for HSF1 and MYC in co-amplified ovarian cancer cells revealed that HSF1 and MYC have overlapping binding at a substantial number of locations throughout the genome where their binding peaks are near identical. Consistent with these data, a protein-protein interaction between HSF1 and MYC was detected in ovarian cancer cells, implying these two transcription factors have a molecular cooperation. Further supporting their cooperation, growth of HSF1-MYC co-amplified ovarian cancer cells were found to be dependent on both HSF1 and MYC. In an attempt to identify a therapeutic target that could take advantage of this dependency on both HSF1 and MYC, PLK1 was identified as being correlated with HSF1 and MYC in primary human tumor specimens, consistent with a previously established effect of PLK1 on HSF1 and MYC protein levels. Targeting PLK1 with the compound volasertib (BI-6727) revealed a greater than 200-fold increased potency of volasertib in HSF1-MYC co-amplified ovarian cancer cells compared to ovarian cancer cells wild-type HSF1 and MYC copy number, which extended to several growth assays, including spheroid growth. Volasertib, and other PLK1 inhibitors, have not shown great success in clinical trials and this study suggests that targeting PLK1 may be viable in a precision medicine approach using HSF1-MYC co-amplification as a biomarker for response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21e2e5303aa923a4a45241efa9a5394dc2df5936" target='_blank'>
              MYC and HSF1 Cooperate to Drive PLK1 inhibitor Sensitivity in High Grade Serous Ovarian Cancer
              </a>
            </td>
          <td>
            Imade Williams, Haddie DeHart, Matthew O’Malley, Bobby Walker, Vrushabh Ulhaskumar, Haimanti Ray, Joe R. Delaney, K. Nephew, Richard L. Carpenter
          </td>
          <td>2024-06-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64dd346087b1d64ab0e5ccbf0c92c3994e5f755a" target='_blank'>
              Characterization and regulation of cell cycle-independent noncanonical gene targeting
              </a>
            </td>
          <td>
            Shinta Saito, N. Adachi
          </td>
          <td>2024-06-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06e722e4e423764d35e29b66ac1408c4a29aa4fd" target='_blank'>
              Targeting SOX4/PCK2 signaling suppresses neuroendocrine trans-differentiation of castration-resistant prostate cancer
              </a>
            </td>
          <td>
            Nan Jing, Zhenkeke Tao, Xinxing Du, Zhenzhen Wen, Wei-Qiang Gao, Baijun Dong, Yu-Xiang Fang
          </td>
          <td>2024-07-16</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21a2a05c7ef392ef903b88cba17ae6f2c9311cf5" target='_blank'>
              Mechanistic patterns and clinical implications of oncogenic tyrosine kinase fusions in human cancers
              </a>
            </td>
          <td>
            T. Cheong, Ahram Jang, Qi Wang, Giulia Leonardi, B. Ricciuti, J. Alessi, Alessandro Di Federico, Mark M Awad, Maria K Lehtinen, Marian H Harris, Roberto Chiarle
          </td>
          <td>2024-06-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ee3ef93f75d18cca805d005d653d62f6f76b88e" target='_blank'>
              Comprehensive pan-cancer analysis reveals that C5orf34 regulates the proliferation and mortality of lung cancer.
              </a>
            </td>
          <td>
            Mengmeng Yang, Yuhan Deng, Yu Ma, Chunli Song, Zhenhua Wu, Xiayimaierdan Yibulayin, Xiaohong Sun, Yunquan Guo, Dan He
          </td>
          <td>2024-06-29</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cae8abbc9e79803db49d453f861eb5943682ed07" target='_blank'>
              Mitochondrial genetics through the lens of single-cell multi-omics.
              </a>
            </td>
          <td>
            Lena Nitsch, Caleb A. Lareau, Leif S. Ludwig
          </td>
          <td>2024-07-01</td>
          <td>Nature genetics</td>
          <td>2</td>
          <td>23</td>
        </tr>

        <tr id="We describe exciting recent advances in fusion-driven sarcoma etiology, from an epigenetics perspective. By exploring the current state of the field, we identify and describe the central mechanisms that determine sarcomagenesis. Further, we discuss seminal studies in translational genomics, which enabled epigenetic characterization of fusion-driven sarcomas. Important context for epigenetic mechanisms include, but are not limited to, cell cycle and metabolism, core regulatory circuitry, 3-dimensional chromatin architectural dysregulation, integration with ATP-dependent chromatin remodeling, and translational animal modeling. Paradoxically, while the genetic requirements for oncogenic transformation are highly specific for the fusion partners, the epigenetic mechanisms we as a community have uncovered are categorically very broad. This dichotomy prompts the question of whether the investigation of rare disease epigenomics should prioritize studying individual cell populations, thereby examining whether the mechanisms of chromatin dysregulation are specific to a particular tumor. We review recent advances focusing on rhabdomyosarcoma, synovial sarcoma, alveolar soft part sarcoma, clear cell sarcoma, undifferentiated round cell sarcoma, Ewing sarcoma, myxoid/round liposarcoma, epithelioid hemangioendothelioma and desmoplastic round cell tumor. The growing number of groundbreaking discoveries in the field, motivated us to anticipate further exciting advances in the area of mechanistic epigenomics and direct targeting of fusion transcription factors in the years ahead.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01bc563cf3c1cf1b5a64e47d7f0fdef6c20adb4b" target='_blank'>
              Epigenetic determinants of fusion-driven sarcomas: paradigms and challenges
              </a>
            </td>
          <td>
            Benjamin Z. Stanton, S. Pomella
          </td>
          <td>2024-06-14</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/729c3507158456cf0b6b747d89cf47634b551b4c" target='_blank'>
              BUB1B monoallelic germline variants contribute to prostate cancer predisposition by triggering chromosomal instability
              </a>
            </td>
          <td>
            Maria P. Silva, Luísa T Ferreira, Natércia F. Brás, L. Torres, Andreia Brandão, M. Pinheiro, Marta Cardoso, Adriana Resende, Joana Vieira, Carlos Palmeira, Gabriela Martins, Miguel Silva, C. Pinto, Ana Peixoto, João Silva, Rui Henrique, S. Maia, H. Maiato, Manuel R. Teixeira, P. Paulo
          </td>
          <td>2024-07-16</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a976d04ebae96418666fe6d36b4350c7aceba4b4" target='_blank'>
              Single-cell transcriptomic insights into chemotherapy-induced remodeling of the osteosarcoma tumor microenvironment
              </a>
            </td>
          <td>
            Xuejing Zheng, Wence Wu, Zhen-guo Zhao, Xin-xin Zhang, Shengji Yu
          </td>
          <td>2024-07-20</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Colorectal carcinoma (CRC) is a heterogeneous group of tumors with varying therapeutic response and prognosis, and evidence suggests the tumor immune microenvironment (TIME) plays a pivotal role. Using advanced molecular and spatial biology technologies, we aimed to evaluate the TIME in patients with CRC to determine whether specific alterations in the immune composition correlated with prognosis. We identified primary and metastatic tumor samples from 31 consented patients, which were profiled with whole-exome sequencing and bulk RNA-seq. Immune cell deconvolution followed by gene set enrichment analysis and unsupervised clustering was performed. A subset of tumors underwent in situ analysis of the TIME spatial composition at single-cell resolution through Imaging Mass Mass Cytometry. Gene set enrichment analysis revealed two distinct groups of advanced CRC, one with an immune activated phenotype and the other with a suppressed immune microenvironment. The activated TIME phenotype contained increased Th1 cells, activated dendritic cells, tertiary lymphoid structures, and higher counts of CD8+ T cells whereas the inactive or suppressed TIME contained increased macrophages and a higher M2/M1 ratio. Our findings were further supported by RNA-seq data analysis from the TCGA CRC database, in which unsupervised clustering also identified two separate groups. The immunosuppressed CRC TIME had a lower overall survival probability (HR 1.66, p=0.007). This study supports the pertinent role of the CRC immune microenvironment in tumor progression and patient prognosis. We characterized the immune cell composition to better understand the complexity and vital role that immune activity states of the TIME play in determining patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c69179f505eff3fb4390c601ab14e625f208dae3" target='_blank'>
              Integrative Transcriptomic and Single-Cell Protein Characterization of Colorectal Carcinoma Delineates Distinct Tumor Immune Microenvironments Associated with Overall Survival
              </a>
            </td>
          <td>
            Erika Hissong, B. Bhinder, Junbum Kim, Kentaro Ohara, H. Ravichandran, M. Assaad, Sarah Elsoukkary, Michael Shusterman, Uqba Khan, K. Eng, R. Bareja, J. Manohar, M. Sigouros, A. Rendeiro, Jose Jessurun, Allyson J. Ocean, A. Sboner, Olivier Elemento, Juan Miguel Mosquera, 
          </td>
          <td>2024-07-25</td>
          <td>Research Square</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="The aim of this study was to evaluate the mutation spectrum of homologous recombination repair (HRR) genes and its association with tumor immune infiltration and prognosis in triple-negative breast cancer (TNBC). TNBC patients (434 patients from Ruijin cohort) were evaluated with targeted next-generating sequencing for mutations in HRR genes. The frequencies of mutations were compared with public reference cohorts (320 TNBC patients from METABRIC, 105 from TCGA, and 225 from MSKCC 2018). Associations between mutation status and tumor immune infiltration and prognosis were analyzed. HRR genes mutations were seen in 21.89% patients, with BRCA1/2 mutations significantly enriched in tumors with breast/ovarian cancer family history (P = 0.025) and high Ki-67 levels (P = 0.018). HRR genes mutations were not related with recurrence-free survival (RFS) (adjusted P = 0.070) and overall survival (OS) (adjusted P = 0.318) for TNBC patients, regardless of carboplatin treatment (P > 0.05). Moreover, tumor immune infiltration and PD-L1 expression was positively associated with HRR or BRCA1/2 mutation (all P < 0.001). Patients with both HRR mutation and high CD8+ T cell counts had the best RFS and OS, whereas patients with no HRR mutation and low CD8+ T cell counts had the worst outcomes (RFS P < 0.001, OS P = 0.019). High frequency of HRR gene mutations was found in early TNBC, with no prognostic significance. Immune infiltration and PD-L1 expression was positively associated with HRR mutation, and both HRR mutation and high CD8+ T cell infiltration levels were associated with superior disease outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/754b9595e6f9b757c8c2106ab35e651d7b39b608" target='_blank'>
              Association of tumor immune infiltration and prognosis with homologous recombination repair genes mutations in early triple-negative breast cancer
              </a>
            </td>
          <td>
            Zheng Wang, A. Li, Yujie Lu, Mengyuan Han, Miao Ruan, Chaofu Wang, Xiaotian Zhang, Changbin Zhu, Kunwei Shen, Lei Dong, Xiao-song Chen
          </td>
          <td>2024-07-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Mesenchymal stem cells (MSCs) have long been postulated as an important source cell in regenerative medicine. During subculture expansion, mesenchymal stem cell (MSC) senescence diminishes their multi-differentiation capabilities, leading to a loss of therapeutic potential. Up to date, the extrachromosomal circular DNAs (eccDNAs) have been demonstrated to be involved in senescence but the roles of eccDNAs during MSC.Here we explored eccDNA profiles in human bone marrow MSCs (BM-MSCs). EccDNA and mRNA was purified and sequenced, followed by quantification and functional annotation. Moreover, we mapped our datasets with the downloading enhancer and transcription factor-regulated genes to explore the potential role of eccDNAs.Sequentially, gene annotation analysis revealed that the majority of eccDNA were mapped in the intron regions with limited BM-MSC enhancer overlaps. We discovered that these eccDNA motifs in senescent BMSCs acted as motifs for binding transcription factors (TFs) of senescence-related genes.These findings are highly significant for identifying biomarkers of senescence and therapeutic targets in mesenchymal stem cells (MSCs) for future clinical applications. The potential of eccDNA as a stable therapeutic target for senescence-related disorders warrants further investigation, particularly exploring chemically synthesized eccDNAs as transcription factor regulatory elements to reverse cellular senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0b0fc0e697c338e4e8ee6ebc06de54d45231d9a" target='_blank'>
              Genome-wide sequencing identified extrachromosomal circular DNA as a transcription factor-binding motif of the senescence genes that govern replicative senescence in human mesenchymal stem cells
              </a>
            </td>
          <td>
            Wei Yang, Wei Ji, Boyu Liao, Zhongbo Li, Jian Wang, Haishu Lin, Jingbo Wang, Qian He
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cellular Neuroscience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Lung cancer is a leading cause of cancer-related death worldwide in both men and women, however mortality in the US and EU are recently declining in parallel with the gradual cut of smoking prevalence. Consequently, the relative frequency of adenocarcinoma increased while that of squamous and small cell carcinomas declined. During the last two decades a plethora of targeted drug therapies have appeared for the treatment of metastasizing non-small cell lung carcinomas (NSCLC). Personalized oncology aims to precisely match patients to treatments with the highest potential of success. Extensive research is done to introduce biomarkers which can predict the effectiveness of a specific targeted therapeutic approach. The EGFR signaling pathway includes several sufficient targets for the treatment of human cancers including NSCLC. Lung adenocarcinoma may harbor both activating and resistance mutations of the EGFR gene, and further, mutations of KRAS and BRAF oncogenes. Less frequent but targetable genetic alterations include ALK, ROS1, RET gene rearrangements, and various alterations of MET proto-oncogene. In addition, the importance of anti-tumor immunity and of tumor microenvironment has become evident recently. Accumulation of mutations generally trigger tumor specific immune defense, but immune protection may be upregulated as an aggressive feature. The blockade of immune checkpoints results in potential reactivation of tumor cell killing and induces significant tumor regression in various tumor types, such as lung carcinoma. Therapeutic responses to anti PD1-PD-L1 treatment may correlate with the expression of PD-L1 by tumor cells. Due to the wide range of diagnostic and predictive features in lung cancer a plenty of tests are required from a single small biopsy or cytology specimen, which is challenged by major issues of sample quantity and quality. Thus, the efficacy of biomarker testing should be warranted by standardized policy and optimal material usage. In this review we aim to discuss major targeted therapy-related biomarkers in NSCLC and testing possibilities comprehensively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/009f3a9ab76c5408a3ffa568b375af58c7a758c0" target='_blank'>
              The rapidly changing field of predictive biomarkers of non-small cell lung cancer
              </a>
            </td>
          <td>
            László József Tóth, A. Mokánszki, Gábor Méhes
          </td>
          <td>2024-06-17</td>
          <td>Pathology and Oncology Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) has a poor survival rate for both pediatric and adult patients due to its frequent relapse. To elucidate the bioenergetic principle underlying AML relapse, we investigated the transcriptional regulation of mitochondrial–nuclear dual genomes responsible for metabolic plasticity in treatment-resistant blasts. Both the gain and loss of function results demonstrated that NFκB2, a noncanonical transcription factor (TF) of the NFκB (nuclear factor kappa-light-chain-enhancer of activated B cells) family, can control the expression of TFAM (mitochondrial transcription factor A), which is known to be essential for metabolic biogenesis. Furthermore, genetic tracking and promoter assays revealed that NFκB2 is in the mitochondria and can bind the specific “TTGGGGGGTG” region of the regulatory D-loop domain to activate the light-strand promoter (LSP) and heavy-strand promoter 1 (HSP1), promoters of the mitochondrial genome. Based on our discovery of NFκB2′s novel function of regulating mitochondrial–nuclear dual genomes, we explored a novel triplet therapy including inhibitors of NFκB2, tyrosine kinase, and mitochondrial ATP synthase that effectively eliminated primary AML blasts with mutations of the FMS-related receptor tyrosine kinase 3 (FLT3) and displayed minimum toxicity to control cells ex vivo. As such, effective treatments for AML must include strong inhibitory actions on the dual genomes mediating metabolic plasticity to improve leukemia prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd91cb6b195c34903e27bb3147d1ba2db984023c" target='_blank'>
              Discovery of NFκB2-Coordinated Dual Regulation of Mitochondrial and Nuclear Genomes Leads to an Effective Therapy for Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Yi Xu, David J. Baylink, Jeffrey Xiao, Lily Tran, Vinh Nguyen, Brandon Park, Ismael Valladares, Scott Lee, Kevin Codorniz, Laren D Tan, Chien-Shing Chen, Hisham Abdel-Azim, Mark E. Reeves, Hamid Mirshahidi, Guido Marcucci, H. Cao
          </td>
          <td>2024-08-05</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e4d778cae3d7e3203dfad059efb7f4865240dc4" target='_blank'>
              Insights into treatment-specific prognostic somatic mutations in NSCLC from the AACR NSCLC GENIE BPC cohort analysis
              </a>
            </td>
          <td>
            Yi Liu, Sindhu Yalavarthi, Fan Yang, Yusif Abdul-Rashid, Shenkun Tang, Zihe Long, Yongkai Qin, Kerui Wu, Zhifei Wang
          </td>
          <td>2024-07-02</td>
          <td>BMC Pulmonary Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genome editing technologies have seen remarkable progress in recent years, enabling precise regulation of exogenous and endogenous genes. These advances have been extensively applied to the engineering of human T lymphocytes, leading to the development of practice changing therapies for patients with cancer and the promise of synthetic immune cell therapies for a variety of non-malignant diseases. Many distinct conceptual and technical approaches have been used to edit T-cell genomes, however targeted assessments of which techniques are most effective for manufacturing, gene editing and transgene expression are rarely reported. Through extensive comparative evaluation, we identified methods that most effectively enhance engineering of research-scale and pre-clinical T-cell products at critical stages of manufacturing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf8cdbd85fcdf596eac3e46738e8defa5a7f38e6" target='_blank'>
              Identification of core techniques that enhance genome editing of human T cells expressing synthetic antigen receptors.
              </a>
            </td>
          <td>
            Ju-Fang Chang, Nils Wellhausen, Nils W Engel, Jack H Landmann, Caitlin R. Hopkins, January Salas-McKee, Adham S Bear, M. E. Selli, S. Agarwal, J. Jadlowsky, Gerald P. Linette, Saar Gill, Carl H. June, J. Fraietta, Nathan Singh
          </td>
          <td>2024-06-13</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is an important driver of tumor evolution and therapy response. Advances in precision cancer treatment will require understanding of mutation clonality and subclonal architecture. Currently the slow computational speed of subclonal reconstruction hinders large cohort studies. To overcome this bottleneck, we developed Clonal structure identification through Pairwise Penalization, or CliPP, which clusters subclonal mutations using a regularized likelihood model. CliPP reliably processed whole-genome and whole-exome sequencing data from over 12,000 tumor samples within 24 hours, thus enabling large-scale downstream association analyses between subclonal structures and clinical outcomes. Through a pan-cancer investigation of 7,827 tumors from 32 cancer types, we found that high subclonal mutational load (sML), a measure of latency time in tumor evolution, was significantly associated with better patient outcomes in 16 cancer types with low to moderate tumor mutation burden (TMB). In a cohort of prostate cancer patients participating in an immunotherapy clinical trial, high sML was indicative of favorable response to immune checkpoint blockade. This comprehensive study using CliPP underscores sML as a key feature of cancer. sML may be essential for linking mutation dynamics with immunotherapy response in the large population of non-high TMB cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eca4fd8b4bb1bf5ae8673767195fa7c9bf0f23c" target='_blank'>
              Pan-cancer subclonal mutation analysis of 7,827 tumors predicts clinical outcome
              </a>
            </td>
          <td>
            Yujie Jiang, Matthew D. Montierth, Kaixian Yu, Shuangxi Ji, Shuai Guo, Quang Tran, Xiaoqian Liu, Seung Jun Shin, S. Cao, Ruonan Li, Yuxin Tang, T. Lesluyes, S. Kopetz, Jaffer J. Ajani, P. Msaouel, S. Subudhi, Ana Aparicio, Padmanee Sharma, J. P. Shen, Anil K. Sood, M. Tarabichi, Jennifer R. Wang, Marek Kimmel, P. Van Loo, Hongtu Zhu, Wenyi Wang
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="G-quadruplex (G4) sequences, which can fold into higher-order G4 structures, are abundant in the human genome and are over-represented in the promoter regions of many genes involved in human cancer initiation, progression, and metastasis. They are plausible targets for G4-binding small molecules, which would, in the case of promoter G4s, result in the transcriptional downregulation of these genes. However, structural information is currently available on only a very small number of G4s and their ligand complexes. This limitation, coupled with the currently restricted information on the G4-containing genes involved in most complex human cancers, has led to the development of a phenotypic-led approach to G4 ligand drug discovery. This approach was illustrated by the discovery of several generations of tri- and tetra-substituted naphthalene diimide (ND) ligands that were found to show potent growth inhibition in pancreatic cancer cell lines and are active in in vivo models for this hard-to-treat disease. The cycles of discovery have culminated in a highly potent tetra-substituted ND derivative, QN-302, which is currently being evaluated in a Phase 1 clinical trial. The major genes whose expression has been down-regulated by QN-302 are presented here: all contain G4 propensity and have been found to be up-regulated in human pancreatic cancer. Some of these genes are also upregulated in other human cancers, supporting the hypothesis that QN-302 is a pan-G4 drug of potential utility beyond pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f12d7617e98441c66bfb53bc671f9147eed0b1fe" target='_blank'>
              A Phenotypic Approach to the Discovery of Potent G-Quadruplex Targeted Drugs
              </a>
            </td>
          <td>
            Stephen Neidle
          </td>
          <td>2024-08-01</td>
          <td>Molecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary The atavistic theory of cancer suggests that cancer development proceeds through cells reverting to ancient survival mechanisms. The Serial Atavism Model (SAM) expands on this, proposing that cancer progresses through multiple stages of reversion to earlier evolutionary forms with cells losing modern traits and regaining primitive ones. One example is the Warburg effect, where cancer cells prefer a type of energy production used by ancient cells before Earth’s atmosphere had oxygen. However, this review argues that cancer metabolism is too complex to be fully explained by this theory. Cancer cells exhibit a wide range of metabolic behaviors that do not fit neatly into a pattern of reverting to an ancient state, indicating that the SAM may not provide a complete understanding of cancer. Abstract The atavistic theory of cancer posits that cancer emerges and progresses through the reversion of cellular phenotypes to more ancestral types with genomic and epigenetic changes deactivating recently evolved genetic modules and activating ancient survival mechanisms. This theory aims at explaining the known cancer hallmarks and the paradox of cancer’s predictable progression despite the randomness of genetic mutations. Lineweaver and colleagues recently proposed the Serial Atavism Model (SAM), an enhanced version of the atavistic theory, which suggests that cancer progression involves multiple atavistic reversions where cells regress through evolutionary stages, losing recently evolved traits first and reactivating primitive ones later. The Warburg effect, where cancer cells upregulate glycolysis and lactate production in the presence of oxygen instead of using oxidative phosphorylation, is one of the key feature of the SAM. It is associated with the metabolism of ancient cells living on Earth before the oxygenation of the atmosphere. This review addresses the question of whether cancer metabolism can be considered as an atavistic reversion. By analyzing several known characteristics of cancer metabolism, we reach the conclusion that this version of the atavistic theory does not provide an adequate conceptual frame for cancer research. Cancer metabolism spans a whole spectrum of metabolic states which cannot be fully explained by a sequential reversion to an ancient state. Moreover, we interrogate the nature of cancer metabolism and discuss its characteristics within the framework of the SAM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f5f41e098ff498aa32258ffc44af055a9872bf7" target='_blank'>
              Is Cancer Metabolism an Atavism?
              </a>
            </td>
          <td>
            Eric Fanchon, A. Stéphanou
          </td>
          <td>2024-06-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="ConspectusSophisticated genetic networks play a pivotal role in orchestrating cellular responses through intricate signaling pathways across diverse environmental conditions. Beyond the inherent complexity of natural cellular signaling networks, the construction of artificial signaling pathways (ASPs) introduces a vast array of possibilities for reshaping cellular responses, enabling programmable control of living organisms. ASPs can be integrated with existing cellular networks and redirect output responses as desired, allowing seamless communication and coordination with other cellular processes, thereby achieving designable transduction within cells. Among diversified ASPs, establishing connections between originally independent endogenous genes is of particular significance in modifying the genetic networks, so that cells can be endowed with new capabilities to sense and deal with abnormal factors related to differentiated gene expression (i.e., solve the issues of the aberrant gene expression induced by either external or internal stimuli). In a typical scenario, the two genes X and Y in the cell are originally expressed independently. After the introduction of an ASP, changes in the expression of gene X may exert a designed impact on gene Y, subsequently inducing the cellular response related to gene Y. If X represents a disease signal and Y serves as a therapeutic module, the introduction of the ASP empowers cells with a new spontaneous defense system to handle potential risks, which holds great potential for both fundamental and translational studies.In this Account, we primarily review our endeavors in the construction of RNA-mediated ASPs between endogenous genes that can respond to differentiated RNA expression. In contrast to other molecules that may be restricted to specific pathways, synthetic RNA circuits can be easily utilized and expanded as a general platform for constructing ASPs with a high degree of programmability and tunability for diversified functionalities through predictable Watson-Crick base pairing. We first provide an overview of recent advancements in RNA-based genetic circuits, encompassing but not limited to utilization of RNA toehold switches, siRNA and CRISPR systems. Despite notable progress, most reported RNA circuits have to contain at least one exogenous RNA X as input or one engineered RNA Y as a target, which is not suitable for establishing endogenous gene connections. While exogenous RNAs can be engineered and controlled as desired, constructing a general and efficient platform for manipulation of naturally occurring RNAs poses a formidable challenge, especially for the mammalian system. With a focus on this goal, we are devoted to developing efficient strategies to manipulate cell responses by establishing RNA-mediated ASPs between endogenous genes, particularly in mammalian cells. Our step-by-step progress in engineering customized cell signaling circuits, from bacterial cells to mammalian cells, from gene expression regulation to phenotype control, and from small RNA to long mRNA of low abundance and more complex secondary structures, is systematically described. Finally, future perspectives and potential applications of these RNA-mediated ASPs between endogenous genes are also discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3c9af75c2eecb3458a01918730b3af83f34ad44" target='_blank'>
              Building RNA-Mediated Artificial Signaling Pathways between Endogenous Genes.
              </a>
            </td>
          <td>
            Ruo-Yue Wu, Chao-Qun Wu, Fangzhou Xie, Xiwen Xing, Liang Xu
          </td>
          <td>2024-06-13</td>
          <td>Accounts of chemical research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The main epidemiological and clinical data on colorectal cancer, as well as the features of molecular pathology, are discussed in the literature review. Efforts are being putto identify promising targets, particularly small non-coding nucleotide sequences, which can lead to new treatments for this disease. The discovery of significant mutations that contribute to the development of colorectal tumors is a major step in the advancement of molecular oncology, as these mutations give rise to heterogeneous tumors that differ in their origin. These mutations play a significant role in the progression of the disease and are now being targeted for treatment. The prognosis for a disease is influenced by the patient's sensitivity to antitumor therapy. However, new approaches to finding effective targets for antitumor treatments face new fundamental challenges due to clinical issues. These issues include the epigenetic regulation of markers of oncogenesis, which allows for the development of new therapeutic strategies. RNA interference, in particular, has been linked to non-copying RNA sequences such as microRNAs. These microRNAs are associated with certain processes that can influence all aspects of oncogenesis. The diversity of microRNAs allows for a differentiated approach when treating tumors in various locations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4796e80e11216dc287a04bf6cbff831c3d542a28" target='_blank'>
              New Perspectives in Colorectal Cancers Treatment, the Role of MicroRNAs.
              </a>
            </td>
          <td>
            V. A. Belova, Liudmila V Spirina, A. Avgustinovich, S. G. Afanasiev, Maxim Yu Volkov M, D. I. Azovsky, Alexander M Volkov, T. S. Klyushina
          </td>
          <td>2024-07-24</td>
          <td>Current drug targets</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92d22ec19c45e84786f29e00379dcc92c97dbcbf" target='_blank'>
              Identifying key genes and functionally enriched pathways in acute myeloid leukemia by weighted gene co-expression network analysis.
              </a>
            </td>
          <td>
            Jimo Jian, Chenglu Yuan, Hongyuan Hao
          </td>
          <td>2024-07-09</td>
          <td>Journal of applied genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/502a5fb073ed81392e0d208c32c77f69b33b47ec" target='_blank'>
              The contributions of cytogenetics, genetics, and epigenetics to the stability of plants polyploidy
              </a>
            </td>
          <td>
            Arrashid Harun, Zhongming Fang, Chunli Chen
          </td>
          <td>2024-07-30</td>
          <td>Discover Plants</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5096f65eca5658128881222e362e03cc7a2beb2" target='_blank'>
              H3K27ac acts as a molecular switch for doxorubicin-induced activation of cardiotoxic genes
              </a>
            </td>
          <td>
            Yu Hong, Xinlan Li, Jia Li, Qiuyi He, Manbing Huang, Yubo Tang, Xiao Chen, Jie Chen, Ke-Jing Tang, Chao Wei
          </td>
          <td>2024-07-16</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The amplification of MET is a major cause of acquired resistance to targeted therapy in EGFR-mutant non-small-cell lung cancer (NSCLC), only to be temporarily restrained by the partial efficacy of MET inhibitors. This study reveals that the MET inhibitor has unexpectedly limited efficacy due to amplified MET triggering a strong positive feedback loop in the Wnt/β-catenin signaling pathway, allowing optimal functionality even when the MET pathway is suppressed again. To test this conjecture and specifically target the Wnt/β-catenin pathway, a cleverly designed Wnt condensative pro drug called WntSI is developed using reversible supramolecular self-assembly driven by liquidliquid phase separation (LLPS). This process involves a MET/pH-responsive peptide (Tyr-Pep) and a potent Wnt inhibitor known as CA. Upon recognition and phosphorylation of Tyr-Pep by over expressed MET in cells, it disrupts LLPS propensity and facilitates the disintegration of WntSI. Consequently,this enables it to suppress the carcinogenic effect mediated by β-catenin,effectively overcoming acquired resistance to EGFR-TKIs caused by MET amplification in both cell line-derived and patient-derived tumor xenograft (PDX) mouse models while maintaining exceptional biosecurity. This effective strategy not only suppresses the Wnt/β-catenin signaling pathway selectively, but also serves as an innovative example for pro-drug development through biologically responsive LLPS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7920ce89e1700b0b27a46645e0f64ce45c19988" target='_blank'>
              Reversing MET-Mediated Resistance in Oncogene-Driven NSCLC by MET-Activated Wnt Condensative Prodrug.
              </a>
            </td>
          <td>
            Na Liu, Xiaoqiang Zheng, Jin Yan, Aimin Jiang, Yu Yao, Wangxiao He
          </td>
          <td>2024-06-13</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Pancreatic malignancy is the fourth cause of cancer-related death in Western countries and is predicted to become the second leading cause of cancer-related mortality by 2030. The standard therapies (FOLFIRINOX and gemcitabine with nab-paclitaxel) are not resolutive because this type of cancer is also characterized by a high chemoresistance, due in part to the activity of the ATP Binding Cassette (ABC) pumps accounting for the reduction in the intracellular concentration of the drugs. In this work, we analyze the occurrence of single-nucleotide polymorphisms (SNPs) in the MDR-1 gene, in different pancreatic cancer cell lines, and in tissues from pancreatic cancer patients by DNA sequencing, as well as the expression levels of MDR-1 mRNA and protein, by qRT-PCR and Western Blot analysis. We found that gemcitabine-resistant cells, in conjunction with homozygosis of analyzed SNPs, showed high MDR-1 basal levels with further increases after gemcitabine treatment. Nevertheless, we did not observe in the human PDAC samples a correlation between the level of MDR-1 mRNA and protein expression and SNPs. Preliminary, we conclude that in our small cohort, these SNPs cannot be used as molecular markers for predicting the levels of MDR-1 mRNA/protein levels and drug responses in patients with PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64846ba52dc007b0047fe4ee9f27909ed0b816fb" target='_blank'>
              Characterization of Chemoresistance in Pancreatic Cancer: A Look at MDR-1 Polymorphisms and Expression in Cancer Cells and Patients
              </a>
            </td>
          <td>
            Giulia Girolimetti, Barbara Balena, Paola Cordella, T. Verri, L. Eusebi, M. Bozzetti, Cecilia Bucci, Flora Guerra
          </td>
          <td>2024-08-04</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d2a4f583e73c9388ee06b9414fc44f7c5cdf6bf" target='_blank'>
              Whole genome sequencing refines stratification and therapy of patients with clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            R. Culliford, Sam Lawrence, Charlie Mills, Z. Tippu, D. Chubb, A. Cornish, Lisa Browning, B. Kinnersley, R. Bentham, Amit Sud, H. Pallikonda, A. Frangou, Andreas J. Gruber, K. Litchfield, David C Wedge, James Larkin, S. Turajlic, R. Houlston
          </td>
          <td>2024-07-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Simple Summary This study investigates the impact of the BRAFV600E mutation, present in half of melanoma cases, on immune microenvironment and therapeutic response. Analyzing the Cancer Genome Atlas data, we uncover that the mutation correlates with reduced tumor mutation burden, indicating a lower generation of immune-stimulating neopeptides. Examination of immune subtypes reveal heightened immunosuppression in BRAFV600E-mutated tumors. Using melanoma cell lines harboring different genomic profiles, we observed an enhanced response to immunogenic cell death (ICD)—a crucial process eliciting anti-tumor immune responses—mediated by photodynamic therapy (PDT). Transcriptomic analysis highlights upregulation of IFNAR1, IFNAR2, and CXCL10 genes in BRAFV600E-mutated cells, suggesting their role in ICD. Our results describe the intricate relationship between the BRAFV600E mutation and immune responses, hinting at a potential link between this mutation and responsiveness to ICD-inducing therapies, particularly PDT, putatively linked to increased IFN-1 pathway activation. Abstract The BRAFV600E mutation, found in approximately 50% of melanoma cases, plays a crucial role in the activation of the MAPK/ERK signaling pathway, which promotes tumor cell proliferation. This study aimed to evaluate its impact on the melanoma immune microenvironment and therapeutic responses, particularly focusing on immunogenic cell death (ICD), a pivotal cytotoxic process triggering anti-tumor immune responses. Through comprehensive in silico analysis of the Cancer Genome Atlas data, we explored the association between the BRAFV600E mutation, immune subtype dynamics, and tumor mutation burden (TMB). Our findings revealed that the mutation correlated with a lower TMB, indicating a reduced generation of immunogenic neoantigens. Investigation into immune subtypes reveals an exacerbation of immunosuppression mechanisms in BRAFV600E-mutated tumors. To assess the response to ICD inducers, including doxorubicin and Me-ALA-based photodynamic therapy (PDT), compared to the non-ICD inducer cisplatin, we used distinct melanoma cell lines with wild-type BRAF (SK-MEL-2) and BRAFV600E mutation (SK-MEL-28, A375). We demonstrated a differential response to PDT between the WT and BRAFV600E cell lines. Further transcriptomic analysis revealed upregulation of IFNAR1, IFNAR2, and CXCL10 genes associated with the BRAFV600E mutation, suggesting their involvement in ICD. Using a gene reporter assay, we showed that PDT robustly activated the IFN-1 pathway through cGAS-STING signaling. Collectively, our results underscore the complex interplay between the BRAFV600E mutation and immune responses, suggesting a putative correlation between tumors carrying the mutation and their responsiveness to therapies inducing the IFN-1 pathway, such as the ICD inducer PDT, possibly mediated by the elevated expression of IFNAR1/2 receptors">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56ed26388b396064bd1f63bcbdf14dd7a6adc97c" target='_blank'>
              Impact of Genomic Mutation on Melanoma Immune Microenvironment and IFN-1 Pathway-Driven Therapeutic Responses
              </a>
            </td>
          <td>
            F. Mentucci, Elisa Ayelén Romero Nuñez, Agustina Ercole, Valentina Silvetti, J. Dal Col, M. Lamberti
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b903232278019fc20d595adb95c47c4a0e3a7532" target='_blank'>
              Deletion of IRC19 Causes Defects in DNA Double-Strand Break Repair Pathways in Saccharomyces cerevisiae.
              </a>
            </td>
          <td>
            Ju-Hee Choi, Oyungoo Bayarmagnai, Sung-Ho Bae
          </td>
          <td>2024-07-12</td>
          <td>Journal of microbiology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Actionable driver mutations account for 40–50% of NSCLC cases, and their identification clearly affects treatment choices and outcomes. Conversely, non-actionable mutations are genetic alterations that do not currently have established treatment implications. Among co-occurring alterations, the identification of concurrent actionable genomic alterations is a rare event, potentially impacting prognosis and treatment outcomes. Methods: We retrospectively evaluated the prevalence and patterns of concurrent driver genomic alterations in a large series of NSCLCs to investigate their association with clinicopathological characteristics, to assess the prognosis of patients whose tumor harbors concurrent alterations in the genes of interest and to explore their potential therapeutic implications. Results: Co-occurring driver alterations were identified in 26 out of 1520 patients with at least one gene alteration (1.7%). Within these cases, the incidence of concurrent actionable gene alterations was 39% (0.7% of the overall cohort). Among compound actionable gene mutations, EGFR was the most frequently involved gene (70%). The most frequent association was EGFR mutations with ROS1 rearrangement. Front-line targeted treatments were the preferred approach in patients with compound actionable mutations, with dismal median PFS observed (6 months). Conclusions: Advances in genomic profiling technologies are facilitating the identification of concurrent mutations. In patients with concurrent actionable gene alterations, integrated molecular and clinical data should be used to guide treatment decisions, always considering rebiopsy at the moment of disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59022475d568d630ac2b88c69a13d9f08f1373da" target='_blank'>
              Co-Occurring Driver Genomic Alterations in Advanced Non-Small-Cell Lung Cancer (NSCLC): A Retrospective Analysis
              </a>
            </td>
          <td>
            I. Attili, Riccardo Asnaghi, D. Vacirca, Riccardo Adorisio, A. Rappa, Alberto Ranghiero, Mariano Lombardi, C. Corvaja, V. Fuorivia, Ambra Carnevale Schianca, P. Trillo Aliaga, G. Spitaleri, E. del Signore, J. Guarize, Lorenzo Spaggiari, E. Guerini-Rocco, Nicola Fusco, F. de Marinis, A. Passaro
          </td>
          <td>2024-07-31</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="OBJECTIVE
Ionizing radiation generates genomic instability by promoting the accumulation of chromosomal rearrangements. The oncogenic translocation RET/PTC1 is present in more than 70% of radiation-induced thyroid cancers. Both RET and CCDC6, the genes implicated in RET/PTC1, are found within common fragile sites - chromosomal regions prone to DNA breakage during slight replication stress. Given that irradiated cells become more susceptible to genomic destabilization due to the accumulation of replication-stress-related double-strand breaks (DSBs), we explored whether RET and CCDC6 exhibit DNA breakage under replicative stress several days post-irradiation of thyroid cells.


METHODS
We analyzed the dynamic of DNA replication in human thyroid epithelial cells (HThy-ori-3.1) 4 days post a 5-Gy exposure using molecular DNA combing. The DNA replication schedule was evaluated through replication-timing experiments. We implemented a ChIP-qPCR assay to determine whether the RET and CCDC6 genes break following irradiation.


RESULTS
Our study indicates that replicative stress, occurring several days post-irradiation in thyroid cells, primarily causes DSBs in the RET gene. We discovered that both the RET and CCDC6 genes undergo late replication in thyroid cells. However, only RET's replication rate is notably delayed after irradiation.


CONCLUSION
The findings suggest that post-irradiation in the RET gene causes a breakage in the replication fork, which could potentially invade another genomic area, including CCDC6. As a result, this could greatly contribute to the high prevalence of chromosomal RET/PTC rearrangements seen in patients exposed to external radiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eea49490c4342b6b738c402ce79c72ea23e5f552" target='_blank'>
              A post-irradiation-induced replication stress promotes RET proto-oncogene breakage.
              </a>
            </td>
          <td>
            Fabio Hecht, Laura Valerio, Carlos Frederico Lima Gonçalves, Marylin Harinquet, Rabii Ameziane El Hassani, Denise P. Carvalho, Stephane Koundrioukoff, Jean-Charles Cadoret, Corinne Dupuy
          </td>
          <td>2024-07-01</td>
          <td>European thyroid journal</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Transcriptional enhancers can regulate individual or multiple genes through long-range three-dimensional (3D) genome interactions, and these interactions are commonly altered in cancer. Yet, the functional relationship between changes in 3D interactions associated with regulatory regions and differential gene expression appears context-dependent. In this study, we used HiChiP to capture changes in 3D genome interactions between active regulatory regions of endometrial cancer cells in response to estrogen treatment and uncovered significant differential long-range interactions that are strongly enriched for estrogen receptor α (ER) bound sites (ERBS). The ERBS anchoring differential loops with either a gene’s promoter or distal regions were correlated with larger transcriptional responses to estrogen compared to ERBS not involved in differential interactions. To functionally test this observation, CRISPR-based Enhancer-i was used to deactivate specific ERBS, which revealed a wide range of effects on the transcriptional response to estrogen. However, these effects are only subtly and not significantly stronger for ERBS in differential loops. In addition, we observed an enrichment of 3D interactions between the promoters of estrogen up-regulated genes and found that looped promoters can work together cooperatively. Overall, our work suggests that changes in 3D genome structure upon estrogen treatment identify some functionally important regulatory regions; however, these changes aren’t required for a transcriptional response to E2 in endometrial cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d0d32b1b8568f5b7b8778716b0bae7e2d53785" target='_blank'>
              Estrogen-induced chromatin looping changes identify a subset of functional regulatory elements
              </a>
            </td>
          <td>
            Hosiana Abewe, Alexandra Richey, Jeffery M. Vahrenkamp, Matthew Ginley-Hidinger, Craig M Rush, Noel Kitchen, Xiaoyang Zhang, Jason Gertz
          </td>
          <td>2024-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Although the study of leukemogenesis has traditionally focused on protein-coding genes, the role of enhancer dysregulation is becoming increasingly recognized. The advent of high-throughput sequencing, together with a better understanding of enhancer biology, has revealed how various genetic and epigenetic lesions produce oncogenic enhancers that drive transformation. These aberrations include translocations that lead to enhancer hijacking, point mutations that modulate enhancer activity, and copy number alterations that modify enhancer dosage. In this review, we describe these mechanisms in the context of leukemia and discuss potential therapeutic avenues to target these regulatory elements. Significance: Large-scale sequencing projects have uncovered recurrent gene mutations in leukemia, but the picture remains incomplete: some patients harbor no such aberrations, whereas others carry only a few that are insufficient to bring about transformation on their own. One of the missing pieces is enhancer dysfunction, which only recently has emerged as a critical driver of leukemogenesis. Knowledge of the various mechanisms of enhancer dysregulation is thus key for a complete understanding of leukemia and its causes, as well as the development of targeted therapies in the era of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4963169247626a5146ad859dcbd326045f924327" target='_blank'>
              Oncogenic Enhancers in Leukemia.
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Ruud Delwel
          </td>
          <td>2024-08-02</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="ABSTRACT Natural killer (NK) cells play a crucial role in antitumoral and antiviral responses. Yet, cancer cells can alter themselves or the microenvironment through the secretion of cytokines or other factors, hindering NK cell activation and promoting a less cytotoxic phenotype. These resistance mechanisms, often referred to as the “hallmarks of cancer” are significantly influenced by the activation of oncogenes, impacting most, if not all, of the described hallmarks. Along with oncogenes, other types of genes, the tumor suppressor genes are frequently mutated or modified during cancer. Traditionally, these genes have been associated with uncontrollable tumor growth and apoptosis resistance. Recent evidence suggests oncogenic mutations extend beyond modulating cell death/proliferation programs, influencing cancer immunosurveillance. While T cells have been more studied, the results obtained highlight NK cells as emerging key protagonists for enhancing tumor cell elimination by modulating oncogenic activity. A few recent studies highlight the crucial role of oncogenic mutations in NK cell-mediated cancer recognition, impacting angiogenesis, stress ligands, and signaling balance within the tumor microenvironment. This review will critically examine recent discoveries correlating oncogenic mutations to NK cell-mediated cancer immunosurveillance, a relatively underexplored area, particularly in the era dominated by immune checkpoint inhibitors and CAR-T cells. Building on these insights, we will explore opportunities to improve NK cell-based immunotherapies, which are increasingly recognized as promising alternatives for treating low-antigenic tumors, offering significant advantages in terms of safety and manufacturing suitability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfefe1207e6a05f83b08c29611ae01cb2a8d9e94" target='_blank'>
              In-depth analysis of the interplay between oncogenic mutations and NK cell-mediated cancer surveillance in solid tumors
              </a>
            </td>
          <td>
            C. Pesini, Laura Artal, Jorge Paúl Bernal, Diego Sánchez Martinez, Julián Pardo, A. Ramírez-Labrada
          </td>
          <td>2024-07-18</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="INTRODUCTION
Several biomarkers are currently available to address targeted treatments in cancer patients, with lung malignancies representing one of the best examples.


CASE DESCRIPTION
We report the case of a patient affected by advanced non-small cell lung cancer with an uncommon histology and a complex biology. The use of a large next-generation sequencing (NGS) NGS panel allowed us to identify an extremely rare BRAF mutation (V600Q), a MET amplification, a high tumor mutational burden, a germline pathogenetic BRCA1 mutation and a homologous recombination deficiency through RAD51 assay. The treatment decision was driven by the abundance of molecular information.


CONCLUSIONS
This case highlights that an attentive and critical evaluation of molecular reports is key for the tailoring of treatment algorithms at the patient-level scale.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3210c6a7f6bdd7275b1dd451dca4a9b663d889f7" target='_blank'>
              Multiple targets, germline BRCA1 mutation and HRD in a lung cancer patient: Molecular considerations and treatment decision-making.
              </a>
            </td>
          <td>
            F. Perrone, F. Facchinetti, Benedetta Pellegrino, R. Minari, L. Gnetti, C. Azzoni, L. Bottarelli, N. Campanini, Juan Francisco Grau-Bejar, Anna Mingozzi, V. Cognigni, Marcello Tiseo
          </td>
          <td>2024-06-13</td>
          <td>Tumori</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fb4b36c7afe762fcfe4d2e605f3f7060a7a118b" target='_blank'>
              ATM inhibition exploits checkpoint defects and ATM-dependent double strand break repair in TP53-mutant glioblastoma
              </a>
            </td>
          <td>
            D. Laverty, Shiv K Gupta, Gary A. Bradshaw, Alexander S Hunter, B. Carlson, Nery Matias Calmo, Jiajia Chen, Shulan Tian, J. Sarkaria, Zachary D. Nagel
          </td>
          <td>2024-06-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Tumor protein p53 (TP53) mediates DNA repair and cell proliferation in growing cells. The TP53 gene is a tumor suppressor that regulates the expression of target genes in response to multiple cellular stress factors. Key target genes are involved in crucial cellular events such as DNA repair, cell cycle regulation, apoptosis, metabolism, and senescence. TP53 genetic variants and the activity of the wild-type p53 protein (WT-p53) have been linked to a wide range of tumorigenesis. Various genetic and epigenetic alterations, including germline and somatic mutations, loss of heterozygosity, and DNA methylation, can alter TP53 activity, potentially resulting in cancer initiation and progression. This study was designed to screen three reported mutations in the DNA-binding domain of the p53 protein in breast cancer, to evaluate the relative susceptibility and risk associated with breast cancer in the local population. Genomic DNA was isolated from 30 breast tumor tissues along with controls. Tetra and Tri ARMS PCR were performed to detect mutations in the TP53 coding region. For SNPs c.637C>T and c.733C>T, all analyzed cases were homozygous for the wild-type allele ‘C,’ while for SNP c.745A>G, all cases were homozygous for the wild-type allele ‘A.’ These results indicate no relevance of these three SNPs to cancer progression in our study cohort. Additionally, the findings from whole exon sequencing will help to predict more precise outcomes and assess the importance of TP53 gene mutations in breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bffe267deb645fc61554e684edc8c0bafc2c11ee" target='_blank'>
              Analysis of Genetic Alterations in TP53 Gene in Breast Cancer – A Secondary Publication
              </a>
            </td>
          <td>
            Baqaur Rehman, Muhammad Abubakar, Muhammad Naeem Kiani, Rooma Ayyoub
          </td>
          <td>2024-06-19</td>
          <td>Proceedings of Anticancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9fb6e4e6751f59e5a7f630fe095f0e2f627ca4d" target='_blank'>
              Copy number losses of oncogenes and gains of tumor suppressor genes generate common driver mutations
              </a>
            </td>
          <td>
            Elizaveta Besedina, F. Supek
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Mendelian disorders, arising from pathogenic variations within single genetic loci, often manifest as neurodevelopmental disorders (NDDs), affecting a significant portion of the pediatric population worldwide. These disorders are marked by atypical brain development, intellectual disabilities, and various associated phenotypic traits. Genetic testing aids in clinical diagnoses, but inconclusive results can prolong confirmation processes. Recent focus on epigenetic dysregulation has led to the discovery of DNA methylation signatures, or episignatures, associated with NDDs, accelerating diagnostic precision. Notably, TRIP12 and USP7, genes involved in the ubiquitination pathway, exhibit specific episignatures. Understanding the roles of these genes within the ubiquitination pathway sheds light on their potential influence on episignature formation. While TRIP12 acts as an E3 ligase, USP7 functions as a deubiquitinase, presenting contrasting roles within ubiquitination. Comparison of phenotypic traits in patients with pathogenic variations in these genes reveals both distinctions and commonalities, offering insights into underlying pathophysiological mechanisms. This review contextualizes the roles of TRIP12 and USP7 within the ubiquitination pathway, their influence on episignature formation, and the potential implications for NDD pathogenesis. Understanding these intricate relationships may unveil novel therapeutic targets and diagnostic strategies for NDDs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caf7944ed6368b43b27c4e12ad9a33aa3e98512d" target='_blank'>
              Molecular signatures in Mendelian neurodevelopment: a focus on ubiquitination driven DNA methylation aberrations
              </a>
            </td>
          <td>
            Liselot van der Laan, Nicky ten Voorde, Marcel M A M Mannens, P. Henneman
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Molecular Neuroscience</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Human telomerase reverse transcriptase (hTERT) may have noncanonical functions in transcriptional regulation and metabolic reprogramming in cancer cells, but it is a challenging target. We thus developed small-molecule ligands targeting hTERT promoter G-quadruplex DNA structures (hTERT G4) to downregulate hTERT expression. Ligand 5 showed high affinity toward hTERT G4 (Kd = 1.1 μM) and potent activity against triple-negative breast cancer cells (MDA-MB-231, IC50 = 1 μM). In cell-based assays, 5 not only exerts markedly inhibitory activity on classical telomere functions including decreased telomerase activity, shortened telomere length, and cellular senescence but also induces DNA damage, acute cellular senescence, and apoptosis. This study reveals that hTERT G4-targeting ligand may cause mitochondrial dysfunction, disrupt iron metabolism and activate ferroptosis in cancer cells. The in vivo antitumor efficacy of 5 was also evaluated in an MDA-MB-231 xenograft mouse model and approximately 78.7% tumor weight reduction was achieved. No observable toxicity against the major organs was observed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1d3d47d8b1822b8c2901a320f1107b7691744c1" target='_blank'>
              Targeting hTERT Promoter G-Quadruplex DNA Structures with Small-Molecule Ligand to Downregulate hTERT Expression for Triple-Negative Breast Cancer Therapy.
              </a>
            </td>
          <td>
            W. Long, Yao-Xun Zeng, Bo-Xin Zheng, Yu-Bo Li, Ya-Kun Wang, K. Chan, Meng-Ting She, Yu-Jing Lu, Chunyang Cao, Wing-Leung Wong
          </td>
          <td>2024-07-10</td>
          <td>Journal of medicinal chemistry</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7189b9bbbd6276b2c06531ac44b4da640c222158" target='_blank'>
              Small cell lung cancer: emerging subtypes, signaling pathways, and therapeutic vulnerabilities
              </a>
            </td>
          <td>
            Jing Zhang, Xiaoping Zeng, Qiji Guo, Zhenxin Sheng, Yan Chen, Shiyue Wan, Lele Zhang, Peng Zhang
          </td>
          <td>2024-08-05</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Strong epigenetic pan-cancer biomarkers are required to meet several current, urgent clinical needs and to further improve the present chemotherapeutic standard. We have concentrated on the investigation of epigenetic alteration of the hTERT gene, which is frequently epigenetically dysregulated in a number of cancers in specific developmental stages. Distinct DNA methylation profiles were identified in our data on early urothelial cancer. An efficient EpihTERT assay could be developed utilizing suitable combinations with sequence-dependent thermodynamic parameters to distinguish between differentially methylated states. We infer from this data set, the epigenetic context, and the related literature that a CpG-rich, 2800 bp region, a prominent CpG island, surrounding the transcription start of the hTERT gene is the crucial epigenetic zone for the development of a potent biomarker. In order to accurately describe this region, we have named it “Acheron” (Ἀχέρων). In Greek mythology, this is the river of woe and misery and the path to the underworld. Exploitation of the DNA methylation profiles focused on this region, e.g., idiolocal normalized Methylation Specific PCR (IDLN-MSP), opens up a wide range of new possibilities for diagnosis, determination of prognosis, follow-up, and detection of residual disease. It may also have broad implications for the choice of chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3ee3e7f8ca8ba62fbce3d513f74a833baab6859" target='_blank'>
              hTERT Epigenetics Provides New Perspectives for Diagnosis and Evidence-Based Guidance of Chemotherapy in Cancer
              </a>
            </td>
          <td>
            S. Santourlidis, Marcos J. Araúzo-Bravo, Robert T. Brodell, Mohamed Hassan, M. Bendhack
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The repair of DNA double strand breaks (DSBs) is crucial for the maintenance of genome stability and cell viability. DSB repair (DSBR) in cells is mediated through several mechanisms: homologous recombination (HR), non-homologous end joining (NHEJ), microhomology-mediated end joining (MMEJ), and single strand annealing (SSA). Cellular assays are essential to measure the proficiency and modulation of these pathways in response to various stimuli. Here, we present a suite of extrachromosomal reporter assays that each measure the reconstitution of a nanoluciferase reporter gene by one of the four major DSBR pathways in cells. Upon transient transfection into cells of interest, repair of pathway-specific reporter substrates can be measured in under 24 h by the detection of Nanoluciferase (NanoLuc) luminescence. These robust assays are quantitative, sensitive, titratable, and amenable to a high-throughput screening format. These properties provide broad applications in DNA repair research and drug discovery, complementing the currently available toolkit of cellular DSBR assays.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/270bb7c41b9ec2cbfe7fe21ae86f536aa1f2b28a" target='_blank'>
              Assessment of DNA Double Strand Break Repair Activity Using High-throughput and Quantitative Luminescence-based Reporter Assays.
              </a>
            </td>
          <td>
            Diego Grande, E. Rajendra, Bethany Mason, Alessandro Galbiati, S. Boulton, Graeme C. M. Smith, Helen M R Robinson
          </td>
          <td>2024-06-14</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>69</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [20],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>